<!DOCTYPE html>
<html>
<head>
<meta charset='utf-8' />
<link rel='stylesheet' media='all' href='../libs/cancer-model.css' />
<link href='../libs/vis/vis.css' rel='stylesheet' type='text/css' />
<script type='text/javascript' src='../libs/three.js'></script>
<script type='text/javascript' src='../libs/dat.gui.js'></script>
<script type='text/javascript' src='../libs/stats.js'></script>
<script type='text/javascript' src='../libs/OrbitControls.js'></script>
<script type='text/javascript' src='../libs/rcolor.js'></script>
<script src='../libs/vis/vis.js'></script>
<script src='../libs/cancer-model.js'></script>
<!-- Plotly.js -->
<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
<title>Results from running cancer simulation</title>
<script>
var l = [];
var s = [];
var replicates = [];
var a = [];
var number_of_replicates_requested = 50; // added manually
var start_time = '1 Jan 2000 midnight'; // added manually
var running_3D = true;
var replicate_counter = 0;
var gene_nodes = [];
var input_nodes = [];
var output_nodes = [];
var links_between_nodes = [];var minimum_x = Number.MAX_VALUE;
var maximum_x = Number.MIN_VALUE;
var minimum_y = Number.MAX_VALUE;
var maximum_y = Number.MIN_VALUE;
var parameters,
    number_of_replicates_requested,
    mutations_file_contents;

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>
<!--
			This GraphML document was generated from GXL by
			a GXL-to-GraphML conversion style sheet.
		-->
<graphml>
   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>
   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>
   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>
   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>
   <key id="MY-VALUE"
        for="node"
        attr.name="MY-VALUE"
        attr.type="string"/>
   <key id="MY-SHAPE"
        for="node"
        attr.name="MY-SHAPE"
        attr.type="string"/>
   <key id="PROBABILITY-OF-LINK"
        for="edge"
        attr.name="PROBABILITY-OF-LINK"
        attr.type="string"/>
   <key id="SIGN-OF-LINK"
        for="edge"
        attr.name="SIGN-OF-LINK"
        attr.type="string"/>
   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>
   <graph id="MAPK_large_HIF1" edgedefault="directed">
      <node id="EGFR_stimulus">
         <data key="MY-ID">EGFR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce EGFR activation
</node>
      <node id="FGFR3_stimulus">
         <data key="MY-ID">FGFR3_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce FGFR3 activation
</node>
      <node id="TGFBR_stimulus">
         <data key="MY-ID">TGFBR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce TGFBR activation

</node>
      <node id="DNA_damage">
         <data key="MY-ID">DNA_damage</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: DNA damaging agents/therapies action

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</node>
      <node id="Apoptosis">
         <data key="MY-ID">Apoptosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: apoptosis enablement

Modelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="Growth_Arrest">
         <data key="MY-ID">Growth_Arrest</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p21</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: growth arrest enablement

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="Proliferation">
         <data key="MY-ID">Proliferation</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and MYC and not p21</data>
         <data key="MY-SHAPE">rect</data>Proliferation

VARIABLE: proliferation enablement

Proliferation is an output variable of the model.

Modelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832

</node>
      <node id="ERK">
         <data key="MY-ID">ERK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MEK1_2</data>
         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1
MAPK3 = mitogen-activated protein kinase 3

VARIABLE: ERK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
RSKs interact with ERK and dissociate upon activation.
PMID:15187187

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p38">
         <data key="MY-ID">p38</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11
MAPK12 = mitogen-activated protein kinase 12
MAPK13 = mitogen-activated protein kinase 13
MAPK14 = mitogen-activated protein kinase 14

VARIABLE: p38 (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
MAX is phosphorylated (and activated) by p38, through complex formation.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187

ELK1 is a nuclear p38 target.
PMID:20506250

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="JNK">
         <data key="MY-ID">JNK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8
MAPK9 = mitogen-activated protein kinase 9
MAPK10 = mitogen-activated protein kinase 10

VARIABLE: JNK (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
JNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
ELK1 is a nuclear JNK target.
MEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="p53">
         <data key="MY-ID">p53</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>
         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53

VARIABLE: p53 phosphorylation level

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942

Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.
PMID:17158541

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.
PMID:12217521</node>
      <node id="p21">
         <data key="MY-ID">p21</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not AKT and p53</data>
         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)

VARIABLE: p21 activation level

p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

p21 is transcriptionally induced by the p53 protein.
p21 is a CDK inhibitor.
PMID:17158541

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="TGFBR">
         <data key="MY-ID">TGFBR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR_stimulus</data>
         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1
TGFBR2 = transforming growth factor, beta receptor II (70/80kDa)
TGFBR3 = transforming growth factor, beta receptor III

VARIABLE: TGFBR activation level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta.
PMID:21614932

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</node>
      <node id="EGFR">
         <data key="MY-ID">EGFR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>
         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor

VARIABLE: EGFR activation level

The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</node>
      <node id="FGFR3">
         <data key="MY-ID">FGFR3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>
         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3

VARIABLE: FGFR3 activation level

Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="ATM">
         <data key="MY-ID">ATM</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">DNA_damage</data>
         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated

VARIABLE: ATM phosphorylation level

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942
</node>
      <node id="TAOK">
         <data key="MY-ID">TAOK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ATM</data>
         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1
TAOK2 = TAO kinase 2
TAOK3 = TAO kinase 3

VARIABLE: TAOK (any isoform) phosphorylation level

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</node>
      <node id="MAX">
         <data key="MY-ID">MAX</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X

VARIABLE: MAX phosphorylation level

MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</node>
      <node id="GRB2">
         <data key="MY-ID">GRB2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2

VARIABLE: GRB2 recruitment by activated RTKs

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</node>
      <node id="FRS2">
         <data key="MY-ID">FRS2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>
         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2

VARIABLE: FRS2 phosphorylation level

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</node>
      <node id="PI3K">
         <data key="MY-ID">PI3K</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>
         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide

VARIABLE: PI3K activation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Recruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497</node>
      <node id="AP1">
         <data key="MY-ID">AP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JUN and (FOS or ATF2)</data>
         <data key="MY-SHAPE">ellipse</data>AP-1

VARIABLE: AP-1 formation

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345

AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="PPP2CA">
         <data key="MY-ID">PPP2CA</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme

VARIABLE: PPP2CA activation level

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</node>
      <node id="MEK1_2">
         <data key="MY-ID">MEK1_2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1
MAP2K2 = mitogen-activated protein kinase kinase 2

VARIABLE: MEK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="DUSP1">
         <data key="MY-ID">DUSP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB</data>
         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1

VARIABLE: DUSP1 gene expression level

DUSP1 is a CREB target gene.
PMID:19815709

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</node>
      <node id="MYC">
         <data key="MY-ID">MYC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>
         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)

VARIABLE: MYC activation level (implies gene expression)

Modelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.
MYC inhibits p16, whereas it activates p14.
PMID:17158541

MAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345

Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</node>
      <node id="AKT">
         <data key="MY-ID">AKT</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and not PTEN</data>
         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1
AKT2 = v-akt murine thymoma viral oncogene homolog 2
AKT3 = v-akt murine thymoma viral oncogene homolog 3

VARIABLE: AKT (any isoform) phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186

p21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="PLCG">
         <data key="MY-ID">PLCG</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FGFR3</data>
         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1
PLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)

VARIABLE: PLCG (any isoform) phosphorylation level

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="PKC">
         <data key="MY-ID">PKC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PLCG</data>
         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha
PRKCB = protein kinase C, beta
PRKCG = protein kinase C, gamma

VARIABLE: PKC (any isoform) enzymatic activity

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="GADD45">
         <data key="MY-ID">GADD45</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SMAD or p53</data>
         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha
GADD45B = growth arrest and DNA-damage-inducible, beta
GADD45G = growth arrest and DNA-damage-inducible, gamma

VARIABLE: GADD45 gene (any isoform) expression level

The GADD45 gene is inducible by various environmental stresses.
Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</node>
      <node id="ELK1">
         <data key="MY-ID">ELK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family

VARIABLE: ELK1 phosphorylation level

The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

ELK1 is a nuclear p38 target.
PMID:20506250

ELK1 is a nuclear JNK target.
ELK1 induces c-fos expression.
PMID:11274345</node>
      <node id="FOS">
         <data key="MY-ID">FOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>
         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog

VARIABLE: FOS phosphorylation level

Further observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
ELK1 induces c-fos expression.
PMID:11274345

FOS is a CREB target gene.
PMID:19815709

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</node>
      <node id="ATF2">
         <data key="MY-ID">ATF2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2

VARIABLE: ATF2 phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</node>
      <node id="JUN">
         <data key="MY-ID">JUN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK</data>
         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene

VARIABLE: JUN phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</node>
      <node id="MSK">
         <data key="MY-ID">MSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or p38</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4
RPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5

VARIABLE: MSK (any isoform) phosphorylation level

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</node>
      <node id="CREB">
         <data key="MY-ID">CREB</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MSK</data>
         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1

VARIABLE: CREB phosphorylation level

MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</node>
      <node id="RSK">
         <data key="MY-ID">RSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1
RPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2
RPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3
RPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6

VARIABLE: RSK (any isoform) phosphorylation level

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</node>
      <node id="SMAD">
         <data key="MY-ID">SMAD</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2
SMAD3 = SMAD family member 3
SMAD4 = SMAD family member 4

VARIABLE: SMAD gene (any isoform) expression level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</node>
      <node id="MTK1">
         <data key="MY-ID">MTK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GADD45</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4

VARIABLE: MTK1 phosphorylation level

The delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</node>
      <node id="SPRY">
         <data key="MY-ID">SPRY</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)

VARIABLE: SPRY2 phosphorylation level

Spry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.
PMID:15173823</node>
      <node id="RAF">
         <data key="MY-ID">RAF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>
         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1
BRAF = v-raf murine sarcoma viral oncogene homolog B1
ARAF = v-raf murine sarcoma 3611 viral oncogene homolog

VARIABLE: RAF (any isoform) phosphorylation

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

PKCalpha directly phosphorylates and activates Raf-1.
PMID:8321321

Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</node>
      <node id="GAB1">
         <data key="MY-ID">GAB1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 or PI3K</data>
         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1

VARIABLE: GAB1 phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
PMID:15567848

Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</node>
      <node id="PDK1">
         <data key="MY-ID">PDK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PI3K</data>
         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1

VARIABLE: PDK1 activation level

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</node>
      <node id="p70">
         <data key="MY-ID">p70</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1
RPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2

VARIABLE: p70 (any isoform) phosphorylation level

PDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.
PMID:12040186

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p14">
         <data key="MY-ID">p14</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MYC</data>
         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)

VARIABLE: p14 activation level

MYC inhibits p16, whereas it activates p14.
The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</node>
      <node id="FOXO3">
         <data key="MY-ID">FOXO3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK and not AKT</data>
         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3

VARIABLE: FOXO3 phosphorylation level

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</node>
      <node id="RAS">
         <data key="MY-ID">RAS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SOS or PLCG</data>
         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog
KRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog
NRAS = neuroblastoma RAS viral (v-ras) oncogene homolog

VARIABLE: RAS (any isoform) phosphorylation level

Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Most of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="SOS">
         <data key="MY-ID">SOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 and not RSK</data>
         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)
SOS2 = son of sevenless homolog 2 (Drosophila)

VARIABLE: SOS (any isoform) recruitment by GRB2

Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373
</node>
      <node id="MDM2">
         <data key="MY-ID">MDM2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(p53 or AKT) and not p14</data>
         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)

VARIABLE: MDM2 gene expression level

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.
PMID:17158541

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850
</node>
      <node id="BCL2">
         <data key="MY-ID">BCL2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB and AKT</data>
         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2

VARIABLE: BCL2 activation level (implies gene expression)

Modelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.

DUSP1, FOS, BCL2 are CREB target genes.
PMID:19815709

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="TAK1">
         <data key="MY-ID">TAK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7

VARIABLE: TAK1 phosphorylation level

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="MAP3K1_3">
         <data key="MY-ID">MAP3K1_3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">RAS</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1
MAP3K2 = mitogen-activated protein kinase kinase kinase 2
MAP3K3 = mitogen-activated protein kinase kinase kinase 3

VARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</node>
      <node id="PTEN">
         <data key="MY-ID">PTEN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p53</data>
         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog

VARIABLE: PTEN protein expression level

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="HIF1">
         <data key="MY-ID">HIF1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="EGF">
         <data key="MY-ID">EGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="VEGF">
         <data key="MY-ID">VEGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Oxygen_supply">
         <data key="MY-ID">Oxygen_supply</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Necrosis">
         <data key="MY-ID">Necrosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not Oxygen_supply</data>
         <data key="MY-SHAPE">rect</data>
      </node>
      <edge source="AP1" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</edge>
      <edge source="EGFR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910</edge>
      <edge source="EGFR" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</edge>
      <edge source="Oxygen_supply" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="Oxygen_supply" target="Necrosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MSK" target="CREB" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187</edge>
      <edge source="MSK" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</edge>
      <edge source="MEK1_2" target="ERK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="HIF1" target="EGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="HIF1" target="VEGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="DUSP1" target="p38" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="DUSP1" target="JNK" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="PI3K" target="PDK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124</edge>
      <edge source="PI3K" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="FGFR3" target="FRS2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="FGFR3" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221</edge>
      <edge source="DNA_damage" target="ATM" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</edge>
      <edge source="FRS2" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="MAX" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345</edge>
      <edge source="ATF2" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="ERK" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="ERK" target="RSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952</edge>
      <edge source="ERK" target="SPRY" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823</edge>
      <edge source="ERK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="ERK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ERK" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650
</edge>
      <edge source="ERK" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</edge>
      <edge source="ERK" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</edge>
      <edge source="ERK" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="CREB" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="DUSP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;



DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="JUN" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="FOXO3" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166</edge>
      <edge source="PLCG" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</edge>
      <edge source="PLCG" target="PKC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848</edge>
      <edge source="SOS" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929</edge>
      <edge source="SOS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="p70" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="p70" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MDM2" target="p53" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="TGFBR" target="SMAD" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="TGFBR" target="TAK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TGFBR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</edge>
      <edge source="GAB1" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="MTK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="MTK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="GRB2" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848</edge>
      <edge source="GRB2" target="SOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910</edge>
      <edge source="GRB2" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="GRB2" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848</edge>
      <edge source="GRB2" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="BCL2" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="ATM" target="TAOK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897</edge>
      <edge source="ATM" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942</edge>
      <edge source="GADD45" target="MTK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="PTEN" target="AKT" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="p14" target="MDM2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</edge>
      <edge source="PPP2CA" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="PPP2CA" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="p38" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932</edge>
      <edge source="p38" target="MAX" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</edge>
      <edge source="p38" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.
PMID:20506250
</edge>
      <edge source="p38" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="p53" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</edge>
      <edge source="p53" target="p21" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.
PMID:17158541</edge>
      <edge source="p53" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="p53" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508</edge>
      <edge source="p53" target="PTEN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="SMAD" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="SPRY" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="SPRY" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="EGFR_stimulus" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ELK1" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 induces c-fos expression.
PMID:11274345</edge>
      <edge source="TAK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TAK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="AKT" target="FOXO3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</edge>
      <edge source="AKT" target="p21" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383</edge>
      <edge source="AKT" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="AKT" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186</edge>
      <edge source="AKT" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850</edge>
      <edge source="AKT" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359</edge>
      <edge source="AKT" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="RAF" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="FOS" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="TAOK" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="TAOK" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="JNK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.
PMID:11274345</edge>
      <edge source="JNK" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="JNK" target="JUN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</edge>
      <edge source="JNK" target="FOXO3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971</edge>
      <edge source="p21" target="Proliferation" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="p21" target="Growth_Arrest" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="PKC" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473</edge>
      <edge source="PKC" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321</edge>
      <edge source="PKC" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</edge>
      <edge source="RSK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="RSK" target="SOS" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</edge>
      <edge source="PDK1" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="PDK1" target="AKT" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</edge>
      <edge source="MYC" target="p14" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.
PMID:17158541</edge>
      <edge source="MYC" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</edge>
      <edge source="RAS" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929</edge>
      <edge source="RAS" target="MAP3K1_3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="RAS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.
Proliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.
Input signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>
</graphml>
';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
l[1]=
[{added: 17},
{added: 35},
{added: 31},
{added: 39},
{added: 15},
{added: 52},
{added: 12},
{added: 6},
{added: 50},
{added: 32},
{added: 23},
{added: 41},
{added: 7},
{added: 25},
{added: 22},
{added: 47},
{added: 14},
{added: 20},
{added: 28},
{added: 13},
{added: 44},
{added: 37},
{added: 4},
{added: 49},
{added: 27},
{added: 53},
{added: 40},
{added: 9},
{added: 36},
{added: 46},
{added: 11},
{added: 10},
{added: 38},
{added: 24},
{added: 45},
{added: 26},
{added: 30},
{added: 8},
{added: 43},
{added: 48},
{added: 5},
{added: 42},
{added: 21},
{added: 33},
{added: 34},
{added: 19},
{added: 18},
{added: 29},
{added: 16},
{added: 51},
{x: 2, y: 2, z: 1, who: 50},
{c: 1, x: 2, y: 1, z: 1, who: 4},
{x: 2, y: 1, z: 0, who: 10},
{x: -2, y: 1, z: 1, who: 36},
{c: 1, x: 2, y: 0, z: 1, who: 32},
{c: 1, x: 0, y: -2, z: 1, who: 21},
{x: 2, y: -2, z: 0, who: 26},
{x: 2, y: -2, z: 1, who: 5},
{x: 1, y: 2, z: 1, who: 52},
{x: -1, y: 1, z: 1, who: 47},
{c: 1, x: 0, y: 2, z: 1, who: 45},
{x: -2, y: -1, z: 1, who: 34},
{c: 1, x: 2, y: -1, z: 1, who: 42},
{x: -1, y: -2, z: 1, who: 30},
{c: 1, x: -1, y: 0, z: 1, who: 41},
{c: 1, x: 0, y: -1, z: 1, who: 29},
{c: 1, x: 2, y: 0, z: 0, who: 23},
{c: 1, x: 2, y: -1, z: 0, who: 20},
{x: 1, y: -2, z: 1, who: 49},
{c: 1, x: 1, y: 0, z: 1, who: 28},
{x: 0, y: 0, z: 0, who: 33},
{x: -1, y: 2, z: 1, who: 15},
{x: 1, y: 0, z: 0, who: 37},
{c: 1, x: -2, y: -2, z: 0, who: 17},
{c: 1, x: -1, y: 1, z: 0, who: 51},
{x: 1, y: 2, z: 0, who: 8},
{c: 1, x: 1, y: -2, z: 0, who: 18},
{x: -1, y: 2, z: 0, who: 44},
{c: 1, x: -2, y: 0, z: 0, who: 11},
{c: 1, x: -1, y: -1, z: 1, who: 25},
{x: 0, y: -1, z: 0, who: 16},
{c: 1, x: 1, y: -1, z: 1, who: 46},
{c: 1, x: 1, y: -1, z: 0, who: 31},
{c: 1, x: 0, y: 1, z: 0, who: 35},
{x: -2, y: 2, z: 1, who: 38},
{c: 1, x: -1, y: -2, z: 0, who: 22},
{x: 1, y: 1, z: 1, who: 6},
{c: 1, x: -2, y: -1, z: 0, who: 13},
{c: 1, x: -2, y: 2, z: 0, who: 27},
{x: 0, y: -2, z: 0, who: 14},
{x: 1, y: 1, z: 0, who: 19},
{x: 0, y: 0, z: 1, who: 40},
{c: 1, x: -1, y: -1, z: 0, who: 7},
{c: 1, x: 0, y: 2, z: 0, who: 43},
{x: -2, y: 1, z: 0, who: 24},
{c: 1, x: -2, y: 0, z: 1, who: 53},
{x: -1, y: 0, z: 0, who: 12},
{c: 1, x: 0, y: 1, z: 1, who: 48},
{x: -2, y: -2, z: 1, who: 9},
{x: 2, y: 2, z: 0, who: 39}];
a[1]=
{2: {'RAF': {a:0.62,c:0},
'MTK1': {a:0.66,c:0},
'PLCG': {a:0.42,c:0},
'MEK1_2': {a:0.46,c:0},
'PPP2CA': {a:0.58,c:0},
'BCL2': {a:0.52,c:0},
'FOXO3': {a:0.58,c:0},
'p38': {a:0.56,c:0},
'RSK': {a:0.6,c:0},
'FOS': {a:0.36,c:0},
'ERK': {a:0.54,c:0},
'MAX': {a:0.44,c:0},
'SMAD': {a:0.56,c:0},
'ATF2': {a:0.52,c:0},
'GADD45': {a:0.58,c:0},
'PTEN': {a:0.52,c:0},
'MDM2': {a:0.54,c:0},
'p53': {a:0.18,c:0},
'p70': {a:0.54,c:0},
'FRS2': {a:0.56,c:0},
'PDK1': {a:0.48,c:0},
'AKT': {a:0.52,c:0},
'MSK': {a:0.5,c:0},
'GRB2': {a:0.66,c:0},
'PKC': {a:0.46,c:0},
'HIF1': {a:0.48,c:0.04},
'JUN': {a:0.6,c:0},
'p14': {a:0.5,c:0},
'MYC': {a:0.48,c:0},
'MAP3K1_3': {a:0.54,c:0},
'ATM': {a:0.34,c:0.16},
'EGFR': {a:0.68,c:0.12},
'ELK1': {a:0.5,c:0},
'TAOK': {a:0.4,c:0},
'JNK': {a:0.6,c:0},
'SPRY': {a:0.48,c:0},
'TGFBR': {a:0.42,c:0.16},
'GAB1': {a:0.44,c:0},
'RAS': {a:0.54,c:0},
'CREB': {a:0.58,c:0},
'DUSP1': {a:0.5,c:0},
'TAK1': {a:0.58,c:0},
'PI3K': {a:0.4,c:0},
'AP1': {a:0.44,c:0},
'SOS': {a:0.46,c:0},
'p21': {a:0.42,c:0},
'FGFR3': {a:0.38,c:0.16}}
};
a[1]=
{1: {'PLCG': {a:0.42,c:0},
'SPRY': {a:0.48,c:0},
'ELK1': {a:0.5,c:0},
'SOS': {a:0.46,c:0},
'FOXO3': {a:0.58,c:0},
'TGFBR': {a:0.42,c:0.16},
'TAOK': {a:0.4,c:0},
'MDM2': {a:0.54,c:0},
'p14': {a:0.5,c:0},
'MSK': {a:0.5,c:0},
'FRS2': {a:0.56,c:0},
'p70': {a:0.54,c:0},
'JUN': {a:0.6,c:0},
'p38': {a:0.56,c:0},
'JNK': {a:0.6,c:0},
'BCL2': {a:0.52,c:0},
'MEK1_2': {a:0.46,c:0},
'RAF': {a:0.62,c:0},
'DUSP1': {a:0.5,c:0},
'GADD45': {a:0.58,c:0},
'HIF1': {a:0.48,c:0.04},
'EGFR': {a:0.7,c:0.12},
'RSK': {a:0.6,c:0},
'AP1': {a:0.44,c:0},
'GRB2': {a:0.66,c:0},
'PPP2CA': {a:0.58,c:0},
'p21': {a:0.42,c:0},
'RAS': {a:0.54,c:0},
'MAX': {a:0.44,c:0},
'ATM': {a:0.34,c:0.16},
'AKT': {a:0.52,c:0},
'ERK': {a:0.54,c:0},
'PDK1': {a:0.48,c:0},
'SMAD': {a:0.56,c:0},
'CREB': {a:0.58,c:0},
'MAP3K1_3': {a:0.54,c:0},
'PTEN': {a:0.52,c:0},
'FGFR3': {a:0.38,c:0.16},
'GAB1': {a:0.44,c:0},
'PKC': {a:0.46,c:0},
'MTK1': {a:0.66,c:0},
'ATF2': {a:0.52,c:0},
'PI3K': {a:0.4,c:0},
'p53': {a:0.18,c:0},
'FOS': {a:0.36,c:0},
'MYC': {a:0.48,c:0},
'TAK1': {a:0.58,c:0}}
};
a[2]=
{2: {'PKC': {a:0.46,c:0},
'FOS': {a:0.36,c:0},
'PDK1': {a:0.48,c:0},
'p21': {a:0.42,c:0},
'p53': {a:0.22,c:0.08},
'FGFR3': {a:0.38,c:0.16},
'MSK': {a:0.5,c:0},
'AP1': {a:0.44,c:0},
'HIF1': {a:0.48,c:0.04},
'MTK1': {a:0.66,c:0},
'SOS': {a:0.46,c:0},
'GADD45': {a:0.58,c:0},
'TGFBR': {a:0.4,c:0.18},
'SPRY': {a:0.48,c:0},
'FRS2': {a:0.48,c:0.08},
'CREB': {a:0.58,c:0},
'RAF': {a:0.62,c:0},
'p14': {a:0.5,c:0},
'p38': {a:0.56,c:0},
'FOXO3': {a:0.58,c:0},
'ATM': {a:0.34,c:0.16},
'TAK1': {a:0.48,c:0.1},
'PPP2CA': {a:0.58,c:0},
'PLCG': {a:0.48,c:0.1},
'RSK': {a:0.6,c:0},
'BCL2': {a:0.52,c:0},
'SMAD': {a:0.54,c:0.02},
'AKT': {a:0.52,c:0},
'p70': {a:0.54,c:0},
'MAX': {a:0.44,c:0},
'MEK1_2': {a:0.46,c:0},
'ATF2': {a:0.52,c:0},
'JUN': {a:0.6,c:0},
'ELK1': {a:0.5,c:0},
'MYC': {a:0.48,c:0},
'PI3K': {a:0.4,c:0},
'GRB2': {a:0.72,c:0.06},
'EGFR': {a:0.76,c:0.12},
'TAOK': {a:0.36,c:0.04},
'PTEN': {a:0.52,c:0},
'MAP3K1_3': {a:0.54,c:0},
'GAB1': {a:0.44,c:0},
'JNK': {a:0.6,c:0},
'DUSP1': {a:0.5,c:0},
'MDM2': {a:0.54,c:0},
'RAS': {a:0.54,c:0},
'ERK': {a:0.54,c:0}}
};
a[3]=
{1: {'SPRY': {a:0.48,c:0},
'SOS': {a:0.46,c:0},
'PDK1': {a:0.48,c:0},
'AP1': {a:0.44,c:0},
'PPP2CA': {a:0.58,c:0},
'p38': {a:0.56,c:0},
'GAB1': {a:0.44,c:0},
'CREB': {a:0.58,c:0},
'SMAD': {a:0.54,c:0.02},
'MEK1_2': {a:0.46,c:0},
'FGFR3': {a:0.38,c:0.16},
'GRB2': {a:0.72,c:0.06},
'GADD45': {a:0.58,c:0},
'MYC': {a:0.48,c:0},
'EGFR': {a:0.76,c:0.12},
'MTK1': {a:0.66,c:0},
'MDM2': {a:0.54,c:0},
'RAF': {a:0.62,c:0},
'ATM': {a:0.34,c:0.16},
'RSK': {a:0.6,c:0},
'AKT': {a:0.52,c:0},
'BCL2': {a:0.52,c:0},
'FOS': {a:0.36,c:0},
'ELK1': {a:0.5,c:0},
'DUSP1': {a:0.5,c:0},
'p70': {a:0.54,c:0},
'MSK': {a:0.5,c:0},
'TAOK': {a:0.36,c:0.04},
'p21': {a:0.42,c:0},
'MAX': {a:0.44,c:0},
'JNK': {a:0.6,c:0},
'TAK1': {a:0.48,c:0.1},
'PLCG': {a:0.48,c:0.1},
'RAS': {a:0.54,c:0},
'ERK': {a:0.54,c:0},
'ATF2': {a:0.52,c:0},
'PKC': {a:0.46,c:0},
'HIF1': {a:0.48,c:0.04},
'p53': {a:0.22,c:0.08},
'FOXO3': {a:0.58,c:0},
'JUN': {a:0.6,c:0},
'TGFBR': {a:0.4,c:0.18},
'MAP3K1_3': {a:0.54,c:0},
'PI3K': {a:0.4,c:0},
'FRS2': {a:0.48,c:0.08},
'p14': {a:0.5,c:0},
'PTEN': {a:0.52,c:0}}
};
a[4]=
{1: {'PDK1': {a:0.48,c:0},
'BCL2': {a:0.52,c:0},
'SMAD': {a:0.54,c:0.02},
'GAB1': {a:0.44,c:0},
'HIF1': {a:0.48,c:0.04},
'JNK': {a:0.64,c:0.08},
'AP1': {a:0.44,c:0},
'p38': {a:0.58,c:0.02},
'ATF2': {a:0.52,c:0},
'p21': {a:0.4,c:0.02},
'SPRY': {a:0.48,c:0},
'RAF': {a:0.62,c:0},
'PTEN': {a:0.5,c:0.02},
'GRB2': {a:0.8,c:0.14},
'SOS': {a:0.46,c:0.04},
'CREB': {a:0.58,c:0},
'RSK': {a:0.6,c:0},
'ELK1': {a:0.5,c:0},
'FGFR3': {a:0.32,c:0.22},
'FRS2': {a:0.46,c:0.1},
'TGFBR': {a:0.4,c:0.18},
'p14': {a:0.5,c:0},
'TAOK': {a:0.36,c:0.04},
'MTK1': {a:0.66,c:0},
'MDM2': {a:0.54,c:0},
'DUSP1': {a:0.5,c:0},
'ATM': {a:0.34,c:0.16},
'EGFR': {a:0.7,c:0.18},
'MAX': {a:0.44,c:0},
'FOXO3': {a:0.58,c:0},
'PLCG': {a:0.48,c:0.1},
'RAS': {a:0.58,c:0.04},
'PPP2CA': {a:0.58,c:0},
'JUN': {a:0.6,c:0},
'p70': {a:0.54,c:0},
'ERK': {a:0.54,c:0},
'p53': {a:0.2,c:0.08},
'AKT': {a:0.52,c:0},
'PKC': {a:0.48,c:0.06},
'MSK': {a:0.5,c:0},
'GADD45': {a:0.58,c:0},
'PI3K': {a:0.4,c:0},
'MYC': {a:0.48,c:0},
'MEK1_2': {a:0.46,c:0},
'FOS': {a:0.36,c:0},
'MAP3K1_3': {a:0.54,c:0},
'TAK1': {a:0.48,c:0.1}}
};
a[5]=
{2: {'GADD45': {a:0.58,c:0},
'JUN': {a:0.6,c:0},
'MSK': {a:0.5,c:0},
'JNK': {a:0.64,c:0.08},
'FGFR3': {a:0.32,c:0.22},
'FOXO3': {a:0.58,c:0},
'FRS2': {a:0.46,c:0.1},
'PI3K': {a:0.4,c:0},
'AP1': {a:0.44,c:0},
'BCL2': {a:0.52,c:0},
'HIF1': {a:0.48,c:0.04},
'MYC': {a:0.48,c:0},
'p53': {a:0.2,c:0.08},
'PDK1': {a:0.48,c:0},
'PKC': {a:0.48,c:0.06},
'CREB': {a:0.58,c:0},
'SOS': {a:0.46,c:0.04},
'TGFBR': {a:0.4,c:0.18},
'GRB2': {a:0.8,c:0.14},
'p70': {a:0.54,c:0},
'p14': {a:0.5,c:0},
'RAS': {a:0.58,c:0.04},
'SMAD': {a:0.54,c:0.02},
'MAX': {a:0.44,c:0},
'RSK': {a:0.6,c:0},
'PPP2CA': {a:0.58,c:0},
'TAOK': {a:0.36,c:0.04},
'DUSP1': {a:0.5,c:0},
'MEK1_2': {a:0.46,c:0},
'MTK1': {a:0.66,c:0},
'RAF': {a:0.62,c:0},
'p38': {a:0.58,c:0.02},
'FOS': {a:0.36,c:0},
'MAP3K1_3': {a:0.54,c:0},
'ELK1': {a:0.5,c:0},
'p21': {a:0.4,c:0.02},
'GAB1': {a:0.44,c:0},
'MDM2': {a:0.54,c:0},
'PTEN': {a:0.5,c:0.02},
'EGFR': {a:0.7,c:0.18},
'SPRY': {a:0.48,c:0},
'AKT': {a:0.52,c:0},
'ATM': {a:0.34,c:0.16},
'TAK1': {a:0.48,c:0.1},
'ATF2': {a:0.52,c:0},
'PLCG': {a:0.48,c:0.1},
'ERK': {a:0.54,c:0}}
};
a[6]=
{1: {'JUN': {a:0.62,c:0.02},
'HIF1': {a:0.48,c:0.04},
'MDM2': {a:0.54,c:0},
'MYC': {a:0.48,c:0},
'AKT': {a:0.52,c:0.04},
'FRS2': {a:0.44,c:0.12},
'GAB1': {a:0.48,c:0.04},
'RAS': {a:0.6,c:0.06},
'p14': {a:0.5,c:0},
'GADD45': {a:0.58,c:0},
'JNK': {a:0.64,c:0.08},
'PTEN': {a:0.5,c:0.02},
'ELK1': {a:0.5,c:0},
'BCL2': {a:0.52,c:0},
'AP1': {a:0.44,c:0},
'TAK1': {a:0.48,c:0.1},
'SMAD': {a:0.54,c:0.02},
'PKC': {a:0.48,c:0.06},
'PDK1': {a:0.48,c:0},
'p21': {a:0.4,c:0.02},
'PPP2CA': {a:0.6,c:0.02},
'FOXO3': {a:0.58,c:0},
'p70': {a:0.54,c:0},
'GRB2': {a:0.8,c:0.18},
'p53': {a:0.22,c:0.1},
'FGFR3': {a:0.32,c:0.22},
'EGFR': {a:0.68,c:0.22},
'SPRY': {a:0.48,c:0},
'ATM': {a:0.34,c:0.16},
'PLCG': {a:0.48,c:0.1},
'p38': {a:0.58,c:0.02},
'TGFBR': {a:0.4,c:0.18},
'CREB': {a:0.58,c:0},
'MAP3K1_3': {a:0.56,c:0.02},
'FOS': {a:0.36,c:0},
'MTK1': {a:0.66,c:0},
'ATF2': {a:0.54,c:0.02},
'PI3K': {a:0.34,c:0.06},
'ERK': {a:0.54,c:0},
'TAOK': {a:0.36,c:0.04},
'MSK': {a:0.5,c:0},
'MAX': {a:0.46,c:0.02},
'MEK1_2': {a:0.46,c:0},
'DUSP1': {a:0.5,c:0},
'SOS': {a:0.46,c:0.04},
'RSK': {a:0.6,c:0},
'RAF': {a:0.64,c:0.02}}
};
a[7]=
{2: {'FGFR3': {a:0.32,c:0.22},
'ATM': {a:0.34,c:0.16},
'MAX': {a:0.46,c:0.02},
'ELK1': {a:0.5,c:0},
'MEK1_2': {a:0.46,c:0},
'MDM2': {a:0.54,c:0},
'BCL2': {a:0.52,c:0},
'p14': {a:0.5,c:0},
'PKC': {a:0.48,c:0.06},
'SMAD': {a:0.54,c:0.02},
'GAB1': {a:0.48,c:0.04},
'GADD45': {a:0.58,c:0},
'PI3K': {a:0.34,c:0.06},
'MAP3K1_3': {a:0.56,c:0.02},
'TAOK': {a:0.36,c:0.04},
'p21': {a:0.4,c:0.02},
'GRB2': {a:0.8,c:0.18},
'FOS': {a:0.36,c:0},
'JUN': {a:0.62,c:0.02},
'RSK': {a:0.6,c:0},
'FRS2': {a:0.44,c:0.12},
'AKT': {a:0.52,c:0.04},
'PTEN': {a:0.5,c:0.02},
'EGFR': {a:0.68,c:0.22},
'p70': {a:0.54,c:0},
'TGFBR': {a:0.4,c:0.18},
'DUSP1': {a:0.5,c:0},
'PPP2CA': {a:0.6,c:0.02},
'RAS': {a:0.6,c:0.06},
'MTK1': {a:0.66,c:0},
'MSK': {a:0.5,c:0},
'SOS': {a:0.46,c:0.04},
'JNK': {a:0.64,c:0.08},
'FOXO3': {a:0.58,c:0},
'SPRY': {a:0.48,c:0},
'PDK1': {a:0.48,c:0},
'ATF2': {a:0.54,c:0.02},
'p53': {a:0.22,c:0.1},
'MYC': {a:0.48,c:0},
'HIF1': {a:0.48,c:0.04},
'TAK1': {a:0.48,c:0.1},
'AP1': {a:0.44,c:0},
'CREB': {a:0.58,c:0},
'PLCG': {a:0.48,c:0.1},
'p38': {a:0.58,c:0.02},
'RAF': {a:0.64,c:0.02},
'ERK': {a:0.54,c:0}}
};
a[8]=
{1: {'PDK1': {a:0.46,c:0.02},
'TAK1': {a:0.48,c:0.1},
'ATM': {a:0.34,c:0.16},
'HIF1': {a:0.48,c:0.04},
'ELK1': {a:0.5,c:0},
'ERK': {a:0.54,c:0},
'MAX': {a:0.46,c:0.02},
'FOS': {a:0.3,c:0.06},
'PTEN': {a:0.5,c:0.02},
'RAF': {a:0.64,c:0.02},
'DUSP1': {a:0.5,c:0},
'JNK': {a:0.64,c:0.08},
'SMAD': {a:0.54,c:0.02},
'SOS': {a:0.46,c:0.08},
'BCL2': {a:0.54,c:0.02},
'MYC': {a:0.48,c:0},
'MAP3K1_3': {a:0.56,c:0.02},
'PLCG': {a:0.48,c:0.14},
'p14': {a:0.5,c:0},
'MTK1': {a:0.66,c:0},
'TAOK': {a:0.36,c:0.04},
'AP1': {a:0.42,c:0.06},
'AKT': {a:0.52,c:0.04},
'PKC': {a:0.48,c:0.06},
'p53': {a:0.22,c:0.1},
'GAB1': {a:0.5,c:0.06},
'SPRY': {a:0.48,c:0},
'FOXO3': {a:0.54,c:0.04},
'MEK1_2': {a:0.46,c:0.04},
'PI3K': {a:0.42,c:0.14},
'FRS2': {a:0.42,c:0.14},
'PPP2CA': {a:0.6,c:0.02},
'EGFR': {a:0.68,c:0.22},
'CREB': {a:0.58,c:0},
'TGFBR': {a:0.38,c:0.2},
'RSK': {a:0.6,c:0},
'p70': {a:0.54,c:0},
'FGFR3': {a:0.32,c:0.22},
'ATF2': {a:0.54,c:0.02},
'GADD45': {a:0.58,c:0},
'p38': {a:0.62,c:0.06},
'MSK': {a:0.5,c:0},
'GRB2': {a:0.78,c:0.2},
'MDM2': {a:0.54,c:0},
'RAS': {a:0.6,c:0.06},
'p21': {a:0.42,c:0.04},
'JUN': {a:0.62,c:0.02}}
};
a[9]=
{2: {'AKT': {a:0.52,c:0.04},
'PKC': {a:0.48,c:0.06},
'MSK': {a:0.5,c:0},
'FOXO3': {a:0.54,c:0.04},
'AP1': {a:0.42,c:0.06},
'RSK': {a:0.6,c:0},
'JNK': {a:0.64,c:0.08},
'HIF1': {a:0.48,c:0.04},
'ATM': {a:0.34,c:0.16},
'ERK': {a:0.54,c:0},
'MTK1': {a:0.66,c:0},
'PTEN': {a:0.5,c:0.02},
'EGFR': {a:0.68,c:0.22},
'PDK1': {a:0.46,c:0.02},
'GADD45': {a:0.58,c:0},
'SOS': {a:0.46,c:0.08},
'GAB1': {a:0.5,c:0.06},
'p14': {a:0.5,c:0},
'ATF2': {a:0.54,c:0.02},
'MDM2': {a:0.54,c:0},
'p21': {a:0.42,c:0.04},
'MEK1_2': {a:0.46,c:0.04},
'PPP2CA': {a:0.6,c:0.02},
'MAX': {a:0.46,c:0.02},
'TGFBR': {a:0.38,c:0.2},
'PI3K': {a:0.42,c:0.14},
'GRB2': {a:0.78,c:0.2},
'RAS': {a:0.6,c:0.06},
'p70': {a:0.54,c:0},
'CREB': {a:0.58,c:0},
'ELK1': {a:0.5,c:0},
'FOS': {a:0.3,c:0.06},
'DUSP1': {a:0.5,c:0},
'SMAD': {a:0.54,c:0.02},
'PLCG': {a:0.48,c:0.14},
'TAK1': {a:0.48,c:0.1},
'p53': {a:0.22,c:0.1},
'RAF': {a:0.64,c:0.02},
'BCL2': {a:0.54,c:0.02},
'MYC': {a:0.48,c:0},
'MAP3K1_3': {a:0.56,c:0.02},
'FRS2': {a:0.42,c:0.14},
'p38': {a:0.62,c:0.06},
'SPRY': {a:0.48,c:0},
'FGFR3': {a:0.32,c:0.22},
'JUN': {a:0.62,c:0.02},
'TAOK': {a:0.36,c:0.04}}
};
a[10]=
{2: {'CREB': {a:0.58,c:0},
'ERK': {a:0.5,c:0.08},
'MYC': {a:0.48,c:0},
'FGFR3': {a:0.3,c:0.24},
'TAK1': {a:0.48,c:0.1},
'MTK1': {a:0.66,c:0},
'GADD45': {a:0.58,c:0},
'ELK1': {a:0.5,c:0},
'PTEN': {a:0.5,c:0.02},
'RSK': {a:0.6,c:0},
'p53': {a:0.22,c:0.1},
'RAF': {a:0.64,c:0.02},
'RAS': {a:0.6,c:0.06},
'PDK1': {a:0.48,c:0.04},
'MAX': {a:0.46,c:0.02},
'FOXO3': {a:0.54,c:0.04},
'ATF2': {a:0.56,c:0.04},
'TGFBR': {a:0.36,c:0.22},
'p70': {a:0.52,c:0.02},
'JNK': {a:0.64,c:0.08},
'AP1': {a:0.4,c:0.08},
'JUN': {a:0.62,c:0.02},
'p21': {a:0.42,c:0.04},
'TAOK': {a:0.36,c:0.04},
'MAP3K1_3': {a:0.56,c:0.02},
'MSK': {a:0.5,c:0},
'PPP2CA': {a:0.62,c:0.04},
'DUSP1': {a:0.5,c:0},
'FRS2': {a:0.42,c:0.14},
'MDM2': {a:0.54,c:0},
'SOS': {a:0.48,c:0.1},
'FOS': {a:0.3,c:0.06},
'PKC': {a:0.5,c:0.08},
'EGFR': {a:0.66,c:0.24},
'HIF1': {a:0.48,c:0.04},
'GRB2': {a:0.78,c:0.2},
'AKT': {a:0.52,c:0.04},
'BCL2': {a:0.54,c:0.02},
'SMAD': {a:0.54,c:0.02},
'PLCG': {a:0.5,c:0.16},
'ATM': {a:0.34,c:0.16},
'GAB1': {a:0.52,c:0.08},
'SPRY': {a:0.48,c:0},
'PI3K': {a:0.48,c:0.2},
'p38': {a:0.62,c:0.06},
'MEK1_2': {a:0.4,c:0.1},
'p14': {a:0.48,c:0.02}}
};
replicates[replicate_counter++] = {l: l, s: s, a: a};
l = [];
s = [];
a = [];

minimum_x = Math.min(minimum_x, -2);
maximum_x = Math.max(maximum_x, 2);
minimum_y = Math.min(minimum_y, -2);
maximum_y = Math.max(maximum_y, 2);

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>
<!--
			This GraphML document was generated from GXL by
			a GXL-to-GraphML conversion style sheet.
		-->
<graphml>
   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>
   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>
   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>
   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>
   <key id="MY-VALUE"
        for="node"
        attr.name="MY-VALUE"
        attr.type="string"/>
   <key id="MY-SHAPE"
        for="node"
        attr.name="MY-SHAPE"
        attr.type="string"/>
   <key id="PROBABILITY-OF-LINK"
        for="edge"
        attr.name="PROBABILITY-OF-LINK"
        attr.type="string"/>
   <key id="SIGN-OF-LINK"
        for="edge"
        attr.name="SIGN-OF-LINK"
        attr.type="string"/>
   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>
   <graph id="MAPK_large_HIF1" edgedefault="directed">
      <node id="EGFR_stimulus">
         <data key="MY-ID">EGFR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce EGFR activation
</node>
      <node id="FGFR3_stimulus">
         <data key="MY-ID">FGFR3_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce FGFR3 activation
</node>
      <node id="TGFBR_stimulus">
         <data key="MY-ID">TGFBR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce TGFBR activation

</node>
      <node id="DNA_damage">
         <data key="MY-ID">DNA_damage</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: DNA damaging agents/therapies action

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</node>
      <node id="Apoptosis">
         <data key="MY-ID">Apoptosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: apoptosis enablement

Modelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="Growth_Arrest">
         <data key="MY-ID">Growth_Arrest</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p21</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: growth arrest enablement

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="Proliferation">
         <data key="MY-ID">Proliferation</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and MYC and not p21</data>
         <data key="MY-SHAPE">rect</data>Proliferation

VARIABLE: proliferation enablement

Proliferation is an output variable of the model.

Modelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832

</node>
      <node id="ERK">
         <data key="MY-ID">ERK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MEK1_2</data>
         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1
MAPK3 = mitogen-activated protein kinase 3

VARIABLE: ERK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
RSKs interact with ERK and dissociate upon activation.
PMID:15187187

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p38">
         <data key="MY-ID">p38</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11
MAPK12 = mitogen-activated protein kinase 12
MAPK13 = mitogen-activated protein kinase 13
MAPK14 = mitogen-activated protein kinase 14

VARIABLE: p38 (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
MAX is phosphorylated (and activated) by p38, through complex formation.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187

ELK1 is a nuclear p38 target.
PMID:20506250

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="JNK">
         <data key="MY-ID">JNK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8
MAPK9 = mitogen-activated protein kinase 9
MAPK10 = mitogen-activated protein kinase 10

VARIABLE: JNK (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
JNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
ELK1 is a nuclear JNK target.
MEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="p53">
         <data key="MY-ID">p53</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>
         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53

VARIABLE: p53 phosphorylation level

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942

Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.
PMID:17158541

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.
PMID:12217521</node>
      <node id="p21">
         <data key="MY-ID">p21</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not AKT and p53</data>
         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)

VARIABLE: p21 activation level

p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

p21 is transcriptionally induced by the p53 protein.
p21 is a CDK inhibitor.
PMID:17158541

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="TGFBR">
         <data key="MY-ID">TGFBR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR_stimulus</data>
         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1
TGFBR2 = transforming growth factor, beta receptor II (70/80kDa)
TGFBR3 = transforming growth factor, beta receptor III

VARIABLE: TGFBR activation level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta.
PMID:21614932

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</node>
      <node id="EGFR">
         <data key="MY-ID">EGFR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>
         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor

VARIABLE: EGFR activation level

The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</node>
      <node id="FGFR3">
         <data key="MY-ID">FGFR3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>
         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3

VARIABLE: FGFR3 activation level

Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="ATM">
         <data key="MY-ID">ATM</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">DNA_damage</data>
         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated

VARIABLE: ATM phosphorylation level

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942
</node>
      <node id="TAOK">
         <data key="MY-ID">TAOK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ATM</data>
         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1
TAOK2 = TAO kinase 2
TAOK3 = TAO kinase 3

VARIABLE: TAOK (any isoform) phosphorylation level

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</node>
      <node id="MAX">
         <data key="MY-ID">MAX</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X

VARIABLE: MAX phosphorylation level

MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</node>
      <node id="GRB2">
         <data key="MY-ID">GRB2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2

VARIABLE: GRB2 recruitment by activated RTKs

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</node>
      <node id="FRS2">
         <data key="MY-ID">FRS2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>
         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2

VARIABLE: FRS2 phosphorylation level

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</node>
      <node id="PI3K">
         <data key="MY-ID">PI3K</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>
         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide

VARIABLE: PI3K activation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Recruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497</node>
      <node id="AP1">
         <data key="MY-ID">AP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JUN and (FOS or ATF2)</data>
         <data key="MY-SHAPE">ellipse</data>AP-1

VARIABLE: AP-1 formation

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345

AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="PPP2CA">
         <data key="MY-ID">PPP2CA</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme

VARIABLE: PPP2CA activation level

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</node>
      <node id="MEK1_2">
         <data key="MY-ID">MEK1_2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1
MAP2K2 = mitogen-activated protein kinase kinase 2

VARIABLE: MEK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="DUSP1">
         <data key="MY-ID">DUSP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB</data>
         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1

VARIABLE: DUSP1 gene expression level

DUSP1 is a CREB target gene.
PMID:19815709

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</node>
      <node id="MYC">
         <data key="MY-ID">MYC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>
         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)

VARIABLE: MYC activation level (implies gene expression)

Modelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.
MYC inhibits p16, whereas it activates p14.
PMID:17158541

MAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345

Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</node>
      <node id="AKT">
         <data key="MY-ID">AKT</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and not PTEN</data>
         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1
AKT2 = v-akt murine thymoma viral oncogene homolog 2
AKT3 = v-akt murine thymoma viral oncogene homolog 3

VARIABLE: AKT (any isoform) phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186

p21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="PLCG">
         <data key="MY-ID">PLCG</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FGFR3</data>
         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1
PLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)

VARIABLE: PLCG (any isoform) phosphorylation level

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="PKC">
         <data key="MY-ID">PKC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PLCG</data>
         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha
PRKCB = protein kinase C, beta
PRKCG = protein kinase C, gamma

VARIABLE: PKC (any isoform) enzymatic activity

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="GADD45">
         <data key="MY-ID">GADD45</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SMAD or p53</data>
         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha
GADD45B = growth arrest and DNA-damage-inducible, beta
GADD45G = growth arrest and DNA-damage-inducible, gamma

VARIABLE: GADD45 gene (any isoform) expression level

The GADD45 gene is inducible by various environmental stresses.
Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</node>
      <node id="ELK1">
         <data key="MY-ID">ELK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family

VARIABLE: ELK1 phosphorylation level

The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

ELK1 is a nuclear p38 target.
PMID:20506250

ELK1 is a nuclear JNK target.
ELK1 induces c-fos expression.
PMID:11274345</node>
      <node id="FOS">
         <data key="MY-ID">FOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>
         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog

VARIABLE: FOS phosphorylation level

Further observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
ELK1 induces c-fos expression.
PMID:11274345

FOS is a CREB target gene.
PMID:19815709

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</node>
      <node id="ATF2">
         <data key="MY-ID">ATF2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2

VARIABLE: ATF2 phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</node>
      <node id="JUN">
         <data key="MY-ID">JUN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK</data>
         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene

VARIABLE: JUN phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</node>
      <node id="MSK">
         <data key="MY-ID">MSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or p38</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4
RPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5

VARIABLE: MSK (any isoform) phosphorylation level

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</node>
      <node id="CREB">
         <data key="MY-ID">CREB</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MSK</data>
         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1

VARIABLE: CREB phosphorylation level

MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</node>
      <node id="RSK">
         <data key="MY-ID">RSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1
RPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2
RPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3
RPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6

VARIABLE: RSK (any isoform) phosphorylation level

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</node>
      <node id="SMAD">
         <data key="MY-ID">SMAD</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2
SMAD3 = SMAD family member 3
SMAD4 = SMAD family member 4

VARIABLE: SMAD gene (any isoform) expression level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</node>
      <node id="MTK1">
         <data key="MY-ID">MTK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GADD45</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4

VARIABLE: MTK1 phosphorylation level

The delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</node>
      <node id="SPRY">
         <data key="MY-ID">SPRY</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)

VARIABLE: SPRY2 phosphorylation level

Spry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.
PMID:15173823</node>
      <node id="RAF">
         <data key="MY-ID">RAF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>
         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1
BRAF = v-raf murine sarcoma viral oncogene homolog B1
ARAF = v-raf murine sarcoma 3611 viral oncogene homolog

VARIABLE: RAF (any isoform) phosphorylation

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

PKCalpha directly phosphorylates and activates Raf-1.
PMID:8321321

Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</node>
      <node id="GAB1">
         <data key="MY-ID">GAB1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 or PI3K</data>
         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1

VARIABLE: GAB1 phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
PMID:15567848

Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</node>
      <node id="PDK1">
         <data key="MY-ID">PDK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PI3K</data>
         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1

VARIABLE: PDK1 activation level

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</node>
      <node id="p70">
         <data key="MY-ID">p70</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1
RPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2

VARIABLE: p70 (any isoform) phosphorylation level

PDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.
PMID:12040186

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p14">
         <data key="MY-ID">p14</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MYC</data>
         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)

VARIABLE: p14 activation level

MYC inhibits p16, whereas it activates p14.
The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</node>
      <node id="FOXO3">
         <data key="MY-ID">FOXO3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK and not AKT</data>
         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3

VARIABLE: FOXO3 phosphorylation level

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</node>
      <node id="RAS">
         <data key="MY-ID">RAS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SOS or PLCG</data>
         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog
KRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog
NRAS = neuroblastoma RAS viral (v-ras) oncogene homolog

VARIABLE: RAS (any isoform) phosphorylation level

Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Most of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="SOS">
         <data key="MY-ID">SOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 and not RSK</data>
         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)
SOS2 = son of sevenless homolog 2 (Drosophila)

VARIABLE: SOS (any isoform) recruitment by GRB2

Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373
</node>
      <node id="MDM2">
         <data key="MY-ID">MDM2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(p53 or AKT) and not p14</data>
         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)

VARIABLE: MDM2 gene expression level

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.
PMID:17158541

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850
</node>
      <node id="BCL2">
         <data key="MY-ID">BCL2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB and AKT</data>
         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2

VARIABLE: BCL2 activation level (implies gene expression)

Modelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.

DUSP1, FOS, BCL2 are CREB target genes.
PMID:19815709

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="TAK1">
         <data key="MY-ID">TAK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7

VARIABLE: TAK1 phosphorylation level

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="MAP3K1_3">
         <data key="MY-ID">MAP3K1_3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">RAS</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1
MAP3K2 = mitogen-activated protein kinase kinase kinase 2
MAP3K3 = mitogen-activated protein kinase kinase kinase 3

VARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</node>
      <node id="PTEN">
         <data key="MY-ID">PTEN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p53</data>
         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog

VARIABLE: PTEN protein expression level

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="HIF1">
         <data key="MY-ID">HIF1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="EGF">
         <data key="MY-ID">EGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="VEGF">
         <data key="MY-ID">VEGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Oxygen_supply">
         <data key="MY-ID">Oxygen_supply</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Necrosis">
         <data key="MY-ID">Necrosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not Oxygen_supply</data>
         <data key="MY-SHAPE">rect</data>
      </node>
      <edge source="AP1" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</edge>
      <edge source="EGFR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910</edge>
      <edge source="EGFR" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</edge>
      <edge source="Oxygen_supply" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="Oxygen_supply" target="Necrosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MSK" target="CREB" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187</edge>
      <edge source="MSK" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</edge>
      <edge source="MEK1_2" target="ERK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="HIF1" target="EGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="HIF1" target="VEGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="DUSP1" target="p38" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="DUSP1" target="JNK" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="PI3K" target="PDK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124</edge>
      <edge source="PI3K" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="FGFR3" target="FRS2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="FGFR3" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221</edge>
      <edge source="DNA_damage" target="ATM" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</edge>
      <edge source="FRS2" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="MAX" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345</edge>
      <edge source="ATF2" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="ERK" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="ERK" target="RSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952</edge>
      <edge source="ERK" target="SPRY" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823</edge>
      <edge source="ERK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="ERK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ERK" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650
</edge>
      <edge source="ERK" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</edge>
      <edge source="ERK" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</edge>
      <edge source="ERK" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="CREB" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="DUSP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;



DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="JUN" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="FOXO3" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166</edge>
      <edge source="PLCG" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</edge>
      <edge source="PLCG" target="PKC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848</edge>
      <edge source="SOS" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929</edge>
      <edge source="SOS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="p70" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="p70" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MDM2" target="p53" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="TGFBR" target="SMAD" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="TGFBR" target="TAK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TGFBR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</edge>
      <edge source="GAB1" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="MTK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="MTK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="GRB2" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848</edge>
      <edge source="GRB2" target="SOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910</edge>
      <edge source="GRB2" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="GRB2" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848</edge>
      <edge source="GRB2" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="BCL2" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="ATM" target="TAOK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897</edge>
      <edge source="ATM" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942</edge>
      <edge source="GADD45" target="MTK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="PTEN" target="AKT" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="p14" target="MDM2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</edge>
      <edge source="PPP2CA" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="PPP2CA" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="p38" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932</edge>
      <edge source="p38" target="MAX" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</edge>
      <edge source="p38" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.
PMID:20506250
</edge>
      <edge source="p38" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="p53" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</edge>
      <edge source="p53" target="p21" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.
PMID:17158541</edge>
      <edge source="p53" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="p53" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508</edge>
      <edge source="p53" target="PTEN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="SMAD" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="SPRY" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="SPRY" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="EGFR_stimulus" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ELK1" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 induces c-fos expression.
PMID:11274345</edge>
      <edge source="TAK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TAK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="AKT" target="FOXO3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</edge>
      <edge source="AKT" target="p21" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383</edge>
      <edge source="AKT" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="AKT" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186</edge>
      <edge source="AKT" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850</edge>
      <edge source="AKT" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359</edge>
      <edge source="AKT" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="RAF" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="FOS" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="TAOK" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="TAOK" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="JNK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.
PMID:11274345</edge>
      <edge source="JNK" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="JNK" target="JUN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</edge>
      <edge source="JNK" target="FOXO3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971</edge>
      <edge source="p21" target="Proliferation" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="p21" target="Growth_Arrest" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="PKC" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473</edge>
      <edge source="PKC" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321</edge>
      <edge source="PKC" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</edge>
      <edge source="RSK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="RSK" target="SOS" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</edge>
      <edge source="PDK1" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="PDK1" target="AKT" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</edge>
      <edge source="MYC" target="p14" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.
PMID:17158541</edge>
      <edge source="MYC" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</edge>
      <edge source="RAS" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929</edge>
      <edge source="RAS" target="MAP3K1_3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="RAS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.
Proliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.
Input signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>
</graphml>
';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
l[1]=
[{added: 30},
{added: 31},
{added: 36},
{added: 21},
{added: 53},
{added: 41},
{added: 11},
{added: 4},
{added: 24},
{added: 12},
{added: 25},
{added: 15},
{added: 51},
{added: 19},
{added: 29},
{added: 32},
{added: 23},
{added: 9},
{added: 33},
{added: 35},
{added: 45},
{added: 28},
{added: 27},
{added: 18},
{added: 6},
{added: 34},
{added: 47},
{added: 22},
{added: 48},
{added: 7},
{added: 16},
{added: 43},
{added: 5},
{added: 39},
{added: 10},
{added: 52},
{added: 17},
{added: 20},
{added: 46},
{added: 50},
{added: 42},
{added: 44},
{added: 14},
{added: 26},
{added: 8},
{added: 38},
{added: 40},
{added: 37},
{added: 49},
{added: 13},
{x: 1, y: 1, z: 1, who: 26},
{c: 1, x: -2, y: -2, z: 1, who: 19},
{c: 1, x: 1, y: 2, z: 0, who: 15},
{x: 1, y: -2, z: 1, who: 17},
{x: 0, y: -1, z: 1, who: 49},
{c: 1, x: 0, y: 2, z: 0, who: 42},
{c: 1, x: 0, y: 1, z: 0, who: 30},
{c: 1, x: -1, y: -2, z: 0, who: 40},
{c: 1, x: 2, y: 2, z: 1, who: 32},
{x: -1, y: 2, z: 1, who: 21},
{c: 1, x: 1, y: 0, z: 1, who: 27},
{c: 1, x: -1, y: 1, z: 0, who: 7},
{c: 1, x: 1, y: 0, z: 0, who: 45},
{x: -1, y: -2, z: 1, who: 20},
{x: 0, y: -1, z: 0, who: 39},
{x: 0, y: -2, z: 1, who: 48},
{x: -1, y: 0, z: 1, who: 31},
{x: 2, y: 0, z: 0, who: 46},
{c: 1, x: -2, y: 0, z: 0, who: 41},
{c: 1, x: 0, y: 0, z: 0, who: 34},
{x: -2, y: 1, z: 1, who: 4},
{c: 1, x: 2, y: -2, z: 1, who: 11},
{c: 1, x: 1, y: 2, z: 1, who: 33},
{c: 1, x: 1, y: -2, z: 0, who: 23},
{x: 2, y: 1, z: 0, who: 8},
{c: 1, x: 2, y: 2, z: 0, who: 29},
{c: 1, x: 2, y: -1, z: 1, who: 35},
{x: -1, y: 1, z: 1, who: 44},
{x: 1, y: -1, z: 0, who: 16},
{c: 1, x: 0, y: 0, z: 1, who: 53},
{x: -1, y: 2, z: 0, who: 6},
{c: 1, x: -2, y: -1, z: 1, who: 10},
{c: 1, x: 0, y: -2, z: 0, who: 22},
{x: 2, y: 1, z: 1, who: 13},
{x: -1, y: 0, z: 0, who: 37},
{x: -2, y: 0, z: 1, who: 14},
{c: 1, x: -2, y: 2, z: 0, who: 9},
{c: 1, x: 2, y: -2, z: 0, who: 12},
{x: -1, y: -1, z: 1, who: 50},
{x: -2, y: 2, z: 1, who: 52},
{x: 2, y: -1, z: 0, who: 36},
{x: -2, y: 1, z: 0, who: 24},
{c: 1, x: 2, y: 0, z: 1, who: 5},
{x: 1, y: -1, z: 1, who: 43},
{x: -2, y: -2, z: 0, who: 51},
{c: 1, x: -2, y: -1, z: 0, who: 38},
{c: 1, x: 1, y: 1, z: 0, who: 25},
{c: 1, x: 0, y: 1, z: 1, who: 47},
{x: 0, y: 2, z: 1, who: 28},
{x: -1, y: -1, z: 0, who: 18}];
a[1]=
{2: {'GRB2': {a:0.48,c:0},
'RAF': {a:0.42,c:0},
'TAK1': {a:0.52,c:0},
'PLCG': {a:0.42,c:0},
'JNK': {a:0.5,c:0},
'MDM2': {a:0.62,c:0},
'ATM': {a:0.48,c:0.08},
'CREB': {a:0.52,c:0},
'p70': {a:0.4,c:0},
'GADD45': {a:0.36,c:0},
'MAP3K1_3': {a:0.56,c:0},
'PPP2CA': {a:0.46,c:0},
'MEK1_2': {a:0.44,c:0},
'FRS2': {a:0.6,c:0},
'AP1': {a:0.52,c:0},
'RAS': {a:0.42,c:0},
'EGFR': {a:0.52,c:0.14},
'p53': {a:0.2,c:0},
'ELK1': {a:0.48,c:0},
'DUSP1': {a:0.5,c:0},
'MAX': {a:0.52,c:0},
'MTK1': {a:0.46,c:0},
'p14': {a:0.56,c:0},
'FGFR3': {a:0.4,c:0.1},
'FOS': {a:0.5,c:0},
'TGFBR': {a:0.36,c:0.1},
'MSK': {a:0.58,c:0},
'ERK': {a:0.5,c:0},
'SOS': {a:0.44,c:0},
'PTEN': {a:0.46,c:0},
'ATF2': {a:0.38,c:0},
'FOXO3': {a:0.56,c:0},
'AKT': {a:0.42,c:0},
'MYC': {a:0.5,c:0},
'SPRY': {a:0.46,c:0},
'p38': {a:0.52,c:0},
'PI3K': {a:0.56,c:0},
'GAB1': {a:0.48,c:0},
'SMAD': {a:0.44,c:0},
'TAOK': {a:0.58,c:0},
'BCL2': {a:0.42,c:0},
'JUN': {a:0.56,c:0},
'PKC': {a:0.44,c:0},
'PDK1': {a:0.4,c:0},
'RSK': {a:0.44,c:0},
'HIF1': {a:0.36,c:0.06},
'p21': {a:0.52,c:0}}
};
a[1]=
{2: {'p53': {a:0.2,c:0},
'RAS': {a:0.42,c:0},
'PDK1': {a:0.4,c:0},
'MDM2': {a:0.62,c:0},
'GAB1': {a:0.48,c:0},
'PLCG': {a:0.42,c:0},
'DUSP1': {a:0.5,c:0},
'RAF': {a:0.42,c:0},
'FGFR3': {a:0.4,c:0.1},
'GRB2': {a:0.48,c:0},
'TAK1': {a:0.52,c:0},
'SMAD': {a:0.44,c:0},
'JNK': {a:0.5,c:0},
'MEK1_2': {a:0.44,c:0},
'SPRY': {a:0.46,c:0},
'FOXO3': {a:0.56,c:0},
'HIF1': {a:0.36,c:0.06},
'FOS': {a:0.5,c:0},
'ATM': {a:0.48,c:0.08},
'TGFBR': {a:0.36,c:0.1},
'SOS': {a:0.44,c:0},
'GADD45': {a:0.36,c:0},
'ELK1': {a:0.48,c:0},
'AKT': {a:0.42,c:0},
'p38': {a:0.52,c:0},
'CREB': {a:0.52,c:0},
'PTEN': {a:0.46,c:0},
'ERK': {a:0.5,c:0},
'p21': {a:0.52,c:0},
'BCL2': {a:0.42,c:0},
'MYC': {a:0.5,c:0},
'PI3K': {a:0.56,c:0},
'RSK': {a:0.44,c:0},
'MAX': {a:0.52,c:0},
'ATF2': {a:0.38,c:0},
'PPP2CA': {a:0.46,c:0},
'EGFR': {a:0.54,c:0.14},
'p14': {a:0.56,c:0},
'PKC': {a:0.44,c:0},
'AP1': {a:0.52,c:0},
'p70': {a:0.4,c:0},
'MSK': {a:0.58,c:0},
'JUN': {a:0.56,c:0},
'FRS2': {a:0.6,c:0},
'MAP3K1_3': {a:0.56,c:0},
'TAOK': {a:0.58,c:0},
'MTK1': {a:0.46,c:0}}
};
a[2]=
{1: {'ATM': {a:0.48,c:0.08},
'p53': {a:0.2,c:0},
'RAF': {a:0.42,c:0},
'FOXO3': {a:0.56,c:0},
'RSK': {a:0.44,c:0},
'FOS': {a:0.5,c:0},
'TGFBR': {a:0.36,c:0.1},
'AKT': {a:0.42,c:0},
'JNK': {a:0.5,c:0},
'PTEN': {a:0.46,c:0},
'BCL2': {a:0.42,c:0},
'PPP2CA': {a:0.46,c:0},
'CREB': {a:0.52,c:0},
'AP1': {a:0.52,c:0},
'ERK': {a:0.5,c:0},
'PKC': {a:0.44,c:0},
'PI3K': {a:0.56,c:0},
'EGFR': {a:0.64,c:0.14},
'MAP3K1_3': {a:0.56,c:0},
'SPRY': {a:0.46,c:0},
'SOS': {a:0.44,c:0},
'MDM2': {a:0.62,c:0},
'JUN': {a:0.56,c:0},
'PDK1': {a:0.4,c:0},
'FRS2': {a:0.58,c:0.02},
'MTK1': {a:0.46,c:0},
'TAOK': {a:0.52,c:0.06},
'MEK1_2': {a:0.44,c:0},
'HIF1': {a:0.34,c:0.08},
'p14': {a:0.56,c:0},
'GADD45': {a:0.36,c:0},
'MAX': {a:0.52,c:0},
'MYC': {a:0.5,c:0},
'ATF2': {a:0.38,c:0},
'PLCG': {a:0.52,c:0.18},
'GAB1': {a:0.48,c:0},
'RAS': {a:0.42,c:0},
'GRB2': {a:0.54,c:0.06},
'SMAD': {a:0.44,c:0},
'p70': {a:0.4,c:0},
'DUSP1': {a:0.5,c:0},
'MSK': {a:0.58,c:0},
'p38': {a:0.52,c:0},
'p21': {a:0.52,c:0},
'FGFR3': {a:0.36,c:0.14},
'ELK1': {a:0.48,c:0},
'TAK1': {a:0.46,c:0.06}}
};
a[3]=
{2: {'ELK1': {a:0.48,c:0},
'ATM': {a:0.48,c:0.08},
'MDM2': {a:0.62,c:0},
'FGFR3': {a:0.36,c:0.14},
'TGFBR': {a:0.36,c:0.1},
'p70': {a:0.4,c:0},
'MAX': {a:0.52,c:0},
'DUSP1': {a:0.5,c:0},
'JUN': {a:0.56,c:0},
'p14': {a:0.56,c:0},
'HIF1': {a:0.34,c:0.08},
'RAF': {a:0.42,c:0},
'TAK1': {a:0.46,c:0.06},
'GAB1': {a:0.48,c:0},
'PDK1': {a:0.4,c:0},
'FOXO3': {a:0.56,c:0},
'SOS': {a:0.44,c:0},
'CREB': {a:0.52,c:0},
'AKT': {a:0.42,c:0},
'MYC': {a:0.5,c:0},
'GRB2': {a:0.54,c:0.06},
'PKC': {a:0.44,c:0},
'EGFR': {a:0.64,c:0.14},
'p21': {a:0.52,c:0},
'MSK': {a:0.58,c:0},
'PI3K': {a:0.56,c:0},
'SMAD': {a:0.44,c:0},
'ATF2': {a:0.38,c:0},
'PTEN': {a:0.46,c:0},
'RSK': {a:0.44,c:0},
'p53': {a:0.2,c:0},
'p38': {a:0.52,c:0},
'AP1': {a:0.52,c:0},
'FOS': {a:0.5,c:0},
'ERK': {a:0.5,c:0},
'MAP3K1_3': {a:0.56,c:0},
'GADD45': {a:0.36,c:0},
'TAOK': {a:0.52,c:0.06},
'PLCG': {a:0.52,c:0.18},
'PPP2CA': {a:0.46,c:0},
'BCL2': {a:0.42,c:0},
'MEK1_2': {a:0.44,c:0},
'MTK1': {a:0.46,c:0},
'SPRY': {a:0.46,c:0},
'JNK': {a:0.5,c:0},
'RAS': {a:0.42,c:0},
'FRS2': {a:0.58,c:0.02}}
};
a[4]=
{1: {'BCL2': {a:0.42,c:0},
'PDK1': {a:0.4,c:0},
'PKC': {a:0.48,c:0.04},
'GADD45': {a:0.34,c:0.02},
'AP1': {a:0.52,c:0},
'ATM': {a:0.48,c:0.08},
'ATF2': {a:0.38,c:0},
'PLCG': {a:0.54,c:0.2},
'JUN': {a:0.56,c:0},
'p38': {a:0.56,c:0.04},
'HIF1': {a:0.34,c:0.08},
'MSK': {a:0.58,c:0},
'TGFBR': {a:0.34,c:0.12},
'TAK1': {a:0.46,c:0.06},
'p70': {a:0.4,c:0},
'p21': {a:0.52,c:0},
'PPP2CA': {a:0.46,c:0},
'p14': {a:0.56,c:0},
'MDM2': {a:0.6,c:0.02},
'JNK': {a:0.52,c:0.06},
'MTK1': {a:0.46,c:0},
'ERK': {a:0.5,c:0},
'SMAD': {a:0.44,c:0},
'ELK1': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0},
'MAX': {a:0.52,c:0},
'p53': {a:0.2,c:0},
'RAS': {a:0.48,c:0.1},
'SOS': {a:0.46,c:0.02},
'CREB': {a:0.52,c:0},
'GAB1': {a:0.5,c:0.02},
'AKT': {a:0.42,c:0},
'TAOK': {a:0.5,c:0.08},
'RAF': {a:0.42,c:0},
'EGFR': {a:0.6,c:0.18},
'FRS2': {a:0.54,c:0.06},
'RSK': {a:0.44,c:0},
'MYC': {a:0.5,c:0},
'GRB2': {a:0.56,c:0.08},
'DUSP1': {a:0.5,c:0},
'MAP3K1_3': {a:0.56,c:0},
'FGFR3': {a:0.34,c:0.16},
'FOXO3': {a:0.56,c:0},
'PTEN': {a:0.44,c:0.02},
'FOS': {a:0.5,c:0},
'PI3K': {a:0.56,c:0},
'SPRY': {a:0.46,c:0}}
};
a[5]=
{1: {'PLCG': {a:0.54,c:0.2},
'SMAD': {a:0.44,c:0},
'p21': {a:0.52,c:0},
'p70': {a:0.4,c:0},
'p38': {a:0.56,c:0.04},
'ERK': {a:0.5,c:0},
'HIF1': {a:0.34,c:0.08},
'DUSP1': {a:0.5,c:0},
'PKC': {a:0.48,c:0.04},
'JNK': {a:0.52,c:0.06},
'EGFR': {a:0.64,c:0.18},
'TAK1': {a:0.46,c:0.06},
'CREB': {a:0.52,c:0},
'SOS': {a:0.46,c:0.02},
'PPP2CA': {a:0.46,c:0},
'RAF': {a:0.42,c:0},
'ATF2': {a:0.38,c:0},
'TGFBR': {a:0.34,c:0.12},
'MYC': {a:0.5,c:0},
'AP1': {a:0.52,c:0},
'MEK1_2': {a:0.44,c:0},
'SPRY': {a:0.46,c:0},
'FOXO3': {a:0.56,c:0},
'FRS2': {a:0.54,c:0.06},
'TAOK': {a:0.5,c:0.08},
'p53': {a:0.2,c:0},
'MDM2': {a:0.6,c:0.02},
'GAB1': {a:0.5,c:0.02},
'AKT': {a:0.42,c:0},
'MTK1': {a:0.46,c:0},
'FOS': {a:0.5,c:0},
'MAX': {a:0.52,c:0},
'JUN': {a:0.56,c:0},
'ELK1': {a:0.48,c:0},
'BCL2': {a:0.42,c:0},
'PTEN': {a:0.44,c:0.02},
'RAS': {a:0.48,c:0.1},
'FGFR3': {a:0.34,c:0.16},
'p14': {a:0.56,c:0},
'MAP3K1_3': {a:0.56,c:0},
'GADD45': {a:0.34,c:0.02},
'MSK': {a:0.58,c:0},
'PDK1': {a:0.4,c:0},
'PI3K': {a:0.56,c:0},
'RSK': {a:0.44,c:0},
'GRB2': {a:0.56,c:0.08},
'ATM': {a:0.48,c:0.08}}
};
a[6]=
{1: {'PPP2CA': {a:0.46,c:0},
'MDM2': {a:0.6,c:0.02},
'SPRY': {a:0.46,c:0},
'MAX': {a:0.54,c:0.02},
'MYC': {a:0.5,c:0},
'PDK1': {a:0.4,c:0},
'MSK': {a:0.58,c:0},
'FGFR3': {a:0.34,c:0.16},
'ATF2': {a:0.46,c:0.08},
'RAS': {a:0.5,c:0.12},
'PLCG': {a:0.56,c:0.22},
'TAOK': {a:0.48,c:0.1},
'GAB1': {a:0.52,c:0.04},
'EGFR': {a:0.58,c:0.24},
'FOXO3': {a:0.52,c:0.04},
'AKT': {a:0.42,c:0},
'BCL2': {a:0.42,c:0},
'PKC': {a:0.48,c:0.04},
'GRB2': {a:0.54,c:0.18},
'SMAD': {a:0.44,c:0},
'AP1': {a:0.52,c:0},
'HIF1': {a:0.34,c:0.08},
'p70': {a:0.4,c:0},
'MAP3K1_3': {a:0.58,c:0.02},
'ERK': {a:0.5,c:0},
'MTK1': {a:0.46,c:0},
'RAF': {a:0.42,c:0.04},
'RSK': {a:0.44,c:0},
'JUN': {a:0.58,c:0.02},
'DUSP1': {a:0.5,c:0},
'p14': {a:0.56,c:0},
'p21': {a:0.52,c:0},
'TAK1': {a:0.42,c:0.1},
'FOS': {a:0.5,c:0},
'ATM': {a:0.48,c:0.08},
'p53': {a:0.22,c:0.02},
'ELK1': {a:0.48,c:0},
'CREB': {a:0.52,c:0},
'TGFBR': {a:0.34,c:0.12},
'SOS': {a:0.46,c:0.02},
'MEK1_2': {a:0.44,c:0},
'JNK': {a:0.52,c:0.06},
'PI3K': {a:0.56,c:0.04},
'p38': {a:0.56,c:0.04},
'GADD45': {a:0.34,c:0.02},
'FRS2': {a:0.54,c:0.06},
'PTEN': {a:0.44,c:0.02}}
};
a[7]=
{2: {'SMAD': {a:0.44,c:0},
'GADD45': {a:0.34,c:0.02},
'JUN': {a:0.58,c:0.02},
'PKC': {a:0.48,c:0.04},
'GAB1': {a:0.52,c:0.04},
'AP1': {a:0.52,c:0},
'PTEN': {a:0.44,c:0.02},
'MDM2': {a:0.6,c:0.02},
'FOXO3': {a:0.52,c:0.04},
'TGFBR': {a:0.34,c:0.12},
'ATM': {a:0.48,c:0.08},
'p21': {a:0.52,c:0},
'ATF2': {a:0.46,c:0.08},
'PI3K': {a:0.56,c:0.04},
'ERK': {a:0.5,c:0},
'MYC': {a:0.5,c:0},
'BCL2': {a:0.42,c:0},
'MAX': {a:0.54,c:0.02},
'TAK1': {a:0.42,c:0.1},
'PDK1': {a:0.4,c:0},
'DUSP1': {a:0.5,c:0},
'PPP2CA': {a:0.46,c:0},
'MTK1': {a:0.46,c:0},
'ELK1': {a:0.48,c:0},
'FRS2': {a:0.54,c:0.06},
'GRB2': {a:0.54,c:0.18},
'CREB': {a:0.52,c:0},
'JNK': {a:0.52,c:0.06},
'p53': {a:0.22,c:0.02},
'RSK': {a:0.44,c:0},
'FOS': {a:0.5,c:0},
'AKT': {a:0.42,c:0},
'PLCG': {a:0.56,c:0.22},
'SOS': {a:0.46,c:0.02},
'RAF': {a:0.42,c:0.04},
'p38': {a:0.56,c:0.04},
'MSK': {a:0.58,c:0},
'RAS': {a:0.5,c:0.12},
'SPRY': {a:0.46,c:0},
'TAOK': {a:0.48,c:0.1},
'p14': {a:0.56,c:0},
'EGFR': {a:0.62,c:0.24},
'MEK1_2': {a:0.44,c:0},
'p70': {a:0.4,c:0},
'FGFR3': {a:0.34,c:0.16},
'MAP3K1_3': {a:0.58,c:0.02},
'HIF1': {a:0.34,c:0.08}}
};
a[8]=
{2: {'SOS': {a:0.44,c:0.04},
'GADD45': {a:0.34,c:0.02},
'PI3K': {a:0.6,c:0.08},
'HIF1': {a:0.34,c:0.08},
'MAX': {a:0.54,c:0.02},
'GRB2': {a:0.58,c:0.22},
'p21': {a:0.52,c:0},
'RAF': {a:0.42,c:0.04},
'FRS2': {a:0.54,c:0.06},
'p70': {a:0.4,c:0},
'p14': {a:0.56,c:0},
'BCL2': {a:0.42,c:0},
'ATF2': {a:0.46,c:0.08},
'ELK1': {a:0.48,c:0},
'CREB': {a:0.52,c:0},
'RSK': {a:0.44,c:0},
'DUSP1': {a:0.5,c:0},
'RAS': {a:0.5,c:0.12},
'JUN': {a:0.58,c:0.02},
'MTK1': {a:0.46,c:0},
'ATM': {a:0.48,c:0.08},
'TGFBR': {a:0.34,c:0.12},
'ERK': {a:0.5,c:0},
'PPP2CA': {a:0.46,c:0},
'MSK': {a:0.58,c:0},
'GAB1': {a:0.52,c:0.04},
'SMAD': {a:0.44,c:0},
'TAK1': {a:0.42,c:0.1},
'FGFR3': {a:0.32,c:0.18},
'EGFR': {a:0.64,c:0.24},
'JNK': {a:0.56,c:0.1},
'PKC': {a:0.48,c:0.04},
'SPRY': {a:0.46,c:0},
'FOXO3': {a:0.52,c:0.04},
'MEK1_2': {a:0.44,c:0.04},
'PDK1': {a:0.38,c:0.02},
'TAOK': {a:0.48,c:0.1},
'AKT': {a:0.42,c:0},
'p38': {a:0.58,c:0.06},
'PTEN': {a:0.44,c:0.02},
'FOS': {a:0.5,c:0},
'MYC': {a:0.5,c:0.04},
'PLCG': {a:0.54,c:0.24},
'p53': {a:0.2,c:0.02},
'AP1': {a:0.54,c:0.06},
'MDM2': {a:0.6,c:0.02},
'MAP3K1_3': {a:0.6,c:0.04}}
};
a[9]=
{1: {'MAP3K1_3': {a:0.6,c:0.04},
'CREB': {a:0.52,c:0},
'RSK': {a:0.44,c:0},
'ATF2': {a:0.46,c:0.08},
'TAOK': {a:0.48,c:0.1},
'BCL2': {a:0.42,c:0},
'FOS': {a:0.5,c:0},
'MTK1': {a:0.46,c:0},
'JUN': {a:0.58,c:0.02},
'FGFR3': {a:0.32,c:0.18},
'SMAD': {a:0.44,c:0},
'JNK': {a:0.56,c:0.1},
'TGFBR': {a:0.34,c:0.12},
'FOXO3': {a:0.52,c:0.04},
'MAX': {a:0.54,c:0.02},
'AP1': {a:0.54,c:0.06},
'p53': {a:0.2,c:0.02},
'MYC': {a:0.5,c:0.04},
'MEK1_2': {a:0.44,c:0.04},
'p21': {a:0.52,c:0},
'PDK1': {a:0.38,c:0.02},
'GRB2': {a:0.58,c:0.22},
'MDM2': {a:0.6,c:0.02},
'SPRY': {a:0.46,c:0},
'RAS': {a:0.5,c:0.12},
'ATM': {a:0.48,c:0.08},
'GADD45': {a:0.34,c:0.02},
'PKC': {a:0.48,c:0.04},
'SOS': {a:0.44,c:0.04},
'PTEN': {a:0.44,c:0.02},
'ERK': {a:0.5,c:0},
'ELK1': {a:0.48,c:0},
'p14': {a:0.56,c:0},
'p70': {a:0.4,c:0},
'HIF1': {a:0.34,c:0.08},
'EGFR': {a:0.64,c:0.24},
'TAK1': {a:0.42,c:0.1},
'p38': {a:0.58,c:0.06},
'PPP2CA': {a:0.46,c:0},
'AKT': {a:0.42,c:0},
'GAB1': {a:0.52,c:0.04},
'MSK': {a:0.58,c:0},
'PI3K': {a:0.6,c:0.08},
'DUSP1': {a:0.5,c:0},
'PLCG': {a:0.54,c:0.24},
'FRS2': {a:0.54,c:0.06},
'RAF': {a:0.42,c:0.04}}
};
a[10]=
{2: {'MAX': {a:0.54,c:0.02},
'AKT': {a:0.38,c:0.04},
'GAB1': {a:0.52,c:0.04},
'GRB2': {a:0.58,c:0.22},
'MAP3K1_3': {a:0.6,c:0.04},
'p70': {a:0.4,c:0},
'AP1': {a:0.54,c:0.06},
'MTK1': {a:0.46,c:0},
'TAK1': {a:0.4,c:0.12},
'SPRY': {a:0.46,c:0},
'RAF': {a:0.42,c:0.04},
'HIF1': {a:0.34,c:0.08},
'RSK': {a:0.44,c:0},
'FRS2': {a:0.54,c:0.06},
'MYC': {a:0.5,c:0.04},
'FOXO3': {a:0.52,c:0.04},
'SMAD': {a:0.44,c:0},
'ATF2': {a:0.48,c:0.1},
'p53': {a:0.2,c:0.02},
'ELK1': {a:0.48,c:0},
'PTEN': {a:0.44,c:0.02},
'ATM': {a:0.48,c:0.08},
'TAOK': {a:0.48,c:0.1},
'DUSP1': {a:0.5,c:0},
'PKC': {a:0.48,c:0.04},
'PDK1': {a:0.4,c:0.04},
'MSK': {a:0.58,c:0},
'JUN': {a:0.6,c:0.04},
'MEK1_2': {a:0.38,c:0.1},
'PLCG': {a:0.56,c:0.26},
'EGFR': {a:0.64,c:0.24},
'CREB': {a:0.52,c:0},
'GADD45': {a:0.34,c:0.02},
'PI3K': {a:0.62,c:0.1},
'TGFBR': {a:0.34,c:0.12},
'JNK': {a:0.56,c:0.1},
'p38': {a:0.58,c:0.06},
'p21': {a:0.52,c:0},
'BCL2': {a:0.42,c:0},
'p14': {a:0.58,c:0.02},
'FGFR3': {a:0.32,c:0.18},
'PPP2CA': {a:0.46,c:0},
'RAS': {a:0.5,c:0.12},
'ERK': {a:0.46,c:0.08},
'MDM2': {a:0.6,c:0.02},
'FOS': {a:0.5,c:0},
'SOS': {a:0.44,c:0.04}}
};
replicates[replicate_counter++] = {l: l, s: s, a: a};
l = [];
s = [];
a = [];

minimum_x = Math.min(minimum_x, -2);
maximum_x = Math.max(maximum_x, 2);
minimum_y = Math.min(minimum_y, -2);
maximum_y = Math.max(maximum_y, 2);

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>
<!--
			This GraphML document was generated from GXL by
			a GXL-to-GraphML conversion style sheet.
		-->
<graphml>
   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>
   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>
   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>
   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>
   <key id="MY-VALUE"
        for="node"
        attr.name="MY-VALUE"
        attr.type="string"/>
   <key id="MY-SHAPE"
        for="node"
        attr.name="MY-SHAPE"
        attr.type="string"/>
   <key id="PROBABILITY-OF-LINK"
        for="edge"
        attr.name="PROBABILITY-OF-LINK"
        attr.type="string"/>
   <key id="SIGN-OF-LINK"
        for="edge"
        attr.name="SIGN-OF-LINK"
        attr.type="string"/>
   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>
   <graph id="MAPK_large_HIF1" edgedefault="directed">
      <node id="EGFR_stimulus">
         <data key="MY-ID">EGFR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce EGFR activation
</node>
      <node id="FGFR3_stimulus">
         <data key="MY-ID">FGFR3_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce FGFR3 activation
</node>
      <node id="TGFBR_stimulus">
         <data key="MY-ID">TGFBR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce TGFBR activation

</node>
      <node id="DNA_damage">
         <data key="MY-ID">DNA_damage</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: DNA damaging agents/therapies action

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</node>
      <node id="Apoptosis">
         <data key="MY-ID">Apoptosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: apoptosis enablement

Modelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="Growth_Arrest">
         <data key="MY-ID">Growth_Arrest</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p21</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: growth arrest enablement

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="Proliferation">
         <data key="MY-ID">Proliferation</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and MYC and not p21</data>
         <data key="MY-SHAPE">rect</data>Proliferation

VARIABLE: proliferation enablement

Proliferation is an output variable of the model.

Modelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832

</node>
      <node id="ERK">
         <data key="MY-ID">ERK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MEK1_2</data>
         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1
MAPK3 = mitogen-activated protein kinase 3

VARIABLE: ERK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
RSKs interact with ERK and dissociate upon activation.
PMID:15187187

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p38">
         <data key="MY-ID">p38</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11
MAPK12 = mitogen-activated protein kinase 12
MAPK13 = mitogen-activated protein kinase 13
MAPK14 = mitogen-activated protein kinase 14

VARIABLE: p38 (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
MAX is phosphorylated (and activated) by p38, through complex formation.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187

ELK1 is a nuclear p38 target.
PMID:20506250

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="JNK">
         <data key="MY-ID">JNK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8
MAPK9 = mitogen-activated protein kinase 9
MAPK10 = mitogen-activated protein kinase 10

VARIABLE: JNK (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
JNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
ELK1 is a nuclear JNK target.
MEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="p53">
         <data key="MY-ID">p53</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>
         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53

VARIABLE: p53 phosphorylation level

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942

Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.
PMID:17158541

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.
PMID:12217521</node>
      <node id="p21">
         <data key="MY-ID">p21</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not AKT and p53</data>
         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)

VARIABLE: p21 activation level

p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

p21 is transcriptionally induced by the p53 protein.
p21 is a CDK inhibitor.
PMID:17158541

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="TGFBR">
         <data key="MY-ID">TGFBR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR_stimulus</data>
         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1
TGFBR2 = transforming growth factor, beta receptor II (70/80kDa)
TGFBR3 = transforming growth factor, beta receptor III

VARIABLE: TGFBR activation level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta.
PMID:21614932

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</node>
      <node id="EGFR">
         <data key="MY-ID">EGFR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>
         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor

VARIABLE: EGFR activation level

The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</node>
      <node id="FGFR3">
         <data key="MY-ID">FGFR3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>
         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3

VARIABLE: FGFR3 activation level

Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="ATM">
         <data key="MY-ID">ATM</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">DNA_damage</data>
         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated

VARIABLE: ATM phosphorylation level

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942
</node>
      <node id="TAOK">
         <data key="MY-ID">TAOK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ATM</data>
         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1
TAOK2 = TAO kinase 2
TAOK3 = TAO kinase 3

VARIABLE: TAOK (any isoform) phosphorylation level

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</node>
      <node id="MAX">
         <data key="MY-ID">MAX</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X

VARIABLE: MAX phosphorylation level

MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</node>
      <node id="GRB2">
         <data key="MY-ID">GRB2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2

VARIABLE: GRB2 recruitment by activated RTKs

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</node>
      <node id="FRS2">
         <data key="MY-ID">FRS2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>
         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2

VARIABLE: FRS2 phosphorylation level

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</node>
      <node id="PI3K">
         <data key="MY-ID">PI3K</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>
         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide

VARIABLE: PI3K activation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Recruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497</node>
      <node id="AP1">
         <data key="MY-ID">AP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JUN and (FOS or ATF2)</data>
         <data key="MY-SHAPE">ellipse</data>AP-1

VARIABLE: AP-1 formation

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345

AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="PPP2CA">
         <data key="MY-ID">PPP2CA</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme

VARIABLE: PPP2CA activation level

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</node>
      <node id="MEK1_2">
         <data key="MY-ID">MEK1_2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1
MAP2K2 = mitogen-activated protein kinase kinase 2

VARIABLE: MEK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="DUSP1">
         <data key="MY-ID">DUSP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB</data>
         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1

VARIABLE: DUSP1 gene expression level

DUSP1 is a CREB target gene.
PMID:19815709

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</node>
      <node id="MYC">
         <data key="MY-ID">MYC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>
         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)

VARIABLE: MYC activation level (implies gene expression)

Modelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.
MYC inhibits p16, whereas it activates p14.
PMID:17158541

MAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345

Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</node>
      <node id="AKT">
         <data key="MY-ID">AKT</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and not PTEN</data>
         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1
AKT2 = v-akt murine thymoma viral oncogene homolog 2
AKT3 = v-akt murine thymoma viral oncogene homolog 3

VARIABLE: AKT (any isoform) phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186

p21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="PLCG">
         <data key="MY-ID">PLCG</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FGFR3</data>
         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1
PLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)

VARIABLE: PLCG (any isoform) phosphorylation level

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="PKC">
         <data key="MY-ID">PKC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PLCG</data>
         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha
PRKCB = protein kinase C, beta
PRKCG = protein kinase C, gamma

VARIABLE: PKC (any isoform) enzymatic activity

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="GADD45">
         <data key="MY-ID">GADD45</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SMAD or p53</data>
         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha
GADD45B = growth arrest and DNA-damage-inducible, beta
GADD45G = growth arrest and DNA-damage-inducible, gamma

VARIABLE: GADD45 gene (any isoform) expression level

The GADD45 gene is inducible by various environmental stresses.
Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</node>
      <node id="ELK1">
         <data key="MY-ID">ELK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family

VARIABLE: ELK1 phosphorylation level

The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

ELK1 is a nuclear p38 target.
PMID:20506250

ELK1 is a nuclear JNK target.
ELK1 induces c-fos expression.
PMID:11274345</node>
      <node id="FOS">
         <data key="MY-ID">FOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>
         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog

VARIABLE: FOS phosphorylation level

Further observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
ELK1 induces c-fos expression.
PMID:11274345

FOS is a CREB target gene.
PMID:19815709

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</node>
      <node id="ATF2">
         <data key="MY-ID">ATF2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2

VARIABLE: ATF2 phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</node>
      <node id="JUN">
         <data key="MY-ID">JUN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK</data>
         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene

VARIABLE: JUN phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</node>
      <node id="MSK">
         <data key="MY-ID">MSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or p38</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4
RPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5

VARIABLE: MSK (any isoform) phosphorylation level

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</node>
      <node id="CREB">
         <data key="MY-ID">CREB</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MSK</data>
         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1

VARIABLE: CREB phosphorylation level

MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</node>
      <node id="RSK">
         <data key="MY-ID">RSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1
RPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2
RPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3
RPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6

VARIABLE: RSK (any isoform) phosphorylation level

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</node>
      <node id="SMAD">
         <data key="MY-ID">SMAD</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2
SMAD3 = SMAD family member 3
SMAD4 = SMAD family member 4

VARIABLE: SMAD gene (any isoform) expression level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</node>
      <node id="MTK1">
         <data key="MY-ID">MTK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GADD45</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4

VARIABLE: MTK1 phosphorylation level

The delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</node>
      <node id="SPRY">
         <data key="MY-ID">SPRY</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)

VARIABLE: SPRY2 phosphorylation level

Spry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.
PMID:15173823</node>
      <node id="RAF">
         <data key="MY-ID">RAF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>
         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1
BRAF = v-raf murine sarcoma viral oncogene homolog B1
ARAF = v-raf murine sarcoma 3611 viral oncogene homolog

VARIABLE: RAF (any isoform) phosphorylation

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

PKCalpha directly phosphorylates and activates Raf-1.
PMID:8321321

Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</node>
      <node id="GAB1">
         <data key="MY-ID">GAB1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 or PI3K</data>
         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1

VARIABLE: GAB1 phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
PMID:15567848

Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</node>
      <node id="PDK1">
         <data key="MY-ID">PDK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PI3K</data>
         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1

VARIABLE: PDK1 activation level

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</node>
      <node id="p70">
         <data key="MY-ID">p70</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1
RPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2

VARIABLE: p70 (any isoform) phosphorylation level

PDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.
PMID:12040186

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p14">
         <data key="MY-ID">p14</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MYC</data>
         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)

VARIABLE: p14 activation level

MYC inhibits p16, whereas it activates p14.
The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</node>
      <node id="FOXO3">
         <data key="MY-ID">FOXO3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK and not AKT</data>
         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3

VARIABLE: FOXO3 phosphorylation level

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</node>
      <node id="RAS">
         <data key="MY-ID">RAS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SOS or PLCG</data>
         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog
KRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog
NRAS = neuroblastoma RAS viral (v-ras) oncogene homolog

VARIABLE: RAS (any isoform) phosphorylation level

Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Most of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="SOS">
         <data key="MY-ID">SOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 and not RSK</data>
         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)
SOS2 = son of sevenless homolog 2 (Drosophila)

VARIABLE: SOS (any isoform) recruitment by GRB2

Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373
</node>
      <node id="MDM2">
         <data key="MY-ID">MDM2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(p53 or AKT) and not p14</data>
         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)

VARIABLE: MDM2 gene expression level

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.
PMID:17158541

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850
</node>
      <node id="BCL2">
         <data key="MY-ID">BCL2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB and AKT</data>
         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2

VARIABLE: BCL2 activation level (implies gene expression)

Modelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.

DUSP1, FOS, BCL2 are CREB target genes.
PMID:19815709

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="TAK1">
         <data key="MY-ID">TAK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7

VARIABLE: TAK1 phosphorylation level

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="MAP3K1_3">
         <data key="MY-ID">MAP3K1_3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">RAS</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1
MAP3K2 = mitogen-activated protein kinase kinase kinase 2
MAP3K3 = mitogen-activated protein kinase kinase kinase 3

VARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</node>
      <node id="PTEN">
         <data key="MY-ID">PTEN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p53</data>
         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog

VARIABLE: PTEN protein expression level

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="HIF1">
         <data key="MY-ID">HIF1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="EGF">
         <data key="MY-ID">EGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="VEGF">
         <data key="MY-ID">VEGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Oxygen_supply">
         <data key="MY-ID">Oxygen_supply</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Necrosis">
         <data key="MY-ID">Necrosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not Oxygen_supply</data>
         <data key="MY-SHAPE">rect</data>
      </node>
      <edge source="AP1" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</edge>
      <edge source="EGFR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910</edge>
      <edge source="EGFR" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</edge>
      <edge source="Oxygen_supply" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="Oxygen_supply" target="Necrosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MSK" target="CREB" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187</edge>
      <edge source="MSK" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</edge>
      <edge source="MEK1_2" target="ERK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="HIF1" target="EGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="HIF1" target="VEGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="DUSP1" target="p38" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="DUSP1" target="JNK" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="PI3K" target="PDK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124</edge>
      <edge source="PI3K" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="FGFR3" target="FRS2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="FGFR3" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221</edge>
      <edge source="DNA_damage" target="ATM" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</edge>
      <edge source="FRS2" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="MAX" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345</edge>
      <edge source="ATF2" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="ERK" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="ERK" target="RSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952</edge>
      <edge source="ERK" target="SPRY" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823</edge>
      <edge source="ERK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="ERK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ERK" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650
</edge>
      <edge source="ERK" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</edge>
      <edge source="ERK" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</edge>
      <edge source="ERK" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="CREB" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="DUSP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;



DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="JUN" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="FOXO3" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166</edge>
      <edge source="PLCG" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</edge>
      <edge source="PLCG" target="PKC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848</edge>
      <edge source="SOS" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929</edge>
      <edge source="SOS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="p70" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="p70" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MDM2" target="p53" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="TGFBR" target="SMAD" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="TGFBR" target="TAK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TGFBR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</edge>
      <edge source="GAB1" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="MTK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="MTK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="GRB2" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848</edge>
      <edge source="GRB2" target="SOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910</edge>
      <edge source="GRB2" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="GRB2" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848</edge>
      <edge source="GRB2" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="BCL2" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="ATM" target="TAOK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897</edge>
      <edge source="ATM" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942</edge>
      <edge source="GADD45" target="MTK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="PTEN" target="AKT" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="p14" target="MDM2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</edge>
      <edge source="PPP2CA" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="PPP2CA" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="p38" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932</edge>
      <edge source="p38" target="MAX" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</edge>
      <edge source="p38" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.
PMID:20506250
</edge>
      <edge source="p38" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="p53" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</edge>
      <edge source="p53" target="p21" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.
PMID:17158541</edge>
      <edge source="p53" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="p53" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508</edge>
      <edge source="p53" target="PTEN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="SMAD" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="SPRY" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="SPRY" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="EGFR_stimulus" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ELK1" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 induces c-fos expression.
PMID:11274345</edge>
      <edge source="TAK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TAK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="AKT" target="FOXO3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</edge>
      <edge source="AKT" target="p21" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383</edge>
      <edge source="AKT" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="AKT" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186</edge>
      <edge source="AKT" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850</edge>
      <edge source="AKT" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359</edge>
      <edge source="AKT" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="RAF" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="FOS" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="TAOK" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="TAOK" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="JNK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.
PMID:11274345</edge>
      <edge source="JNK" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="JNK" target="JUN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</edge>
      <edge source="JNK" target="FOXO3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971</edge>
      <edge source="p21" target="Proliferation" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="p21" target="Growth_Arrest" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="PKC" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473</edge>
      <edge source="PKC" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321</edge>
      <edge source="PKC" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</edge>
      <edge source="RSK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="RSK" target="SOS" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</edge>
      <edge source="PDK1" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="PDK1" target="AKT" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</edge>
      <edge source="MYC" target="p14" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.
PMID:17158541</edge>
      <edge source="MYC" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</edge>
      <edge source="RAS" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929</edge>
      <edge source="RAS" target="MAP3K1_3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="RAS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.
Proliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.
Input signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>
</graphml>
';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
l[1]=
[{added: 30},
{added: 43},
{added: 13},
{added: 9},
{added: 37},
{added: 48},
{added: 26},
{added: 10},
{added: 8},
{added: 19},
{added: 34},
{added: 50},
{added: 4},
{added: 28},
{added: 42},
{added: 51},
{added: 44},
{added: 45},
{added: 6},
{added: 47},
{added: 31},
{added: 21},
{added: 25},
{added: 17},
{added: 46},
{added: 23},
{added: 49},
{added: 24},
{added: 32},
{added: 29},
{added: 35},
{added: 11},
{added: 33},
{added: 5},
{added: 16},
{added: 14},
{added: 52},
{added: 40},
{added: 20},
{added: 39},
{added: 7},
{added: 36},
{added: 22},
{added: 12},
{added: 15},
{added: 41},
{added: 38},
{added: 27},
{added: 18},
{added: 53},
{x: 0, y: -2, z: 1, who: 31},
{c: 1, x: -2, y: 1, z: 1, who: 19},
{c: 1, x: -1, y: -2, z: 0, who: 23},
{x: -2, y: 1, z: 0, who: 38},
{c: 1, x: 1, y: -1, z: 0, who: 13},
{x: 2, y: 1, z: 1, who: 40},
{x: 0, y: -1, z: 1, who: 53},
{c: 1, x: 1, y: -2, z: 1, who: 14},
{c: 1, x: 1, y: -2, z: 0, who: 12},
{c: 1, x: -2, y: -2, z: 0, who: 41},
{x: 0, y: -1, z: 0, who: 34},
{c: 1, x: 0, y: 2, z: 0, who: 24},
{c: 1, x: 1, y: 0, z: 1, who: 16},
{c: 1, x: 1, y: 1, z: 1, who: 50},
{x: 2, y: 0, z: 0, who: 8},
{x: 2, y: 2, z: 1, who: 10},
{c: 1, x: 1, y: 2, z: 0, who: 52},
{c: 1, x: 2, y: -1, z: 1, who: 45},
{c: 1, x: -1, y: 0, z: 0, who: 30},
{x: -1, y: 2, z: 0, who: 9},
{x: 2, y: -2, z: 1, who: 47},
{c: 1, x: -1, y: 1, z: 1, who: 18},
{x: 0, y: -2, z: 0, who: 4},
{c: 1, x: 2, y: 1, z: 0, who: 33},
{x: -1, y: 0, z: 1, who: 39},
{c: 1, x: -2, y: -1, z: 0, who: 37},
{x: 0, y: 2, z: 1, who: 17},
{x: -2, y: -1, z: 1, who: 27},
{x: 1, y: -1, z: 1, who: 46},
{x: -1, y: -1, z: 0, who: 32},
{x: 0, y: 0, z: 0, who: 15},
{x: 2, y: -1, z: 0, who: 21},
{c: 1, x: 0, y: 0, z: 1, who: 7},
{x: -2, y: 0, z: 1, who: 25},
{c: 1, x: -1, y: -2, z: 1, who: 5},
{x: 2, y: 0, z: 1, who: 22},
{x: -1, y: -1, z: 1, who: 49},
{x: 0, y: 1, z: 0, who: 48},
{c: 1, x: 1, y: 2, z: 1, who: 20},
{c: 1, x: -2, y: 2, z: 0, who: 42},
{x: -2, y: -2, z: 1, who: 44},
{x: -2, y: 2, z: 1, who: 29},
{c: 1, x: 0, y: 1, z: 1, who: 11},
{c: 1, x: -1, y: 2, z: 1, who: 36},
{c: 1, x: -1, y: 1, z: 0, who: 51},
{c: 1, x: 2, y: 2, z: 0, who: 35},
{c: 1, x: 1, y: 0, z: 0, who: 28},
{x: 1, y: 1, z: 0, who: 43},
{x: 2, y: -2, z: 0, who: 6},
{c: 1, x: -2, y: 0, z: 0, who: 26}];
a[1]=
{2: {'MYC': {a:0.46,c:0},
'TAK1': {a:0.62,c:0},
'MDM2': {a:0.5,c:0},
'FRS2': {a:0.46,c:0},
'MTK1': {a:0.52,c:0},
'MAX': {a:0.56,c:0},
'p14': {a:0.66,c:0},
'JUN': {a:0.4,c:0},
'TGFBR': {a:0.38,c:0.14},
'ATM': {a:0.44,c:0.1},
'MEK1_2': {a:0.44,c:0},
'RAF': {a:0.64,c:0},
'AKT': {a:0.48,c:0},
'DUSP1': {a:0.46,c:0},
'p53': {a:0.34,c:0},
'RAS': {a:0.42,c:0},
'CREB': {a:0.48,c:0},
'HIF1': {a:0.32,c:0.1},
'PDK1': {a:0.42,c:0},
'GAB1': {a:0.5,c:0},
'p70': {a:0.54,c:0},
'ELK1': {a:0.46,c:0},
'FOS': {a:0.5,c:0},
'MSK': {a:0.48,c:0},
'MAP3K1_3': {a:0.58,c:0},
'ERK': {a:0.46,c:0},
'PTEN': {a:0.5,c:0},
'PLCG': {a:0.52,c:0},
'GRB2': {a:0.54,c:0},
'SMAD': {a:0.5,c:0},
'SOS': {a:0.54,c:0},
'PKC': {a:0.5,c:0},
'p38': {a:0.62,c:0},
'ATF2': {a:0.64,c:0},
'SPRY': {a:0.52,c:0},
'GADD45': {a:0.54,c:0},
'p21': {a:0.5,c:0},
'AP1': {a:0.48,c:0},
'EGFR': {a:0.66,c:0.08},
'PI3K': {a:0.42,c:0},
'RSK': {a:0.52,c:0},
'JNK': {a:0.48,c:0},
'FGFR3': {a:0.38,c:0.08},
'PPP2CA': {a:0.64,c:0},
'BCL2': {a:0.38,c:0},
'TAOK': {a:0.46,c:0},
'FOXO3': {a:0.52,c:0}}
};
a[1]=
{1: {'RAF': {a:0.64,c:0},
'MSK': {a:0.48,c:0},
'BCL2': {a:0.38,c:0},
'SPRY': {a:0.52,c:0},
'RAS': {a:0.42,c:0},
'TGFBR': {a:0.38,c:0.14},
'HIF1': {a:0.32,c:0.1},
'GADD45': {a:0.54,c:0},
'AP1': {a:0.48,c:0},
'CREB': {a:0.48,c:0},
'ELK1': {a:0.46,c:0},
'FOXO3': {a:0.52,c:0},
'MYC': {a:0.46,c:0},
'MAP3K1_3': {a:0.58,c:0},
'p21': {a:0.5,c:0},
'ATM': {a:0.44,c:0.1},
'TAOK': {a:0.46,c:0},
'PTEN': {a:0.5,c:0},
'GAB1': {a:0.5,c:0},
'MTK1': {a:0.52,c:0},
'FOS': {a:0.5,c:0},
'GRB2': {a:0.54,c:0},
'PDK1': {a:0.42,c:0},
'MAX': {a:0.56,c:0},
'DUSP1': {a:0.46,c:0},
'JUN': {a:0.4,c:0},
'SMAD': {a:0.5,c:0},
'PKC': {a:0.5,c:0},
'MDM2': {a:0.5,c:0},
'EGFR': {a:0.66,c:0.08},
'RSK': {a:0.52,c:0},
'PPP2CA': {a:0.64,c:0},
'SOS': {a:0.54,c:0},
'PLCG': {a:0.52,c:0},
'FRS2': {a:0.46,c:0},
'p70': {a:0.54,c:0},
'ATF2': {a:0.64,c:0},
'JNK': {a:0.48,c:0},
'p53': {a:0.34,c:0},
'p38': {a:0.62,c:0},
'p14': {a:0.66,c:0},
'AKT': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0},
'PI3K': {a:0.42,c:0},
'ERK': {a:0.46,c:0},
'TAK1': {a:0.62,c:0},
'FGFR3': {a:0.38,c:0.08}}
};
a[2]=
{2: {'ATF2': {a:0.64,c:0},
'MYC': {a:0.46,c:0},
'ATM': {a:0.44,c:0.1},
'FOS': {a:0.5,c:0},
'AKT': {a:0.48,c:0},
'PPP2CA': {a:0.64,c:0},
'p38': {a:0.62,c:0},
'TAK1': {a:0.62,c:0},
'GRB2': {a:0.56,c:0.02},
'PKC': {a:0.5,c:0},
'MTK1': {a:0.52,c:0},
'BCL2': {a:0.38,c:0},
'FOXO3': {a:0.52,c:0},
'p53': {a:0.34,c:0.04},
'HIF1': {a:0.3,c:0.12},
'SOS': {a:0.54,c:0},
'MAP3K1_3': {a:0.58,c:0},
'RSK': {a:0.52,c:0},
'p14': {a:0.66,c:0},
'PTEN': {a:0.5,c:0},
'JNK': {a:0.48,c:0},
'FRS2': {a:0.42,c:0.04},
'RAF': {a:0.64,c:0},
'AP1': {a:0.48,c:0},
'PDK1': {a:0.42,c:0},
'MSK': {a:0.48,c:0},
'JUN': {a:0.4,c:0},
'CREB': {a:0.48,c:0},
'TGFBR': {a:0.36,c:0.16},
'PLCG': {a:0.5,c:0.06},
'EGFR': {a:0.7,c:0.08},
'PI3K': {a:0.42,c:0},
'MAX': {a:0.56,c:0},
'ERK': {a:0.46,c:0},
'GADD45': {a:0.54,c:0},
'SMAD': {a:0.5,c:0},
'DUSP1': {a:0.46,c:0},
'FGFR3': {a:0.36,c:0.1},
'SPRY': {a:0.52,c:0},
'GAB1': {a:0.5,c:0},
'TAOK': {a:0.4,c:0.06},
'RAS': {a:0.42,c:0},
'p21': {a:0.5,c:0},
'ELK1': {a:0.46,c:0},
'p70': {a:0.54,c:0},
'MDM2': {a:0.5,c:0},
'MEK1_2': {a:0.44,c:0}}
};
a[3]=
{1: {'GAB1': {a:0.5,c:0},
'ATF2': {a:0.64,c:0},
'GRB2': {a:0.56,c:0.02},
'RAF': {a:0.64,c:0},
'HIF1': {a:0.3,c:0.12},
'MSK': {a:0.48,c:0},
'p38': {a:0.62,c:0},
'p21': {a:0.5,c:0},
'ATM': {a:0.44,c:0.1},
'FOS': {a:0.5,c:0},
'SOS': {a:0.54,c:0},
'TAK1': {a:0.62,c:0},
'PDK1': {a:0.42,c:0},
'FOXO3': {a:0.52,c:0},
'JNK': {a:0.48,c:0},
'p53': {a:0.34,c:0.04},
'MAP3K1_3': {a:0.58,c:0},
'p14': {a:0.66,c:0},
'p70': {a:0.54,c:0},
'MDM2': {a:0.5,c:0},
'ELK1': {a:0.46,c:0},
'JUN': {a:0.4,c:0},
'FRS2': {a:0.42,c:0.04},
'SPRY': {a:0.52,c:0},
'PTEN': {a:0.5,c:0},
'CREB': {a:0.48,c:0},
'PLCG': {a:0.5,c:0.06},
'MEK1_2': {a:0.44,c:0},
'ERK': {a:0.46,c:0},
'MAX': {a:0.56,c:0},
'PKC': {a:0.5,c:0},
'SMAD': {a:0.5,c:0},
'PPP2CA': {a:0.64,c:0},
'AP1': {a:0.48,c:0},
'AKT': {a:0.48,c:0},
'RAS': {a:0.42,c:0},
'MTK1': {a:0.52,c:0},
'RSK': {a:0.52,c:0},
'GADD45': {a:0.54,c:0},
'DUSP1': {a:0.46,c:0},
'FGFR3': {a:0.36,c:0.1},
'TGFBR': {a:0.36,c:0.16},
'MYC': {a:0.46,c:0},
'EGFR': {a:0.7,c:0.08},
'TAOK': {a:0.4,c:0.06},
'PI3K': {a:0.42,c:0},
'BCL2': {a:0.38,c:0}}
};
a[4]=
{1: {'GRB2': {a:0.62,c:0.08},
'PLCG': {a:0.48,c:0.12},
'ELK1': {a:0.46,c:0},
'FOXO3': {a:0.52,c:0},
'HIF1': {a:0.3,c:0.12},
'p38': {a:0.66,c:0.04},
'MAX': {a:0.56,c:0},
'p14': {a:0.66,c:0},
'EGFR': {a:0.64,c:0.14},
'ATF2': {a:0.64,c:0},
'RAS': {a:0.5,c:0.08},
'PTEN': {a:0.5,c:0},
'MYC': {a:0.46,c:0},
'SOS': {a:0.56,c:0.02},
'RSK': {a:0.52,c:0},
'ERK': {a:0.46,c:0},
'RAF': {a:0.64,c:0},
'AKT': {a:0.48,c:0},
'CREB': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0},
'AP1': {a:0.48,c:0},
'p70': {a:0.54,c:0},
'DUSP1': {a:0.46,c:0},
'TAOK': {a:0.4,c:0.06},
'FRS2': {a:0.42,c:0.04},
'MSK': {a:0.48,c:0},
'MDM2': {a:0.5,c:0},
'MAP3K1_3': {a:0.58,c:0},
'FOS': {a:0.5,c:0},
'p21': {a:0.5,c:0},
'JNK': {a:0.52,c:0.04},
'GADD45': {a:0.5,c:0.04},
'JUN': {a:0.4,c:0},
'BCL2': {a:0.38,c:0},
'TAK1': {a:0.62,c:0},
'PDK1': {a:0.42,c:0},
'ATM': {a:0.4,c:0.14},
'SMAD': {a:0.5,c:0},
'PI3K': {a:0.42,c:0},
'GAB1': {a:0.52,c:0.02},
'PPP2CA': {a:0.64,c:0},
'TGFBR': {a:0.34,c:0.18},
'FGFR3': {a:0.3,c:0.16},
'PKC': {a:0.5,c:0.04},
'SPRY': {a:0.52,c:0},
'p53': {a:0.32,c:0.04},
'MTK1': {a:0.52,c:0}}
};
a[5]=
{2: {'PKC': {a:0.5,c:0.04},
'AP1': {a:0.48,c:0},
'PTEN': {a:0.5,c:0},
'BCL2': {a:0.38,c:0},
'RSK': {a:0.52,c:0},
'GRB2': {a:0.62,c:0.08},
'GAB1': {a:0.52,c:0.02},
'MYC': {a:0.46,c:0},
'DUSP1': {a:0.46,c:0},
'TAOK': {a:0.4,c:0.06},
'TAK1': {a:0.62,c:0},
'PI3K': {a:0.42,c:0},
'MSK': {a:0.48,c:0},
'FGFR3': {a:0.3,c:0.16},
'p70': {a:0.54,c:0},
'FOS': {a:0.5,c:0},
'SMAD': {a:0.5,c:0},
'ELK1': {a:0.46,c:0},
'p53': {a:0.32,c:0.04},
'MAP3K1_3': {a:0.58,c:0},
'ERK': {a:0.46,c:0},
'RAS': {a:0.5,c:0.08},
'PPP2CA': {a:0.64,c:0},
'MEK1_2': {a:0.44,c:0},
'p21': {a:0.5,c:0},
'ATF2': {a:0.64,c:0},
'FRS2': {a:0.42,c:0.04},
'EGFR': {a:0.64,c:0.14},
'FOXO3': {a:0.52,c:0},
'RAF': {a:0.64,c:0},
'JNK': {a:0.52,c:0.04},
'SPRY': {a:0.52,c:0},
'MTK1': {a:0.52,c:0},
'JUN': {a:0.4,c:0},
'TGFBR': {a:0.34,c:0.18},
'GADD45': {a:0.5,c:0.04},
'p38': {a:0.66,c:0.04},
'MAX': {a:0.56,c:0},
'SOS': {a:0.56,c:0.02},
'AKT': {a:0.48,c:0},
'PLCG': {a:0.48,c:0.12},
'ATM': {a:0.4,c:0.14},
'HIF1': {a:0.3,c:0.12},
'MDM2': {a:0.5,c:0},
'p14': {a:0.66,c:0},
'CREB': {a:0.48,c:0},
'PDK1': {a:0.42,c:0}}
};
a[6]=
{1: {'RAF': {a:0.6,c:0.04},
'MAX': {a:0.56,c:0},
'TAK1': {a:0.6,c:0.02},
'RAS': {a:0.52,c:0.1},
'p38': {a:0.66,c:0.04},
'JUN': {a:0.4,c:0},
'GAB1': {a:0.52,c:0.02},
'EGFR': {a:0.64,c:0.18},
'SOS': {a:0.54,c:0.04},
'MSK': {a:0.5,c:0.02},
'SPRY': {a:0.52,c:0},
'PI3K': {a:0.48,c:0.06},
'TAOK': {a:0.4,c:0.06},
'FRS2': {a:0.4,c:0.06},
'FOS': {a:0.5,c:0},
'SMAD': {a:0.5,c:0},
'p14': {a:0.66,c:0},
'PTEN': {a:0.5,c:0},
'BCL2': {a:0.38,c:0},
'p70': {a:0.54,c:0},
'p53': {a:0.36,c:0.08},
'PKC': {a:0.48,c:0.06},
'GRB2': {a:0.64,c:0.1},
'JNK': {a:0.52,c:0.04},
'ELK1': {a:0.5,c:0.04},
'p21': {a:0.5,c:0},
'CREB': {a:0.48,c:0},
'HIF1': {a:0.3,c:0.12},
'RSK': {a:0.52,c:0},
'DUSP1': {a:0.46,c:0},
'ATF2': {a:0.64,c:0},
'PLCG': {a:0.5,c:0.14},
'MDM2': {a:0.5,c:0},
'MYC': {a:0.46,c:0},
'MEK1_2': {a:0.44,c:0},
'MAP3K1_3': {a:0.6,c:0.02},
'ATM': {a:0.4,c:0.14},
'FGFR3': {a:0.3,c:0.16},
'TGFBR': {a:0.34,c:0.18},
'GADD45': {a:0.5,c:0.04},
'ERK': {a:0.46,c:0},
'MTK1': {a:0.54,c:0.02},
'AKT': {a:0.48,c:0},
'FOXO3': {a:0.52,c:0},
'AP1': {a:0.48,c:0},
'PPP2CA': {a:0.64,c:0},
'PDK1': {a:0.42,c:0}}
};
a[7]=
{1: {'MYC': {a:0.46,c:0},
'PKC': {a:0.48,c:0.06},
'PTEN': {a:0.5,c:0},
'FOXO3': {a:0.52,c:0},
'FRS2': {a:0.4,c:0.06},
'TAK1': {a:0.6,c:0.02},
'MEK1_2': {a:0.44,c:0},
'MDM2': {a:0.5,c:0},
'RSK': {a:0.52,c:0},
'PPP2CA': {a:0.64,c:0},
'HIF1': {a:0.3,c:0.12},
'SOS': {a:0.54,c:0.04},
'ATM': {a:0.4,c:0.14},
'MSK': {a:0.5,c:0.02},
'JNK': {a:0.52,c:0.04},
'AKT': {a:0.48,c:0},
'SPRY': {a:0.52,c:0},
'JUN': {a:0.4,c:0},
'GRB2': {a:0.64,c:0.1},
'GAB1': {a:0.52,c:0.02},
'ELK1': {a:0.5,c:0.04},
'RAF': {a:0.6,c:0.04},
'RAS': {a:0.52,c:0.1},
'FOS': {a:0.5,c:0},
'p38': {a:0.66,c:0.04},
'ATF2': {a:0.64,c:0},
'CREB': {a:0.48,c:0},
'p53': {a:0.36,c:0.08},
'p21': {a:0.5,c:0},
'PDK1': {a:0.42,c:0},
'MAP3K1_3': {a:0.6,c:0.02},
'AP1': {a:0.48,c:0},
'TGFBR': {a:0.34,c:0.18},
'DUSP1': {a:0.46,c:0},
'MAX': {a:0.56,c:0},
'p14': {a:0.66,c:0},
'BCL2': {a:0.38,c:0},
'SMAD': {a:0.5,c:0},
'EGFR': {a:0.64,c:0.18},
'FGFR3': {a:0.3,c:0.16},
'p70': {a:0.54,c:0},
'ERK': {a:0.46,c:0},
'MTK1': {a:0.54,c:0.02},
'PI3K': {a:0.48,c:0.06},
'GADD45': {a:0.5,c:0.04},
'PLCG': {a:0.5,c:0.14},
'TAOK': {a:0.4,c:0.06}}
};
a[8]=
{2: {'SPRY': {a:0.52,c:0},
'MAP3K1_3': {a:0.58,c:0.04},
'ELK1': {a:0.5,c:0.04},
'p14': {a:0.66,c:0},
'MTK1': {a:0.54,c:0.02},
'SOS': {a:0.54,c:0.04},
'JNK': {a:0.54,c:0.06},
'PTEN': {a:0.48,c:0.02},
'MDM2': {a:0.5,c:0},
'FGFR3': {a:0.3,c:0.16},
'EGFR': {a:0.64,c:0.2},
'TAOK': {a:0.4,c:0.06},
'RAS': {a:0.52,c:0.1},
'RSK': {a:0.52,c:0},
'SMAD': {a:0.5,c:0},
'ATF2': {a:0.64,c:0},
'FRS2': {a:0.38,c:0.08},
'CREB': {a:0.5,c:0.02},
'MSK': {a:0.5,c:0.02},
'ATM': {a:0.4,c:0.14},
'MYC': {a:0.46,c:0.04},
'DUSP1': {a:0.46,c:0},
'ERK': {a:0.46,c:0},
'PPP2CA': {a:0.64,c:0},
'BCL2': {a:0.38,c:0},
'FOXO3': {a:0.5,c:0.02},
'MEK1_2': {a:0.38,c:0.06},
'PDK1': {a:0.4,c:0.02},
'FOS': {a:0.46,c:0.04},
'p38': {a:0.74,c:0.12},
'AKT': {a:0.48,c:0},
'PLCG': {a:0.48,c:0.16},
'TGFBR': {a:0.34,c:0.18},
'AP1': {a:0.48,c:0},
'HIF1': {a:0.3,c:0.12},
'GADD45': {a:0.5,c:0.04},
'p53': {a:0.32,c:0.08},
'RAF': {a:0.6,c:0.04},
'TAK1': {a:0.6,c:0.02},
'PKC': {a:0.5,c:0.12},
'MAX': {a:0.56,c:0},
'GRB2': {a:0.66,c:0.12},
'JUN': {a:0.4,c:0},
'GAB1': {a:0.58,c:0.08},
'p21': {a:0.48,c:0.02},
'PI3K': {a:0.52,c:0.1},
'p70': {a:0.54,c:0}}
};
a[9]=
{1: {'PI3K': {a:0.52,c:0.1},
'AKT': {a:0.48,c:0},
'PKC': {a:0.5,c:0.12},
'PDK1': {a:0.4,c:0.02},
'MEK1_2': {a:0.38,c:0.06},
'GAB1': {a:0.58,c:0.08},
'MAP3K1_3': {a:0.58,c:0.04},
'PLCG': {a:0.48,c:0.16},
'MSK': {a:0.5,c:0.02},
'GADD45': {a:0.5,c:0.04},
'FGFR3': {a:0.3,c:0.16},
'EGFR': {a:0.64,c:0.2},
'ATM': {a:0.4,c:0.14},
'MDM2': {a:0.5,c:0},
'ATF2': {a:0.64,c:0},
'SMAD': {a:0.5,c:0},
'TGFBR': {a:0.34,c:0.18},
'CREB': {a:0.5,c:0.02},
'ERK': {a:0.46,c:0},
'FOS': {a:0.46,c:0.04},
'p53': {a:0.32,c:0.08},
'AP1': {a:0.48,c:0},
'HIF1': {a:0.3,c:0.12},
'p14': {a:0.66,c:0},
'TAOK': {a:0.4,c:0.06},
'BCL2': {a:0.38,c:0},
'RAS': {a:0.52,c:0.1},
'TAK1': {a:0.6,c:0.02},
'p21': {a:0.48,c:0.02},
'JNK': {a:0.54,c:0.06},
'RAF': {a:0.6,c:0.04},
'RSK': {a:0.52,c:0},
'SPRY': {a:0.52,c:0},
'p70': {a:0.54,c:0},
'MTK1': {a:0.54,c:0.02},
'PPP2CA': {a:0.64,c:0},
'SOS': {a:0.54,c:0.04},
'ELK1': {a:0.5,c:0.04},
'p38': {a:0.74,c:0.12},
'GRB2': {a:0.66,c:0.12},
'MYC': {a:0.46,c:0.04},
'DUSP1': {a:0.46,c:0},
'FRS2': {a:0.38,c:0.08},
'FOXO3': {a:0.5,c:0.02},
'MAX': {a:0.56,c:0},
'PTEN': {a:0.48,c:0.02},
'JUN': {a:0.4,c:0}}
};
a[10]=
{1: {'RSK': {a:0.52,c:0},
'BCL2': {a:0.38,c:0},
'FOS': {a:0.46,c:0.04},
'ATF2': {a:0.64,c:0},
'HIF1': {a:0.3,c:0.12},
'ATM': {a:0.4,c:0.14},
'GRB2': {a:0.66,c:0.12},
'TAK1': {a:0.6,c:0.02},
'PLCG': {a:0.48,c:0.16},
'ERK': {a:0.44,c:0.02},
'AKT': {a:0.46,c:0.02},
'JUN': {a:0.42,c:0.02},
'PTEN': {a:0.48,c:0.02},
'MYC': {a:0.46,c:0.04},
'p14': {a:0.66,c:0},
'FGFR3': {a:0.3,c:0.16},
'MDM2': {a:0.5,c:0},
'EGFR': {a:0.64,c:0.26},
'p38': {a:0.74,c:0.12},
'PDK1': {a:0.4,c:0.02},
'PPP2CA': {a:0.64,c:0},
'SMAD': {a:0.5,c:0},
'TAOK': {a:0.4,c:0.06},
'RAF': {a:0.58,c:0.06},
'MAP3K1_3': {a:0.58,c:0.04},
'MTK1': {a:0.54,c:0.02},
'p21': {a:0.48,c:0.02},
'MSK': {a:0.5,c:0.02},
'RAS': {a:0.52,c:0.1},
'FRS2': {a:0.36,c:0.1},
'SOS': {a:0.54,c:0.04},
'GAB1': {a:0.58,c:0.08},
'SPRY': {a:0.52,c:0},
'FOXO3': {a:0.5,c:0.02},
'TGFBR': {a:0.34,c:0.18},
'p53': {a:0.32,c:0.08},
'JNK': {a:0.54,c:0.06},
'PI3K': {a:0.56,c:0.14},
'DUSP1': {a:0.46,c:0},
'p70': {a:0.54,c:0},
'ELK1': {a:0.54,c:0.08},
'GADD45': {a:0.5,c:0.04},
'MAX': {a:0.56,c:0},
'CREB': {a:0.5,c:0.02},
'AP1': {a:0.44,c:0.04},
'MEK1_2': {a:0.36,c:0.08},
'PKC': {a:0.48,c:0.14}}
};
replicates[replicate_counter++] = {l: l, s: s, a: a};
l = [];
s = [];
a = [];

minimum_x = Math.min(minimum_x, -2);
maximum_x = Math.max(maximum_x, 2);
minimum_y = Math.min(minimum_y, -2);
maximum_y = Math.max(maximum_y, 2);

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>
<!--
			This GraphML document was generated from GXL by
			a GXL-to-GraphML conversion style sheet.
		-->
<graphml>
   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>
   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>
   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>
   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>
   <key id="MY-VALUE"
        for="node"
        attr.name="MY-VALUE"
        attr.type="string"/>
   <key id="MY-SHAPE"
        for="node"
        attr.name="MY-SHAPE"
        attr.type="string"/>
   <key id="PROBABILITY-OF-LINK"
        for="edge"
        attr.name="PROBABILITY-OF-LINK"
        attr.type="string"/>
   <key id="SIGN-OF-LINK"
        for="edge"
        attr.name="SIGN-OF-LINK"
        attr.type="string"/>
   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>
   <graph id="MAPK_large_HIF1" edgedefault="directed">
      <node id="EGFR_stimulus">
         <data key="MY-ID">EGFR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce EGFR activation
</node>
      <node id="FGFR3_stimulus">
         <data key="MY-ID">FGFR3_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce FGFR3 activation
</node>
      <node id="TGFBR_stimulus">
         <data key="MY-ID">TGFBR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce TGFBR activation

</node>
      <node id="DNA_damage">
         <data key="MY-ID">DNA_damage</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: DNA damaging agents/therapies action

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</node>
      <node id="Apoptosis">
         <data key="MY-ID">Apoptosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: apoptosis enablement

Modelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="Growth_Arrest">
         <data key="MY-ID">Growth_Arrest</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p21</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: growth arrest enablement

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="Proliferation">
         <data key="MY-ID">Proliferation</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and MYC and not p21</data>
         <data key="MY-SHAPE">rect</data>Proliferation

VARIABLE: proliferation enablement

Proliferation is an output variable of the model.

Modelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832

</node>
      <node id="ERK">
         <data key="MY-ID">ERK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MEK1_2</data>
         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1
MAPK3 = mitogen-activated protein kinase 3

VARIABLE: ERK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
RSKs interact with ERK and dissociate upon activation.
PMID:15187187

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p38">
         <data key="MY-ID">p38</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11
MAPK12 = mitogen-activated protein kinase 12
MAPK13 = mitogen-activated protein kinase 13
MAPK14 = mitogen-activated protein kinase 14

VARIABLE: p38 (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
MAX is phosphorylated (and activated) by p38, through complex formation.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187

ELK1 is a nuclear p38 target.
PMID:20506250

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="JNK">
         <data key="MY-ID">JNK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8
MAPK9 = mitogen-activated protein kinase 9
MAPK10 = mitogen-activated protein kinase 10

VARIABLE: JNK (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
JNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
ELK1 is a nuclear JNK target.
MEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="p53">
         <data key="MY-ID">p53</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>
         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53

VARIABLE: p53 phosphorylation level

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942

Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.
PMID:17158541

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.
PMID:12217521</node>
      <node id="p21">
         <data key="MY-ID">p21</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not AKT and p53</data>
         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)

VARIABLE: p21 activation level

p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

p21 is transcriptionally induced by the p53 protein.
p21 is a CDK inhibitor.
PMID:17158541

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="TGFBR">
         <data key="MY-ID">TGFBR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR_stimulus</data>
         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1
TGFBR2 = transforming growth factor, beta receptor II (70/80kDa)
TGFBR3 = transforming growth factor, beta receptor III

VARIABLE: TGFBR activation level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta.
PMID:21614932

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</node>
      <node id="EGFR">
         <data key="MY-ID">EGFR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>
         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor

VARIABLE: EGFR activation level

The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</node>
      <node id="FGFR3">
         <data key="MY-ID">FGFR3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>
         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3

VARIABLE: FGFR3 activation level

Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="ATM">
         <data key="MY-ID">ATM</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">DNA_damage</data>
         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated

VARIABLE: ATM phosphorylation level

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942
</node>
      <node id="TAOK">
         <data key="MY-ID">TAOK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ATM</data>
         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1
TAOK2 = TAO kinase 2
TAOK3 = TAO kinase 3

VARIABLE: TAOK (any isoform) phosphorylation level

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</node>
      <node id="MAX">
         <data key="MY-ID">MAX</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X

VARIABLE: MAX phosphorylation level

MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</node>
      <node id="GRB2">
         <data key="MY-ID">GRB2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2

VARIABLE: GRB2 recruitment by activated RTKs

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</node>
      <node id="FRS2">
         <data key="MY-ID">FRS2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>
         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2

VARIABLE: FRS2 phosphorylation level

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</node>
      <node id="PI3K">
         <data key="MY-ID">PI3K</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>
         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide

VARIABLE: PI3K activation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Recruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497</node>
      <node id="AP1">
         <data key="MY-ID">AP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JUN and (FOS or ATF2)</data>
         <data key="MY-SHAPE">ellipse</data>AP-1

VARIABLE: AP-1 formation

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345

AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="PPP2CA">
         <data key="MY-ID">PPP2CA</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme

VARIABLE: PPP2CA activation level

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</node>
      <node id="MEK1_2">
         <data key="MY-ID">MEK1_2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1
MAP2K2 = mitogen-activated protein kinase kinase 2

VARIABLE: MEK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="DUSP1">
         <data key="MY-ID">DUSP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB</data>
         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1

VARIABLE: DUSP1 gene expression level

DUSP1 is a CREB target gene.
PMID:19815709

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</node>
      <node id="MYC">
         <data key="MY-ID">MYC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>
         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)

VARIABLE: MYC activation level (implies gene expression)

Modelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.
MYC inhibits p16, whereas it activates p14.
PMID:17158541

MAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345

Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</node>
      <node id="AKT">
         <data key="MY-ID">AKT</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and not PTEN</data>
         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1
AKT2 = v-akt murine thymoma viral oncogene homolog 2
AKT3 = v-akt murine thymoma viral oncogene homolog 3

VARIABLE: AKT (any isoform) phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186

p21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="PLCG">
         <data key="MY-ID">PLCG</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FGFR3</data>
         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1
PLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)

VARIABLE: PLCG (any isoform) phosphorylation level

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="PKC">
         <data key="MY-ID">PKC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PLCG</data>
         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha
PRKCB = protein kinase C, beta
PRKCG = protein kinase C, gamma

VARIABLE: PKC (any isoform) enzymatic activity

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="GADD45">
         <data key="MY-ID">GADD45</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SMAD or p53</data>
         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha
GADD45B = growth arrest and DNA-damage-inducible, beta
GADD45G = growth arrest and DNA-damage-inducible, gamma

VARIABLE: GADD45 gene (any isoform) expression level

The GADD45 gene is inducible by various environmental stresses.
Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</node>
      <node id="ELK1">
         <data key="MY-ID">ELK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family

VARIABLE: ELK1 phosphorylation level

The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

ELK1 is a nuclear p38 target.
PMID:20506250

ELK1 is a nuclear JNK target.
ELK1 induces c-fos expression.
PMID:11274345</node>
      <node id="FOS">
         <data key="MY-ID">FOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>
         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog

VARIABLE: FOS phosphorylation level

Further observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
ELK1 induces c-fos expression.
PMID:11274345

FOS is a CREB target gene.
PMID:19815709

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</node>
      <node id="ATF2">
         <data key="MY-ID">ATF2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2

VARIABLE: ATF2 phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</node>
      <node id="JUN">
         <data key="MY-ID">JUN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK</data>
         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene

VARIABLE: JUN phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</node>
      <node id="MSK">
         <data key="MY-ID">MSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or p38</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4
RPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5

VARIABLE: MSK (any isoform) phosphorylation level

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</node>
      <node id="CREB">
         <data key="MY-ID">CREB</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MSK</data>
         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1

VARIABLE: CREB phosphorylation level

MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</node>
      <node id="RSK">
         <data key="MY-ID">RSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1
RPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2
RPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3
RPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6

VARIABLE: RSK (any isoform) phosphorylation level

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</node>
      <node id="SMAD">
         <data key="MY-ID">SMAD</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2
SMAD3 = SMAD family member 3
SMAD4 = SMAD family member 4

VARIABLE: SMAD gene (any isoform) expression level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</node>
      <node id="MTK1">
         <data key="MY-ID">MTK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GADD45</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4

VARIABLE: MTK1 phosphorylation level

The delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</node>
      <node id="SPRY">
         <data key="MY-ID">SPRY</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)

VARIABLE: SPRY2 phosphorylation level

Spry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.
PMID:15173823</node>
      <node id="RAF">
         <data key="MY-ID">RAF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>
         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1
BRAF = v-raf murine sarcoma viral oncogene homolog B1
ARAF = v-raf murine sarcoma 3611 viral oncogene homolog

VARIABLE: RAF (any isoform) phosphorylation

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

PKCalpha directly phosphorylates and activates Raf-1.
PMID:8321321

Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</node>
      <node id="GAB1">
         <data key="MY-ID">GAB1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 or PI3K</data>
         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1

VARIABLE: GAB1 phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
PMID:15567848

Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</node>
      <node id="PDK1">
         <data key="MY-ID">PDK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PI3K</data>
         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1

VARIABLE: PDK1 activation level

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</node>
      <node id="p70">
         <data key="MY-ID">p70</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1
RPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2

VARIABLE: p70 (any isoform) phosphorylation level

PDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.
PMID:12040186

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p14">
         <data key="MY-ID">p14</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MYC</data>
         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)

VARIABLE: p14 activation level

MYC inhibits p16, whereas it activates p14.
The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</node>
      <node id="FOXO3">
         <data key="MY-ID">FOXO3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK and not AKT</data>
         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3

VARIABLE: FOXO3 phosphorylation level

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</node>
      <node id="RAS">
         <data key="MY-ID">RAS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SOS or PLCG</data>
         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog
KRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog
NRAS = neuroblastoma RAS viral (v-ras) oncogene homolog

VARIABLE: RAS (any isoform) phosphorylation level

Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Most of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="SOS">
         <data key="MY-ID">SOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 and not RSK</data>
         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)
SOS2 = son of sevenless homolog 2 (Drosophila)

VARIABLE: SOS (any isoform) recruitment by GRB2

Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373
</node>
      <node id="MDM2">
         <data key="MY-ID">MDM2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(p53 or AKT) and not p14</data>
         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)

VARIABLE: MDM2 gene expression level

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.
PMID:17158541

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850
</node>
      <node id="BCL2">
         <data key="MY-ID">BCL2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB and AKT</data>
         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2

VARIABLE: BCL2 activation level (implies gene expression)

Modelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.

DUSP1, FOS, BCL2 are CREB target genes.
PMID:19815709

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="TAK1">
         <data key="MY-ID">TAK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7

VARIABLE: TAK1 phosphorylation level

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="MAP3K1_3">
         <data key="MY-ID">MAP3K1_3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">RAS</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1
MAP3K2 = mitogen-activated protein kinase kinase kinase 2
MAP3K3 = mitogen-activated protein kinase kinase kinase 3

VARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</node>
      <node id="PTEN">
         <data key="MY-ID">PTEN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p53</data>
         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog

VARIABLE: PTEN protein expression level

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="HIF1">
         <data key="MY-ID">HIF1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="EGF">
         <data key="MY-ID">EGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="VEGF">
         <data key="MY-ID">VEGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Oxygen_supply">
         <data key="MY-ID">Oxygen_supply</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Necrosis">
         <data key="MY-ID">Necrosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not Oxygen_supply</data>
         <data key="MY-SHAPE">rect</data>
      </node>
      <edge source="AP1" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</edge>
      <edge source="EGFR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910</edge>
      <edge source="EGFR" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</edge>
      <edge source="Oxygen_supply" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="Oxygen_supply" target="Necrosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MSK" target="CREB" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187</edge>
      <edge source="MSK" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</edge>
      <edge source="MEK1_2" target="ERK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="HIF1" target="EGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="HIF1" target="VEGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="DUSP1" target="p38" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="DUSP1" target="JNK" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="PI3K" target="PDK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124</edge>
      <edge source="PI3K" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="FGFR3" target="FRS2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="FGFR3" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221</edge>
      <edge source="DNA_damage" target="ATM" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</edge>
      <edge source="FRS2" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="MAX" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345</edge>
      <edge source="ATF2" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="ERK" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="ERK" target="RSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952</edge>
      <edge source="ERK" target="SPRY" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823</edge>
      <edge source="ERK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="ERK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ERK" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650
</edge>
      <edge source="ERK" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</edge>
      <edge source="ERK" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</edge>
      <edge source="ERK" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="CREB" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="DUSP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;



DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="JUN" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="FOXO3" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166</edge>
      <edge source="PLCG" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</edge>
      <edge source="PLCG" target="PKC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848</edge>
      <edge source="SOS" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929</edge>
      <edge source="SOS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="p70" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="p70" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MDM2" target="p53" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="TGFBR" target="SMAD" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="TGFBR" target="TAK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TGFBR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</edge>
      <edge source="GAB1" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="MTK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="MTK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="GRB2" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848</edge>
      <edge source="GRB2" target="SOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910</edge>
      <edge source="GRB2" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="GRB2" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848</edge>
      <edge source="GRB2" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="BCL2" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="ATM" target="TAOK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897</edge>
      <edge source="ATM" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942</edge>
      <edge source="GADD45" target="MTK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="PTEN" target="AKT" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="p14" target="MDM2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</edge>
      <edge source="PPP2CA" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="PPP2CA" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="p38" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932</edge>
      <edge source="p38" target="MAX" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</edge>
      <edge source="p38" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.
PMID:20506250
</edge>
      <edge source="p38" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="p53" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</edge>
      <edge source="p53" target="p21" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.
PMID:17158541</edge>
      <edge source="p53" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="p53" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508</edge>
      <edge source="p53" target="PTEN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="SMAD" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="SPRY" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="SPRY" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="EGFR_stimulus" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ELK1" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 induces c-fos expression.
PMID:11274345</edge>
      <edge source="TAK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TAK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="AKT" target="FOXO3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</edge>
      <edge source="AKT" target="p21" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383</edge>
      <edge source="AKT" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="AKT" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186</edge>
      <edge source="AKT" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850</edge>
      <edge source="AKT" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359</edge>
      <edge source="AKT" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="RAF" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="FOS" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="TAOK" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="TAOK" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="JNK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.
PMID:11274345</edge>
      <edge source="JNK" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="JNK" target="JUN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</edge>
      <edge source="JNK" target="FOXO3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971</edge>
      <edge source="p21" target="Proliferation" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="p21" target="Growth_Arrest" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="PKC" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473</edge>
      <edge source="PKC" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321</edge>
      <edge source="PKC" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</edge>
      <edge source="RSK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="RSK" target="SOS" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</edge>
      <edge source="PDK1" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="PDK1" target="AKT" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</edge>
      <edge source="MYC" target="p14" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.
PMID:17158541</edge>
      <edge source="MYC" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</edge>
      <edge source="RAS" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929</edge>
      <edge source="RAS" target="MAP3K1_3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="RAS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.
Proliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.
Input signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>
</graphml>
';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
l[1]=
[{added: 22},
{added: 11},
{added: 8},
{added: 43},
{added: 33},
{added: 26},
{added: 53},
{added: 18},
{added: 42},
{added: 41},
{added: 45},
{added: 46},
{added: 4},
{added: 28},
{added: 37},
{added: 36},
{added: 44},
{added: 5},
{added: 17},
{added: 19},
{added: 30},
{added: 39},
{added: 47},
{added: 49},
{added: 14},
{added: 51},
{added: 29},
{added: 23},
{added: 10},
{added: 50},
{added: 40},
{added: 27},
{added: 20},
{added: 21},
{added: 9},
{added: 24},
{added: 35},
{added: 13},
{added: 34},
{added: 38},
{added: 48},
{added: 31},
{added: 25},
{added: 7},
{added: 6},
{added: 15},
{added: 52},
{added: 16},
{added: 12},
{added: 32},
{x: 2, y: 1, z: 0, who: 37},
{c: 1, x: 2, y: 1, z: 1, who: 24},
{c: 1, x: 0, y: 2, z: 0, who: 18},
{x: 0, y: 0, z: 0, who: 32},
{c: 1, x: 2, y: 2, z: 0, who: 9},
{c: 1, x: 2, y: -1, z: 1, who: 49},
{x: -2, y: 2, z: 0, who: 28},
{c: 1, x: -2, y: 0, z: 0, who: 7},
{c: 1, x: -1, y: 1, z: 1, who: 29},
{c: 1, x: 1, y: -1, z: 0, who: 15},
{x: 1, y: 1, z: 0, who: 10},
{x: -1, y: 0, z: 0, who: 26},
{c: 1, x: -2, y: -2, z: 0, who: 27},
{c: 1, x: -2, y: 0, z: 1, who: 40},
{x: 2, y: -2, z: 0, who: 22},
{x: -2, y: 2, z: 1, who: 34},
{c: 1, x: -2, y: 1, z: 1, who: 8},
{x: 1, y: -2, z: 0, who: 35},
{x: -1, y: -2, z: 0, who: 38},
{c: 1, x: 0, y: 1, z: 1, who: 6},
{c: 1, x: 1, y: 1, z: 1, who: 13},
{x: 0, y: -2, z: 1, who: 43},
{c: 1, x: -2, y: -2, z: 1, who: 42},
{c: 1, x: 2, y: 2, z: 1, who: 36},
{c: 1, x: 2, y: -2, z: 1, who: 19},
{x: 0, y: 1, z: 0, who: 11},
{c: 1, x: 0, y: -1, z: 1, who: 41},
{x: 2, y: 0, z: 1, who: 4},
{x: 1, y: 0, z: 0, who: 12},
{x: 1, y: 2, z: 0, who: 48},
{x: 1, y: 0, z: 1, who: 51},
{c: 1, x: -1, y: 2, z: 0, who: 25},
{x: 1, y: 2, z: 1, who: 52},
{c: 1, x: -1, y: -2, z: 1, who: 5},
{c: 1, x: -1, y: -1, z: 0, who: 39},
{x: 0, y: 0, z: 1, who: 50},
{x: -2, y: -1, z: 1, who: 21},
{c: 1, x: -1, y: -1, z: 1, who: 17},
{x: 2, y: 0, z: 0, who: 31},
{c: 1, x: 0, y: 2, z: 1, who: 46},
{c: 1, x: -2, y: 1, z: 0, who: 14},
{x: -1, y: 2, z: 1, who: 20},
{c: 1, x: -2, y: -1, z: 0, who: 30},
{x: 1, y: -2, z: 1, who: 47},
{x: -1, y: 1, z: 0, who: 23},
{x: 1, y: -1, z: 1, who: 16},
{c: 1, x: 2, y: -1, z: 0, who: 44},
{x: -1, y: 0, z: 1, who: 45},
{c: 1, x: 0, y: -1, z: 0, who: 33},
{x: 0, y: -2, z: 0, who: 53}];
a[1]=
{2: {'TGFBR': {a:0.52,c:0.1},
'p38': {a:0.5,c:0},
'ATM': {a:0.38,c:0.12},
'DUSP1': {a:0.48,c:0},
'RSK': {a:0.52,c:0},
'EGFR': {a:0.6,c:0.14},
'FRS2': {a:0.42,c:0},
'CREB': {a:0.44,c:0},
'PI3K': {a:0.52,c:0},
'PDK1': {a:0.42,c:0},
'MTK1': {a:0.54,c:0},
'TAK1': {a:0.42,c:0},
'p70': {a:0.48,c:0},
'PTEN': {a:0.5,c:0},
'SMAD': {a:0.54,c:0},
'p14': {a:0.46,c:0},
'MEK1_2': {a:0.44,c:0},
'GAB1': {a:0.5,c:0},
'SOS': {a:0.56,c:0},
'RAS': {a:0.3,c:0},
'FGFR3': {a:0.5,c:0.08},
'p21': {a:0.46,c:0},
'ERK': {a:0.32,c:0},
'AKT': {a:0.48,c:0},
'FOS': {a:0.4,c:0},
'PKC': {a:0.5,c:0},
'MSK': {a:0.46,c:0},
'PLCG': {a:0.4,c:0},
'PPP2CA': {a:0.58,c:0},
'GADD45': {a:0.46,c:0},
'HIF1': {a:0.52,c:0.04},
'FOXO3': {a:0.42,c:0},
'MAX': {a:0.48,c:0},
'BCL2': {a:0.42,c:0},
'GRB2': {a:0.46,c:0},
'p53': {a:0.22,c:0},
'ATF2': {a:0.5,c:0},
'MYC': {a:0.4,c:0},
'ELK1': {a:0.5,c:0},
'MAP3K1_3': {a:0.52,c:0},
'AP1': {a:0.48,c:0},
'RAF': {a:0.68,c:0},
'TAOK': {a:0.64,c:0},
'JUN': {a:0.48,c:0},
'SPRY': {a:0.32,c:0},
'JNK': {a:0.48,c:0},
'MDM2': {a:0.5,c:0}}
};
a[1]=
{2: {'RAF': {a:0.68,c:0},
'MDM2': {a:0.5,c:0},
'PKC': {a:0.5,c:0},
'ERK': {a:0.32,c:0},
'p38': {a:0.5,c:0},
'AP1': {a:0.48,c:0},
'p70': {a:0.48,c:0},
'PTEN': {a:0.5,c:0},
'PPP2CA': {a:0.58,c:0},
'ELK1': {a:0.5,c:0},
'FRS2': {a:0.42,c:0},
'TAK1': {a:0.42,c:0},
'p53': {a:0.22,c:0},
'MYC': {a:0.4,c:0},
'EGFR': {a:0.64,c:0.14},
'AKT': {a:0.48,c:0},
'HIF1': {a:0.52,c:0.04},
'CREB': {a:0.44,c:0},
'MTK1': {a:0.54,c:0},
'RSK': {a:0.52,c:0},
'SPRY': {a:0.32,c:0},
'GAB1': {a:0.5,c:0},
'SMAD': {a:0.54,c:0},
'DUSP1': {a:0.48,c:0},
'PI3K': {a:0.52,c:0},
'FOS': {a:0.4,c:0},
'SOS': {a:0.56,c:0},
'TGFBR': {a:0.52,c:0.1},
'TAOK': {a:0.64,c:0},
'p21': {a:0.46,c:0},
'ATF2': {a:0.5,c:0},
'MAP3K1_3': {a:0.52,c:0},
'JNK': {a:0.48,c:0},
'JUN': {a:0.48,c:0},
'GRB2': {a:0.46,c:0},
'FGFR3': {a:0.5,c:0.08},
'MAX': {a:0.48,c:0},
'MSK': {a:0.46,c:0},
'MEK1_2': {a:0.44,c:0},
'PDK1': {a:0.42,c:0},
'PLCG': {a:0.4,c:0},
'GADD45': {a:0.46,c:0},
'RAS': {a:0.3,c:0},
'FOXO3': {a:0.42,c:0},
'BCL2': {a:0.42,c:0},
'p14': {a:0.46,c:0},
'ATM': {a:0.38,c:0.12}}
};
a[2]=
{2: {'AP1': {a:0.48,c:0},
'EGFR': {a:0.64,c:0.16},
'GAB1': {a:0.5,c:0},
'GRB2': {a:0.48,c:0.02},
'FGFR3': {a:0.48,c:0.1},
'ERK': {a:0.32,c:0},
'PDK1': {a:0.42,c:0},
'p70': {a:0.48,c:0},
'MAX': {a:0.48,c:0},
'ATM': {a:0.36,c:0.14},
'TAOK': {a:0.58,c:0.06},
'JNK': {a:0.48,c:0},
'MYC': {a:0.4,c:0},
'MAP3K1_3': {a:0.52,c:0},
'ATF2': {a:0.5,c:0},
'SPRY': {a:0.32,c:0},
'p38': {a:0.5,c:0},
'JUN': {a:0.48,c:0},
'FOXO3': {a:0.42,c:0},
'PLCG': {a:0.42,c:0.06},
'PTEN': {a:0.5,c:0},
'BCL2': {a:0.42,c:0},
'DUSP1': {a:0.48,c:0},
'PKC': {a:0.5,c:0},
'p21': {a:0.46,c:0},
'p53': {a:0.22,c:0.04},
'GADD45': {a:0.46,c:0},
'TAK1': {a:0.38,c:0.04},
'PI3K': {a:0.52,c:0},
'p14': {a:0.46,c:0},
'SMAD': {a:0.48,c:0.06},
'RAS': {a:0.3,c:0},
'PPP2CA': {a:0.58,c:0},
'MSK': {a:0.46,c:0},
'ELK1': {a:0.5,c:0},
'CREB': {a:0.44,c:0},
'MTK1': {a:0.54,c:0},
'FRS2': {a:0.36,c:0.06},
'FOS': {a:0.4,c:0},
'TGFBR': {a:0.5,c:0.12},
'RSK': {a:0.52,c:0},
'MEK1_2': {a:0.44,c:0},
'RAF': {a:0.68,c:0},
'HIF1': {a:0.52,c:0.04},
'MDM2': {a:0.5,c:0},
'AKT': {a:0.48,c:0},
'SOS': {a:0.56,c:0}}
};
a[3]=
{2: {'PTEN': {a:0.5,c:0},
'MSK': {a:0.46,c:0},
'BCL2': {a:0.42,c:0},
'FOS': {a:0.4,c:0},
'RAF': {a:0.68,c:0},
'PLCG': {a:0.42,c:0.06},
'ERK': {a:0.32,c:0},
'AP1': {a:0.48,c:0},
'GRB2': {a:0.48,c:0.02},
'EGFR': {a:0.64,c:0.16},
'HIF1': {a:0.52,c:0.04},
'PDK1': {a:0.42,c:0},
'p53': {a:0.22,c:0.04},
'AKT': {a:0.48,c:0},
'GADD45': {a:0.46,c:0},
'GAB1': {a:0.5,c:0},
'TAOK': {a:0.58,c:0.06},
'MEK1_2': {a:0.44,c:0},
'MYC': {a:0.4,c:0},
'TAK1': {a:0.38,c:0.04},
'MAP3K1_3': {a:0.52,c:0},
'TGFBR': {a:0.5,c:0.12},
'SPRY': {a:0.32,c:0},
'ATF2': {a:0.5,c:0},
'FGFR3': {a:0.48,c:0.1},
'p70': {a:0.48,c:0},
'PPP2CA': {a:0.58,c:0},
'ATM': {a:0.36,c:0.14},
'p21': {a:0.46,c:0},
'JUN': {a:0.48,c:0},
'MTK1': {a:0.54,c:0},
'FOXO3': {a:0.42,c:0},
'DUSP1': {a:0.48,c:0},
'JNK': {a:0.48,c:0},
'FRS2': {a:0.36,c:0.06},
'PKC': {a:0.5,c:0},
'MDM2': {a:0.5,c:0},
'SOS': {a:0.56,c:0},
'RAS': {a:0.3,c:0},
'p14': {a:0.46,c:0},
'MAX': {a:0.48,c:0},
'p38': {a:0.5,c:0},
'ELK1': {a:0.5,c:0},
'CREB': {a:0.44,c:0},
'SMAD': {a:0.48,c:0.06},
'RSK': {a:0.52,c:0},
'PI3K': {a:0.52,c:0}}
};
a[4]=
{1: {'CREB': {a:0.44,c:0},
'PI3K': {a:0.52,c:0},
'ERK': {a:0.32,c:0},
'p21': {a:0.46,c:0},
'BCL2': {a:0.42,c:0},
'ATF2': {a:0.5,c:0},
'GRB2': {a:0.54,c:0.08},
'RSK': {a:0.52,c:0},
'ATM': {a:0.34,c:0.16},
'MAX': {a:0.48,c:0},
'PPP2CA': {a:0.58,c:0},
'GADD45': {a:0.4,c:0.06},
'PKC': {a:0.54,c:0.04},
'PLCG': {a:0.44,c:0.08},
'MYC': {a:0.4,c:0},
'MTK1': {a:0.54,c:0},
'FGFR3': {a:0.46,c:0.12},
'FRS2': {a:0.34,c:0.08},
'TAK1': {a:0.38,c:0.04},
'FOS': {a:0.4,c:0},
'SPRY': {a:0.32,c:0},
'MAP3K1_3': {a:0.52,c:0},
'p38': {a:0.54,c:0.04},
'FOXO3': {a:0.42,c:0},
'EGFR': {a:0.64,c:0.18},
'MDM2': {a:0.48,c:0.02},
'RAS': {a:0.32,c:0.06},
'PDK1': {a:0.42,c:0},
'DUSP1': {a:0.48,c:0},
'PTEN': {a:0.42,c:0.08},
'RAF': {a:0.68,c:0},
'JNK': {a:0.54,c:0.1},
'p70': {a:0.48,c:0},
'TAOK': {a:0.58,c:0.06},
'HIF1': {a:0.52,c:0.04},
'JUN': {a:0.48,c:0},
'SMAD': {a:0.48,c:0.06},
'GAB1': {a:0.5,c:0},
'MEK1_2': {a:0.44,c:0},
'TGFBR': {a:0.5,c:0.12},
'SOS': {a:0.56,c:0},
'ELK1': {a:0.5,c:0},
'p14': {a:0.46,c:0},
'p53': {a:0.18,c:0.06},
'AP1': {a:0.48,c:0},
'AKT': {a:0.48,c:0},
'MSK': {a:0.46,c:0}}
};
a[5]=
{1: {'CREB': {a:0.44,c:0},
'TAOK': {a:0.58,c:0.06},
'RSK': {a:0.52,c:0},
'PI3K': {a:0.52,c:0},
'GADD45': {a:0.4,c:0.06},
'GRB2': {a:0.54,c:0.08},
'SMAD': {a:0.48,c:0.06},
'PPP2CA': {a:0.58,c:0},
'EGFR': {a:0.66,c:0.18},
'p70': {a:0.48,c:0},
'PTEN': {a:0.42,c:0.08},
'BCL2': {a:0.42,c:0},
'ERK': {a:0.32,c:0},
'MAX': {a:0.48,c:0},
'ATF2': {a:0.5,c:0},
'FGFR3': {a:0.46,c:0.12},
'AP1': {a:0.48,c:0},
'ATM': {a:0.34,c:0.16},
'FOS': {a:0.4,c:0},
'MSK': {a:0.46,c:0},
'RAS': {a:0.32,c:0.06},
'PKC': {a:0.54,c:0.04},
'JNK': {a:0.54,c:0.1},
'MYC': {a:0.4,c:0},
'GAB1': {a:0.5,c:0},
'p38': {a:0.54,c:0.04},
'MDM2': {a:0.48,c:0.02},
'p21': {a:0.46,c:0},
'MTK1': {a:0.54,c:0},
'p53': {a:0.18,c:0.06},
'p14': {a:0.46,c:0},
'AKT': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0},
'TGFBR': {a:0.5,c:0.12},
'DUSP1': {a:0.48,c:0},
'FOXO3': {a:0.42,c:0},
'MAP3K1_3': {a:0.52,c:0},
'TAK1': {a:0.38,c:0.04},
'SPRY': {a:0.32,c:0},
'PLCG': {a:0.44,c:0.08},
'ELK1': {a:0.5,c:0},
'PDK1': {a:0.42,c:0},
'RAF': {a:0.68,c:0},
'JUN': {a:0.48,c:0},
'FRS2': {a:0.34,c:0.08},
'HIF1': {a:0.52,c:0.04},
'SOS': {a:0.56,c:0}}
};
a[6]=
{1: {'p70': {a:0.48,c:0},
'CREB': {a:0.44,c:0},
'MAP3K1_3': {a:0.52,c:0.04},
'AP1': {a:0.48,c:0},
'PDK1': {a:0.42,c:0},
'FOXO3': {a:0.4,c:0.02},
'GAB1': {a:0.52,c:0.02},
'MAX': {a:0.48,c:0},
'JUN': {a:0.48,c:0},
'GADD45': {a:0.4,c:0.06},
'PTEN': {a:0.42,c:0.08},
'SMAD': {a:0.48,c:0.06},
'MEK1_2': {a:0.44,c:0},
'TAK1': {a:0.38,c:0.04},
'TAOK': {a:0.56,c:0.08},
'PPP2CA': {a:0.58,c:0},
'p14': {a:0.46,c:0},
'SOS': {a:0.54,c:0.02},
'MYC': {a:0.4,c:0},
'ELK1': {a:0.52,c:0.02},
'FOS': {a:0.4,c:0},
'GRB2': {a:0.56,c:0.1},
'SPRY': {a:0.32,c:0},
'FRS2': {a:0.34,c:0.08},
'TGFBR': {a:0.5,c:0.12},
'MTK1': {a:0.5,c:0.08},
'ATF2': {a:0.54,c:0.04},
'EGFR': {a:0.66,c:0.18},
'PI3K': {a:0.52,c:0.04},
'DUSP1': {a:0.48,c:0},
'FGFR3': {a:0.4,c:0.18},
'p53': {a:0.18,c:0.1},
'p38': {a:0.54,c:0.04},
'ERK': {a:0.32,c:0},
'p21': {a:0.46,c:0},
'PKC': {a:0.54,c:0.04},
'BCL2': {a:0.42,c:0},
'PLCG': {a:0.44,c:0.08},
'HIF1': {a:0.52,c:0.04},
'MDM2': {a:0.48,c:0.02},
'RAF': {a:0.66,c:0.02},
'AKT': {a:0.5,c:0.02},
'ATM': {a:0.34,c:0.16},
'MSK': {a:0.46,c:0},
'JNK': {a:0.54,c:0.1},
'RAS': {a:0.32,c:0.06},
'RSK': {a:0.52,c:0}}
};
a[7]=
{1: {'SMAD': {a:0.48,c:0.06},
'AKT': {a:0.5,c:0.02},
'AP1': {a:0.48,c:0},
'FOS': {a:0.4,c:0},
'p38': {a:0.54,c:0.04},
'p70': {a:0.48,c:0},
'MDM2': {a:0.48,c:0.02},
'MYC': {a:0.4,c:0},
'JNK': {a:0.54,c:0.1},
'TGFBR': {a:0.5,c:0.12},
'FRS2': {a:0.34,c:0.08},
'MAP3K1_3': {a:0.52,c:0.04},
'GADD45': {a:0.4,c:0.06},
'GRB2': {a:0.56,c:0.1},
'FOXO3': {a:0.4,c:0.02},
'PTEN': {a:0.42,c:0.08},
'BCL2': {a:0.42,c:0},
'PDK1': {a:0.42,c:0},
'HIF1': {a:0.52,c:0.04},
'ERK': {a:0.32,c:0},
'RAF': {a:0.66,c:0.02},
'TAOK': {a:0.56,c:0.08},
'EGFR': {a:0.66,c:0.18},
'RAS': {a:0.32,c:0.06},
'PPP2CA': {a:0.58,c:0},
'ATF2': {a:0.54,c:0.04},
'p53': {a:0.18,c:0.1},
'PLCG': {a:0.44,c:0.08},
'MTK1': {a:0.5,c:0.08},
'SOS': {a:0.54,c:0.02},
'ATM': {a:0.34,c:0.16},
'PI3K': {a:0.52,c:0.04},
'FGFR3': {a:0.4,c:0.18},
'p14': {a:0.46,c:0},
'DUSP1': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0},
'TAK1': {a:0.38,c:0.04},
'MSK': {a:0.46,c:0},
'CREB': {a:0.44,c:0},
'GAB1': {a:0.52,c:0.02},
'MAX': {a:0.48,c:0},
'p21': {a:0.46,c:0},
'ELK1': {a:0.52,c:0.02},
'RSK': {a:0.52,c:0},
'PKC': {a:0.54,c:0.04},
'JUN': {a:0.48,c:0},
'SPRY': {a:0.32,c:0}}
};
a[8]=
{2: {'ATM': {a:0.34,c:0.16},
'AP1': {a:0.5,c:0.06},
'FGFR3': {a:0.4,c:0.18},
'p38': {a:0.52,c:0.06},
'MEK1_2': {a:0.44,c:0.08},
'SOS': {a:0.54,c:0.02},
'GAB1': {a:0.52,c:0.02},
'TGFBR': {a:0.5,c:0.12},
'CREB': {a:0.44,c:0},
'FOS': {a:0.4,c:0},
'ERK': {a:0.32,c:0},
'MSK': {a:0.46,c:0},
'PKC': {a:0.52,c:0.06},
'AKT': {a:0.5,c:0.02},
'GRB2': {a:0.56,c:0.1},
'p53': {a:0.16,c:0.1},
'FOXO3': {a:0.4,c:0.02},
'MDM2': {a:0.48,c:0.02},
'BCL2': {a:0.42,c:0},
'MYC': {a:0.44,c:0.04},
'p70': {a:0.48,c:0},
'MAX': {a:0.48,c:0},
'PDK1': {a:0.42,c:0.04},
'SMAD': {a:0.48,c:0.06},
'EGFR': {a:0.64,c:0.2},
'MTK1': {a:0.5,c:0.08},
'TAK1': {a:0.38,c:0.04},
'ATF2': {a:0.54,c:0.04},
'MAP3K1_3': {a:0.52,c:0.04},
'PTEN': {a:0.42,c:0.08},
'p14': {a:0.46,c:0},
'DUSP1': {a:0.48,c:0},
'RAS': {a:0.34,c:0.08},
'p21': {a:0.44,c:0.02},
'HIF1': {a:0.52,c:0.04},
'JUN': {a:0.48,c:0},
'ELK1': {a:0.52,c:0.02},
'PLCG': {a:0.46,c:0.14},
'SPRY': {a:0.32,c:0},
'FRS2': {a:0.32,c:0.1},
'RSK': {a:0.52,c:0},
'TAOK': {a:0.56,c:0.08},
'JNK': {a:0.56,c:0.2},
'PI3K': {a:0.52,c:0.04},
'GADD45': {a:0.4,c:0.06},
'RAF': {a:0.66,c:0.02},
'PPP2CA': {a:0.58,c:0}}
};
a[9]=
{1: {'RSK': {a:0.52,c:0},
'ERK': {a:0.32,c:0},
'MSK': {a:0.46,c:0},
'JUN': {a:0.48,c:0},
'MEK1_2': {a:0.44,c:0.08},
'MAP3K1_3': {a:0.52,c:0.04},
'PKC': {a:0.52,c:0.06},
'SMAD': {a:0.48,c:0.06},
'MTK1': {a:0.5,c:0.08},
'JNK': {a:0.56,c:0.2},
'BCL2': {a:0.42,c:0},
'MAX': {a:0.48,c:0},
'RAF': {a:0.66,c:0.02},
'GADD45': {a:0.4,c:0.06},
'PTEN': {a:0.42,c:0.08},
'MDM2': {a:0.48,c:0.02},
'SOS': {a:0.54,c:0.02},
'CREB': {a:0.44,c:0},
'ELK1': {a:0.52,c:0.02},
'PPP2CA': {a:0.58,c:0},
'p70': {a:0.48,c:0},
'AKT': {a:0.5,c:0.02},
'FOXO3': {a:0.4,c:0.02},
'GAB1': {a:0.52,c:0.02},
'p38': {a:0.52,c:0.06},
'PI3K': {a:0.52,c:0.04},
'ATM': {a:0.34,c:0.16},
'TAK1': {a:0.38,c:0.04},
'PDK1': {a:0.42,c:0.04},
'AP1': {a:0.5,c:0.06},
'DUSP1': {a:0.48,c:0},
'p14': {a:0.46,c:0},
'TGFBR': {a:0.5,c:0.12},
'FRS2': {a:0.32,c:0.1},
'p53': {a:0.16,c:0.1},
'RAS': {a:0.34,c:0.08},
'HIF1': {a:0.52,c:0.04},
'GRB2': {a:0.56,c:0.1},
'ATF2': {a:0.54,c:0.04},
'PLCG': {a:0.46,c:0.14},
'FGFR3': {a:0.4,c:0.18},
'EGFR': {a:0.66,c:0.2},
'TAOK': {a:0.56,c:0.08},
'SPRY': {a:0.32,c:0},
'MYC': {a:0.44,c:0.04},
'FOS': {a:0.4,c:0},
'p21': {a:0.44,c:0.02}}
};
a[10]=
{1: {'FOXO3': {a:0.4,c:0.02},
'MYC': {a:0.44,c:0.04},
'MTK1': {a:0.5,c:0.08},
'ATF2': {a:0.54,c:0.04},
'MEK1_2': {a:0.42,c:0.14},
'p53': {a:0.18,c:0.12},
'CREB': {a:0.44,c:0},
'FRS2': {a:0.32,c:0.1},
'ERK': {a:0.34,c:0.06},
'FGFR3': {a:0.4,c:0.18},
'p70': {a:0.5,c:0.02},
'RSK': {a:0.52,c:0},
'MDM2': {a:0.48,c:0.02},
'ATM': {a:0.34,c:0.16},
'PKC': {a:0.52,c:0.1},
'p14': {a:0.46,c:0.04},
'TGFBR': {a:0.5,c:0.12},
'MSK': {a:0.46,c:0},
'p38': {a:0.52,c:0.06},
'AP1': {a:0.46,c:0.1},
'PDK1': {a:0.4,c:0.06},
'SMAD': {a:0.48,c:0.06},
'AKT': {a:0.5,c:0.02},
'p21': {a:0.44,c:0.02},
'JUN': {a:0.5,c:0.02},
'GADD45': {a:0.4,c:0.06},
'SPRY': {a:0.32,c:0},
'EGFR': {a:0.66,c:0.2},
'BCL2': {a:0.42,c:0},
'GRB2': {a:0.56,c:0.14},
'TAOK': {a:0.56,c:0.08},
'PTEN': {a:0.42,c:0.08},
'PPP2CA': {a:0.58,c:0},
'SOS': {a:0.52,c:0.04},
'PI3K': {a:0.52,c:0.04},
'MAX': {a:0.48,c:0},
'HIF1': {a:0.52,c:0.04},
'RAS': {a:0.36,c:0.1},
'DUSP1': {a:0.5,c:0.02},
'TAK1': {a:0.38,c:0.04},
'RAF': {a:0.64,c:0.04},
'GAB1': {a:0.52,c:0.02},
'ELK1': {a:0.56,c:0.06},
'MAP3K1_3': {a:0.52,c:0.04},
'PLCG': {a:0.46,c:0.18},
'JNK': {a:0.56,c:0.2},
'FOS': {a:0.4,c:0}}
};
replicates[replicate_counter++] = {l: l, s: s, a: a};
l = [];
s = [];
a = [];

minimum_x = Math.min(minimum_x, -2);
maximum_x = Math.max(maximum_x, 2);
minimum_y = Math.min(minimum_y, -2);
maximum_y = Math.max(maximum_y, 2);

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>
<!--
			This GraphML document was generated from GXL by
			a GXL-to-GraphML conversion style sheet.
		-->
<graphml>
   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>
   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>
   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>
   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>
   <key id="MY-VALUE"
        for="node"
        attr.name="MY-VALUE"
        attr.type="string"/>
   <key id="MY-SHAPE"
        for="node"
        attr.name="MY-SHAPE"
        attr.type="string"/>
   <key id="PROBABILITY-OF-LINK"
        for="edge"
        attr.name="PROBABILITY-OF-LINK"
        attr.type="string"/>
   <key id="SIGN-OF-LINK"
        for="edge"
        attr.name="SIGN-OF-LINK"
        attr.type="string"/>
   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>
   <graph id="MAPK_large_HIF1" edgedefault="directed">
      <node id="EGFR_stimulus">
         <data key="MY-ID">EGFR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce EGFR activation
</node>
      <node id="FGFR3_stimulus">
         <data key="MY-ID">FGFR3_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce FGFR3 activation
</node>
      <node id="TGFBR_stimulus">
         <data key="MY-ID">TGFBR_stimulus</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: any stimulus able to induce TGFBR activation

</node>
      <node id="DNA_damage">
         <data key="MY-ID">DNA_damage</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>Model input

VARIABLE: DNA damaging agents/therapies action

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</node>
      <node id="Apoptosis">
         <data key="MY-ID">Apoptosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: apoptosis enablement

Modelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="Growth_Arrest">
         <data key="MY-ID">Growth_Arrest</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p21</data>
         <data key="MY-SHAPE">rect</data>Phenotype

VARIABLE: growth arrest enablement

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="Proliferation">
         <data key="MY-ID">Proliferation</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and MYC and not p21</data>
         <data key="MY-SHAPE">rect</data>Proliferation

VARIABLE: proliferation enablement

Proliferation is an output variable of the model.

Modelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832

</node>
      <node id="ERK">
         <data key="MY-ID">ERK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MEK1_2</data>
         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1
MAPK3 = mitogen-activated protein kinase 3

VARIABLE: ERK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823

Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
RSKs interact with ERK and dissociate upon activation.
PMID:15187187

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p38">
         <data key="MY-ID">p38</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11
MAPK12 = mitogen-activated protein kinase 12
MAPK13 = mitogen-activated protein kinase 13
MAPK14 = mitogen-activated protein kinase 14

VARIABLE: p38 (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
MAX is phosphorylated (and activated) by p38, through complex formation.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187

ELK1 is a nuclear p38 target.
PMID:20506250

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="JNK">
         <data key="MY-ID">JNK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8
MAPK9 = mitogen-activated protein kinase 9
MAPK10 = mitogen-activated protein kinase 10

VARIABLE: JNK (any isoform) phosphorylation level

p38 and JNK are activated after expression of GADD45, following stress, through MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
JNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
ELK1 is a nuclear JNK target.
MEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="p53">
         <data key="MY-ID">p53</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>
         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53

VARIABLE: p53 phosphorylation level

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942

Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.
PMID:17158541

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345

In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.
PMID:12217521</node>
      <node id="p21">
         <data key="MY-ID">p21</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not AKT and p53</data>
         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)

VARIABLE: p21 activation level

p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

p21 is transcriptionally induced by the p53 protein.
p21 is a CDK inhibitor.
PMID:17158541

Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</node>
      <node id="TGFBR">
         <data key="MY-ID">TGFBR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR_stimulus</data>
         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1
TGFBR2 = transforming growth factor, beta receptor II (70/80kDa)
TGFBR3 = transforming growth factor, beta receptor III

VARIABLE: TGFBR activation level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta.
PMID:21614932

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</node>
      <node id="EGFR">
         <data key="MY-ID">EGFR</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>
         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor

VARIABLE: EGFR activation level

The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</node>
      <node id="FGFR3">
         <data key="MY-ID">FGFR3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>
         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3

VARIABLE: FGFR3 activation level

Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="ATM">
         <data key="MY-ID">ATM</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">DNA_damage</data>
         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated

VARIABLE: ATM phosphorylation level

ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942
</node>
      <node id="TAOK">
         <data key="MY-ID">TAOK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ATM</data>
         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1
TAOK2 = TAO kinase 2
TAOK3 = TAO kinase 3

VARIABLE: TAOK (any isoform) phosphorylation level

Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897

TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</node>
      <node id="MAX">
         <data key="MY-ID">MAX</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X

VARIABLE: MAX phosphorylation level

MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</node>
      <node id="GRB2">
         <data key="MY-ID">GRB2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2

VARIABLE: GRB2 recruitment by activated RTKs

Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
Grb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR
PMID:15567848

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436

TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</node>
      <node id="FRS2">
         <data key="MY-ID">FRS2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>
         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2

VARIABLE: FRS2 phosphorylation level

FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289

Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823

FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</node>
      <node id="PI3K">
         <data key="MY-ID">PI3K</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>
         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide

VARIABLE: PI3K activation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Recruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497</node>
      <node id="AP1">
         <data key="MY-ID">AP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JUN and (FOS or ATF2)</data>
         <data key="MY-SHAPE">ellipse</data>AP-1

VARIABLE: AP-1 formation

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345

AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="PPP2CA">
         <data key="MY-ID">PPP2CA</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p38</data>
         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme

VARIABLE: PPP2CA activation level

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</node>
      <node id="MEK1_2">
         <data key="MY-ID">MEK1_2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>
         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1
MAP2K2 = mitogen-activated protein kinase kinase 2

VARIABLE: MEK (any isoform) phosphorylation level

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</node>
      <node id="DUSP1">
         <data key="MY-ID">DUSP1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB</data>
         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1

VARIABLE: DUSP1 gene expression level

DUSP1 is a CREB target gene.
PMID:19815709

DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</node>
      <node id="MYC">
         <data key="MY-ID">MYC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>
         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)

VARIABLE: MYC activation level (implies gene expression)

Modelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.
MYC inhibits p16, whereas it activates p14.
PMID:17158541

MAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345

Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186

MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</node>
      <node id="AKT">
         <data key="MY-ID">AKT</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and not PTEN</data>
         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1
AKT2 = v-akt murine thymoma viral oncogene homolog 2
AKT3 = v-akt murine thymoma viral oncogene homolog 3

VARIABLE: AKT (any isoform) phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to
recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848

Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186

p21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="PLCG">
         <data key="MY-ID">PLCG</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">EGFR or FGFR3</data>
         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1
PLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)

VARIABLE: PLCG (any isoform) phosphorylation level

Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221

Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="PKC">
         <data key="MY-ID">PKC</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PLCG</data>
         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha
PRKCB = protein kinase C, beta
PRKCG = protein kinase C, gamma

VARIABLE: PKC (any isoform) enzymatic activity

Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848

PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473

PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321

(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</node>
      <node id="GADD45">
         <data key="MY-ID">GADD45</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SMAD or p53</data>
         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha
GADD45B = growth arrest and DNA-damage-inducible, beta
GADD45G = growth arrest and DNA-damage-inducible, gamma

VARIABLE: GADD45 gene (any isoform) expression level

The GADD45 gene is inducible by various environmental stresses.
Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932

Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</node>
      <node id="ELK1">
         <data key="MY-ID">ELK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family

VARIABLE: ELK1 phosphorylation level

The rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.
PMID:16393692

ELK1 is a nuclear p38 target.
PMID:20506250

ELK1 is a nuclear JNK target.
ELK1 induces c-fos expression.
PMID:11274345</node>
      <node id="FOS">
         <data key="MY-ID">FOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>
         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog

VARIABLE: FOS phosphorylation level

Further observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
ELK1 induces c-fos expression.
PMID:11274345

FOS is a CREB target gene.
PMID:19815709

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</node>
      <node id="ATF2">
         <data key="MY-ID">ATF2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK or p38</data>
         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2

VARIABLE: ATF2 phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</node>
      <node id="JUN">
         <data key="MY-ID">JUN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK</data>
         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene

VARIABLE: JUN phosphorylation level

AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</node>
      <node id="MSK">
         <data key="MY-ID">MSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK or p38</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4
RPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5

VARIABLE: MSK (any isoform) phosphorylation level

MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</node>
      <node id="CREB">
         <data key="MY-ID">CREB</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MSK</data>
         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1

VARIABLE: CREB phosphorylation level

MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187

DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</node>
      <node id="RSK">
         <data key="MY-ID">RSK</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1
RPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2
RPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3
RPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6

VARIABLE: RSK (any isoform) phosphorylation level

RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952

Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</node>
      <node id="SMAD">
         <data key="MY-ID">SMAD</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2
SMAD3 = SMAD family member 3
SMAD4 = SMAD family member 4

VARIABLE: SMAD gene (any isoform) expression level

Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</node>
      <node id="MTK1">
         <data key="MY-ID">MTK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GADD45</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4

VARIABLE: MTK1 phosphorylation level

The delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.
Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</node>
      <node id="SPRY">
         <data key="MY-ID">SPRY</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">ERK</data>
         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)

VARIABLE: SPRY2 phosphorylation level

Spry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.
PMID:15173823</node>
      <node id="RAF">
         <data key="MY-ID">RAF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>
         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1
BRAF = v-raf murine sarcoma viral oncogene homolog B1
ARAF = v-raf murine sarcoma 3611 viral oncogene homolog

VARIABLE: RAF (any isoform) phosphorylation

Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910

PKCalpha directly phosphorylates and activates Raf-1.
PMID:8321321

Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359

B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</node>
      <node id="GAB1">
         <data key="MY-ID">GAB1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 or PI3K</data>
         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1

VARIABLE: GAB1 phosphorylation level

Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
PMID:15567848

Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</node>
      <node id="PDK1">
         <data key="MY-ID">PDK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PI3K</data>
         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1

VARIABLE: PDK1 activation level

PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</node>
      <node id="p70">
         <data key="MY-ID">p70</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">PDK1 and ERK</data>
         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1
RPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2

VARIABLE: p70 (any isoform) phosphorylation level

PDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.
PMID:12040186

p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</node>
      <node id="p14">
         <data key="MY-ID">p14</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">MYC</data>
         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)

VARIABLE: p14 activation level

MYC inhibits p16, whereas it activates p14.
The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</node>
      <node id="FOXO3">
         <data key="MY-ID">FOXO3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">JNK and not AKT</data>
         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3

VARIABLE: FOXO3 phosphorylation level

JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971

FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166

Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</node>
      <node id="RAS">
         <data key="MY-ID">RAS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">SOS or PLCG</data>
         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog
KRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog
NRAS = neuroblastoma RAS viral (v-ras) oncogene homolog

VARIABLE: RAS (any isoform) phosphorylation level

Grb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
Most of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</node>
      <node id="SOS">
         <data key="MY-ID">SOS</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">GRB2 and not RSK</data>
         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)
SOS2 = son of sevenless homolog 2 (Drosophila)

VARIABLE: SOS (any isoform) recruitment by GRB2

Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910

The signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.
Grb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.
The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929

PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497

p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373
</node>
      <node id="MDM2">
         <data key="MY-ID">MDM2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">(p53 or AKT) and not p14</data>
         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)

VARIABLE: MDM2 gene expression level

The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.
PMID:17158541

Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850
</node>
      <node id="BCL2">
         <data key="MY-ID">BCL2</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">CREB and AKT</data>
         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2

VARIABLE: BCL2 activation level (implies gene expression)

Modelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.

DUSP1, FOS, BCL2 are CREB target genes.
PMID:19815709

Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186
</node>
      <node id="TAK1">
         <data key="MY-ID">TAK1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">TGFBR</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7

VARIABLE: TAK1 phosphorylation level

Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</node>
      <node id="MAP3K1_3">
         <data key="MY-ID">MAP3K1_3</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">RAS</data>
         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1
MAP3K2 = mitogen-activated protein kinase kinase kinase 2
MAP3K3 = mitogen-activated protein kinase kinase kinase 3

VARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level

The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</node>
      <node id="PTEN">
         <data key="MY-ID">PTEN</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p53</data>
         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog

VARIABLE: PTEN protein expression level

p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</node>
      <node id="HIF1">
         <data key="MY-ID">HIF1</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="EGF">
         <data key="MY-ID">EGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="VEGF">
         <data key="MY-ID">VEGF</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">HIF1</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Oxygen_supply">
         <data key="MY-ID">Oxygen_supply</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-SHAPE">ellipse</data>
      </node>
      <node id="Necrosis">
         <data key="MY-ID">Necrosis</data>
         <data key="BREED">objects</data>
         <data key="KIND">Gene</data>
         <data key="MY-VALUE">1</data>
         <data key="MY-RULE">not Oxygen_supply</data>
         <data key="MY-SHAPE">rect</data>
      </node>
      <edge source="AP1" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.
PMID:18039929</edge>
      <edge source="EGFR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.
PMID:17496910</edge>
      <edge source="EGFR" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.
PMID:2472219</edge>
      <edge source="Oxygen_supply" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="Oxygen_supply" target="Necrosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MSK" target="CREB" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.
PMID:15187187</edge>
      <edge source="MSK" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.
PMID:17158541</edge>
      <edge source="MEK1_2" target="ERK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="HIF1" target="EGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="HIF1" target="VEGF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="DUSP1" target="p38" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="DUSP1" target="JNK" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.
PMID:19436832</edge>
      <edge source="PI3K" target="PDK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.
PMID:15199124</edge>
      <edge source="PI3K" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="FGFR3" target="FRS2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="FGFR3" target="PLCG" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.
PMID:1656221</edge>
      <edge source="DNA_damage" target="ATM" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.
PMID:18235226</edge>
      <edge source="FRS2" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.
PMID:11447289</edge>
      <edge source="MAX" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.
PMID:11274345</edge>
      <edge source="ATF2" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="ERK" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="ERK" target="RSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.
PMID:15239952</edge>
      <edge source="ERK" target="SPRY" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.
PMID:15173823</edge>
      <edge source="ERK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="ERK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ERK" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.
PMID:12792650
</edge>
      <edge source="ERK" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.
PMID:11940578, PMID:10601235, PMID:11431469</edge>
      <edge source="ERK" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.
PMID:19933846</edge>
      <edge source="ERK" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="CREB" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="DUSP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;



DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="CREB" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.
PMID:19815709</edge>
      <edge source="JUN" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="FOXO3" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.
PMID:20978166</edge>
      <edge source="PLCG" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.
PMID:17496910

Calcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.
PMID:16488589</edge>
      <edge source="PLCG" target="PKC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.
PMID:15567848</edge>
      <edge source="SOS" target="RAS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).
PMID:18039929</edge>
      <edge source="SOS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="p70" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="p70" target="HIF1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="MDM2" target="p53" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="TGFBR" target="SMAD" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="TGFBR" target="TAK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TGFBR" target="GRB2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.
PMID:17673906</edge>
      <edge source="GAB1" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.
PMID:15199124</edge>
      <edge source="MTK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="MTK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="GRB2" target="GAB1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.
EGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.
PMID:15567848</edge>
      <edge source="GRB2" target="SOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.
PMID:17496910</edge>
      <edge source="GRB2" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="GRB2" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR
PMID:15567848</edge>
      <edge source="GRB2" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).
PMID:11997436</edge>
      <edge source="BCL2" target="Apoptosis" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="ATM" target="TAOK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.
PMID:18855897</edge>
      <edge source="ATM" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.
PMID:15140942</edge>
      <edge source="GADD45" target="MTK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.
PMID:21614932</edge>
      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="PTEN" target="AKT" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="p14" target="MDM2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.
PMID:17158541</edge>
      <edge source="PPP2CA" target="MEK1_2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="PPP2CA" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.
PMID:18039929</edge>
      <edge source="p38" target="MSK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.
PMID:15187187</edge>
      <edge source="p38" target="p53" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.
PMID:21614932</edge>
      <edge source="p38" target="MAX" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.
PMID:11274345</edge>
      <edge source="p38" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.
PMID:20506250
</edge>
      <edge source="p38" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="p53" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.
PMID:11274345</edge>
      <edge source="p53" target="p21" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.
PMID:17158541</edge>
      <edge source="p53" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.
PMID:17158541</edge>
      <edge source="p53" target="Apoptosis" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).
PMID:19641508</edge>
      <edge source="p53" target="PTEN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.
PMID:12217521</edge>
      <edge source="SMAD" target="GADD45" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.
PMID:21614932</edge>
      <edge source="SPRY" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="SPRY" target="FRS2" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).
PMID:15173823</edge>
      <edge source="EGFR_stimulus" target="EGFR" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="ELK1" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 induces c-fos expression.
PMID:11274345</edge>
      <edge source="TAK1" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="TAK1" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.
PMID:20060931</edge>
      <edge source="AKT" target="FOXO3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.
PMID:12040186</edge>
      <edge source="AKT" target="p21" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.
PMID:11882383</edge>
      <edge source="AKT" target="BCL2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.
PMID:12040186</edge>
      <edge source="AKT" target="MYC" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.
PMID:12040186</edge>
      <edge source="AKT" target="MDM2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.
PMID:11850850</edge>
      <edge source="AKT" target="RAF" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.
PMID:10869359</edge>
      <edge source="AKT" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>
      </edge>
      <edge source="RAF" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.
PMID:17496910</edge>
      <edge source="FOS" target="AP1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.
PMID:11274345</edge>
      <edge source="TAOK" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="TAOK" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.
PMID:21614932</edge>
      <edge source="JNK" target="ELK1" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.
PMID:11274345</edge>
      <edge source="JNK" target="ATF2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.
PMID:11274345</edge>
      <edge source="JNK" target="JUN" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.
PMID:11274345</edge>
      <edge source="JNK" target="FOXO3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.
PMID:20028971</edge>
      <edge source="p21" target="Proliferation" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="p21" target="Growth_Arrest" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.
PMID:8614832</edge>
      <edge source="PKC" target="EGFR" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.
PMID:6321473</edge>
      <edge source="PKC" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.
PMID:8321321</edge>
      <edge source="PKC" target="FGFR3" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.
PMID:9694798, PMID:8622701, PMID:14699054</edge>
      <edge source="RSK" target="FOS" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.
PMID:16393692</edge>
      <edge source="RSK" target="SOS" sign="negative">
         <data key="SIGN-OF-LINK">negative</data>
         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.
PMID:9242373</edge>
      <edge source="PDK1" target="p70" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.
PMID:12040186</edge>
      <edge source="PDK1" target="AKT" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.
PMID:12040186</edge>
      <edge source="MYC" target="p14" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.
PMID:17158541</edge>
      <edge source="MYC" target="Proliferation" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.
PMID:11805123</edge>
      <edge source="RAS" target="RAF" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).
PMID:18039929</edge>
      <edge source="RAS" target="MAP3K1_3" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="RAS" target="PI3K" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.
PMID:21779497
</edge>
      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="JNK" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.
MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>
      <edge source="MAP3K1_3" target="p38" sign="positive">
         <data key="SIGN-OF-LINK">positive</data>
         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.
PMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.
Proliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.
Input signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>
</graphml>
';
parameters = ['the-number-of-replicates', 4.0,'the-maximum-number-of-ticks', 10.0,'the-maximum-cell-count', 300.0,'the-initial-cell-count', 50.0,'the-egf-diffusion-coefficient', 2.7E-14,'the-oxygen-diffusion-coefficient', 1.0E-9,'the-boundary-oxygen', 50.0,'the-initial-oxygen', 0.08,'Oxygen-consumption-rate', 5.0E-4,'Oxygen_supply-activation-threshold', 0.08,'EGFR_stimulus-activation-threshold', 0.01,'EGF-consumption-rate', 0.01,'EGF-production-rate', 1.0,'the-boundary-EGF', 0.0,'the-initial-egf', 0.0,'only-one-downstream-change', true];
start_time = '11:41:56.694 AM 19-Aug-2016';
number_of_replicates_requested = 4;
default_size = 1;
default_colors = ['rgba(223, 91, 84, 0.71)', 'rgba(164, 164, 164, 0.71)'];
running_3D = true;

mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
gene_nodes[2] = [
{id: 1628, label: "PLCG"},
{id: 1627, label: "ERK"},
{id: 1663, label: "SOS"},
{id: 1630, label: "SMAD"},
{id: 1652, label: "FOXO3"},
{id: 1637, label: "CREB"},
{id: 1670, label: "FRS2"},
{id: 1624, label: "RAF"},
{id: 1668, label: "DUSP1"},
{id: 1641, label: "ATM"},
{id: 1664, label: "TGFBR"},
{id: 1625, label: "MSK"},
{id: 1632, label: "RAS"},
{id: 1649, label: "PDK1"},
{id: 1645, label: "p38"},
{id: 1680, label: "MAX"},
{id: 1655, label: "SPRY"},
{id: 1666, label: "JNK"},
{id: 1631, label: "ATF2"},
{id: 1658, label: "MDM2"},
{id: 1677, label: "PPP2CA"},
{id: 1642, label: "BCL2"},
{id: 1671, label: "TAOK"},
{id: 1648, label: "p21"},
{id: 1676, label: "FGFR3"},
{id: 1634, label: "p53"},
{id: 1640, label: "TAK1"},
{id: 1669, label: "ELK1"},
{id: 1639, label: "MEK1_2"},
{id: 1647, label: "EGFR"},
{id: 1679, label: "PTEN"},
{id: 1635, label: "AKT"},
{id: 1657, label: "GAB1"},
{id: 1644, label: "MYC"},
{id: 1650, label: "HIF1"},
{id: 1675, label: "JUN"},
{id: 1665, label: "PI3K"},
{id: 1651, label: "GADD45"},
{id: 1643, label: "GRB2"},
{id: 1623, label: "FOS"},
{id: 1656, label: "AP1"},
{id: 1626, label: "RSK"},
{id: 1672, label: "p70"},
{id: 1653, label: "PKC"},
{id: 1662, label: "MTK1"},
{id: 1633, label: "MAP3K1_3"},
{id: 1667, label: "p14"}
];
input_nodes[2] = [
{id: 1674, label: "FGFR3_stimulus"},
{id: 1659, label: "EGFR_stimulus"},
{id: 1654, label: "TGFBR_stimulus"},
{id: 1636, label: "DNA_damage"},
{id: 1660, label: "Oxygen_supply"}
];
output_nodes[2] = [
{id: 1646, label: "EGF"},
{id: 1638, label: "VEGF"}
];
links_between_nodes[2] = [
{from: 1667, to: 1658, color: "red"},
{from: 1648, to: 1673, color: "green"},
{from: 1662, to: 1666, color: "green"},
{from: 1637, to: 1623, color: "green"},
{from: 1666, to: 1652, color: "green"},
{from: 1643, to: 1676, color: "red"},
{from: 1627, to: 1650, color: "green"},
{from: 1659, to: 1647, color: "green"},
{from: 1648, to: 1661, color: "red"},
{from: 1652, to: 1629, color: "green"},
{from: 1637, to: 1642, color: "green"},
{from: 1641, to: 1671, color: "green"},
{from: 1643, to: 1663, color: "green"},
{from: 1626, to: 1623, color: "green"},
{from: 1625, to: 1644, color: "green"},
{from: 1635, to: 1644, color: "green"},
{from: 1634, to: 1679, color: "green"},
{from: 1645, to: 1680, color: "green"},
{from: 1635, to: 1658, color: "green"},
{from: 1643, to: 1647, color: "red"},
{from: 1655, to: 1647, color: "green"},
{from: 1628, to: 1653, color: "green"},
{from: 1634, to: 1648, color: "green"},
{from: 1666, to: 1631, color: "green"},
{from: 1666, to: 1675, color: "green"},
{from: 1672, to: 1650, color: "green"},
{from: 1639, to: 1627, color: "green"},
{from: 1624, to: 1639, color: "green"},
{from: 1642, to: 1629, color: "red"},
{from: 1627, to: 1669, color: "green"},
{from: 1660, to: 1678, color: "red"},
{from: 1627, to: 1625, color: "green"},
{from: 1632, to: 1665, color: "green"},
{from: 1645, to: 1625, color: "green"},
{from: 1653, to: 1624, color: "green"},
{from: 1662, to: 1645, color: "green"},
{from: 1672, to: 1661, color: "green"},
{from: 1632, to: 1633, color: "green"},
{from: 1645, to: 1669, color: "green"},
{from: 1670, to: 1643, color: "green"},
{from: 1658, to: 1634, color: "red"},
{from: 1643, to: 1657, color: "green"},
{from: 1669, to: 1623, color: "green"},
{from: 1650, to: 1638, color: "green"},
{from: 1625, to: 1637, color: "green"},
{from: 1649, to: 1672, color: "green"},
{from: 1647, to: 1628, color: "green"},
{from: 1633, to: 1666, color: "green"},
{from: 1635, to: 1672, color: "green"},
{from: 1654, to: 1664, color: "green"},
{from: 1635, to: 1652, color: "red"},
{from: 1627, to: 1629, color: "red"},
{from: 1674, to: 1676, color: "green"},
{from: 1644, to: 1667, color: "green"},
{from: 1665, to: 1657, color: "green"},
{from: 1655, to: 1670, color: "red"},
{from: 1650, to: 1646, color: "green"},
{from: 1647, to: 1643, color: "green"},
{from: 1645, to: 1634, color: "green"},
{from: 1634, to: 1658, color: "green"},
{from: 1676, to: 1670, color: "green"},
{from: 1651, to: 1662, color: "green"},
{from: 1634, to: 1629, color: "green"},
{from: 1668, to: 1645, color: "red"},
{from: 1671, to: 1666, color: "green"},
{from: 1677, to: 1639, color: "red"},
{from: 1627, to: 1624, color: "red"},
{from: 1627, to: 1623, color: "green"},
{from: 1656, to: 1639, color: "red"},
{from: 1649, to: 1635, color: "green"},
{from: 1637, to: 1668, color: "green"},
{from: 1635, to: 1624, color: "red"},
{from: 1664, to: 1630, color: "green"},
{from: 1663, to: 1665, color: "green"},
{from: 1626, to: 1663, color: "red"},
{from: 1663, to: 1632, color: "green"},
{from: 1679, to: 1635, color: "red"},
{from: 1632, to: 1624, color: "green"},
{from: 1645, to: 1677, color: "green"},
{from: 1630, to: 1651, color: "green"},
{from: 1653, to: 1647, color: "red"},
{from: 1668, to: 1666, color: "red"},
{from: 1627, to: 1626, color: "green"},
{from: 1657, to: 1665, color: "green"},
{from: 1665, to: 1649, color: "green"},
{from: 1627, to: 1672, color: "green"},
{from: 1641, to: 1634, color: "green"},
{from: 1660, to: 1650, color: "green"},
{from: 1627, to: 1655, color: "green"},
{from: 1676, to: 1628, color: "green"},
{from: 1653, to: 1676, color: "red"},
{from: 1623, to: 1656, color: "green"},
{from: 1645, to: 1631, color: "green"},
{from: 1640, to: 1645, color: "green"},
{from: 1628, to: 1632, color: "green"},
{from: 1633, to: 1639, color: "green"},
{from: 1631, to: 1656, color: "green"},
{from: 1644, to: 1661, color: "green"},
{from: 1635, to: 1642, color: "green"},
{from: 1664, to: 1643, color: "green"},
{from: 1675, to: 1656, color: "green"},
{from: 1634, to: 1651, color: "green"},
{from: 1633, to: 1645, color: "green"},
{from: 1666, to: 1669, color: "green"},
{from: 1664, to: 1640, color: "green"},
{from: 1640, to: 1666, color: "green"},
{from: 1643, to: 1670, color: "red"},
{from: 1635, to: 1648, color: "red"},
{from: 1636, to: 1641, color: "green"},
{from: 1671, to: 1645, color: "green"},
{from: 1680, to: 1644, color: "green"}
];
gene_nodes[1] = [
{id: 2382, label: "SPRY"},
{id: 2381, label: "ATM"},
{id: 2380, label: "PLCG"},
{id: 2403, label: "PKC"},
{id: 2419, label: "TGFBR"},
{id: 2434, label: "CREB"},
{id: 2378, label: "FRS2"},
{id: 2430, label: "PPP2CA"},
{id: 2379, label: "ERK"},
{id: 2388, label: "HIF1"},
{id: 2401, label: "p53"},
{id: 2390, label: "MYC"},
{id: 2400, label: "RAF"},
{id: 2423, label: "EGFR"},
{id: 2404, label: "MAP3K1_3"},
{id: 2412, label: "PI3K"},
{id: 2415, label: "TAOK"},
{id: 2409, label: "TAK1"},
{id: 2385, label: "BCL2"},
{id: 2427, label: "MAX"},
{id: 2395, label: "GADD45"},
{id: 2433, label: "ATF2"},
{id: 2422, label: "GRB2"},
{id: 2417, label: "RAS"},
{id: 2384, label: "JNK"},
{id: 2424, label: "GAB1"},
{id: 2414, label: "MDM2"},
{id: 2392, label: "PTEN"},
{id: 2432, label: "FOS"},
{id: 2410, label: "AP1"},
{id: 2426, label: "DUSP1"},
{id: 2406, label: "FOXO3"},
{id: 2398, label: "FGFR3"},
{id: 2397, label: "p38"},
{id: 2408, label: "p21"},
{id: 2428, label: "ELK1"},
{id: 2413, label: "PDK1"},
{id: 2416, label: "MSK"},
{id: 2431, label: "p14"},
{id: 2396, label: "SMAD"},
{id: 2386, label: "SOS"},
{id: 2411, label: "JUN"},
{id: 2425, label: "p70"},
{id: 2421, label: "MTK1"},
{id: 2420, label: "AKT"},
{id: 2393, label: "RSK"},
{id: 2394, label: "MEK1_2"}
];
input_nodes[1] = [
{id: 2399, label: "Oxygen_supply"},
{id: 2377, label: "EGFR_stimulus"},
{id: 2405, label: "FGFR3_stimulus"},
{id: 2402, label: "TGFBR_stimulus"},
{id: 2407, label: "DNA_damage"}
];
output_nodes[1] = [
{id: 2383, label: "VEGF"},
{id: 2418, label: "EGF"}
];
links_between_nodes[1] = [
{from: 2396, to: 2395, color: "green"},
{from: 2386, to: 2417, color: "green"},
{from: 2434, to: 2385, color: "green"},
{from: 2420, to: 2385, color: "green"},
{from: 2413, to: 2420, color: "green"},
{from: 2379, to: 2388, color: "green"},
{from: 2403, to: 2423, color: "red"},
{from: 2397, to: 2430, color: "green"},
{from: 2400, to: 2394, color: "green"},
{from: 2420, to: 2406, color: "red"},
{from: 2423, to: 2380, color: "green"},
{from: 2421, to: 2384, color: "green"},
{from: 2397, to: 2401, color: "green"},
{from: 2398, to: 2380, color: "green"},
{from: 2379, to: 2432, color: "green"},
{from: 2401, to: 2414, color: "green"},
{from: 2421, to: 2397, color: "green"},
{from: 2390, to: 2431, color: "green"},
{from: 2432, to: 2410, color: "green"},
{from: 2424, to: 2412, color: "green"},
{from: 2382, to: 2378, color: "red"},
{from: 2407, to: 2381, color: "green"},
{from: 2398, to: 2378, color: "green"},
{from: 2420, to: 2408, color: "red"},
{from: 2404, to: 2384, color: "green"},
{from: 2379, to: 2416, color: "green"},
{from: 2379, to: 2428, color: "green"},
{from: 2408, to: 2389, color: "red"},
{from: 2379, to: 2382, color: "green"},
{from: 2420, to: 2390, color: "green"},
{from: 2397, to: 2428, color: "green"},
{from: 2377, to: 2423, color: "green"},
{from: 2414, to: 2401, color: "red"},
{from: 2393, to: 2386, color: "red"},
{from: 2385, to: 2391, color: "red"},
{from: 2420, to: 2414, color: "green"},
{from: 2397, to: 2427, color: "green"},
{from: 2410, to: 2394, color: "red"},
{from: 2403, to: 2400, color: "green"},
{from: 2381, to: 2401, color: "green"},
{from: 2394, to: 2379, color: "green"},
{from: 2420, to: 2425, color: "green"},
{from: 2409, to: 2397, color: "green"},
{from: 2422, to: 2398, color: "red"},
{from: 2401, to: 2395, color: "green"},
{from: 2412, to: 2413, color: "green"},
{from: 2399, to: 2429, color: "red"},
{from: 2433, to: 2410, color: "green"},
{from: 2423, to: 2422, color: "green"},
{from: 2434, to: 2432, color: "green"},
{from: 2388, to: 2383, color: "green"},
{from: 2397, to: 2433, color: "green"},
{from: 2419, to: 2409, color: "green"},
{from: 2428, to: 2432, color: "green"},
{from: 2384, to: 2411, color: "green"},
{from: 2384, to: 2428, color: "green"},
{from: 2380, to: 2417, color: "green"},
{from: 2379, to: 2400, color: "red"},
{from: 2401, to: 2408, color: "green"},
{from: 2404, to: 2394, color: "green"},
{from: 2399, to: 2388, color: "green"},
{from: 2406, to: 2391, color: "green"},
{from: 2388, to: 2418, color: "green"},
{from: 2420, to: 2400, color: "red"},
{from: 2393, to: 2432, color: "green"},
{from: 2402, to: 2419, color: "green"},
{from: 2422, to: 2424, color: "green"},
{from: 2378, to: 2422, color: "green"},
{from: 2422, to: 2386, color: "green"},
{from: 2422, to: 2378, color: "red"},
{from: 2386, to: 2412, color: "green"},
{from: 2379, to: 2391, color: "red"},
{from: 2390, to: 2389, color: "green"},
{from: 2404, to: 2397, color: "green"},
{from: 2415, to: 2397, color: "green"},
{from: 2379, to: 2425, color: "green"},
{from: 2379, to: 2393, color: "green"},
{from: 2401, to: 2392, color: "green"},
{from: 2434, to: 2426, color: "green"},
{from: 2403, to: 2398, color: "red"},
{from: 2426, to: 2384, color: "red"},
{from: 2425, to: 2388, color: "green"},
{from: 2409, to: 2384, color: "green"},
{from: 2422, to: 2423, color: "red"},
{from: 2426, to: 2397, color: "red"},
{from: 2384, to: 2406, color: "green"},
{from: 2416, to: 2434, color: "green"},
{from: 2412, to: 2424, color: "green"},
{from: 2430, to: 2394, color: "red"},
{from: 2419, to: 2396, color: "green"},
{from: 2427, to: 2390, color: "green"},
{from: 2431, to: 2414, color: "red"},
{from: 2395, to: 2421, color: "green"},
{from: 2401, to: 2391, color: "green"},
{from: 2425, to: 2389, color: "green"},
{from: 2380, to: 2403, color: "green"},
{from: 2417, to: 2400, color: "green"},
{from: 2381, to: 2415, color: "green"},
{from: 2415, to: 2384, color: "green"},
{from: 2392, to: 2420, color: "red"},
{from: 2397, to: 2416, color: "green"},
{from: 2417, to: 2404, color: "green"},
{from: 2416, to: 2390, color: "green"},
{from: 2411, to: 2410, color: "green"},
{from: 2405, to: 2398, color: "green"},
{from: 2408, to: 2387, color: "green"},
{from: 2419, to: 2422, color: "green"},
{from: 2417, to: 2412, color: "green"},
{from: 2413, to: 2425, color: "green"},
{from: 2382, to: 2423, color: "green"},
{from: 2384, to: 2433, color: "green"}
];
</script>
</head>
<body>
</body>
</html>

