<!DOCTYPE html>
<html>
<head>
<meta charset='utf-8' />
<link rel='stylesheet' media='all' href='../libs/cancer-model.css' />
<link href='../libs/vis/vis.css' rel='stylesheet' type='text/css' />
<script type='text/javascript' src='../libs/three.js'></script>
<script type='text/javascript' src='../libs/dat.gui.js'></script>
<script type='text/javascript' src='../libs/stats.js'></script>
<script type='text/javascript' src='../libs/OrbitControls.js'></script>
<script type='text/javascript' src='../libs/rcolor.js'></script>
<script src='../libs/vis/vis.js'></script>
<script src='../libs/cancer-model.js'></script>
<!-- Plotly.js -->
<script src='https://cdn.plot.ly/plotly-latest.min.js'></script>
<title>Results from running cancer simulation</title>
<script>
var l = [];
var s = [];
var replicates = [];
var a = [];
var number_of_replicates_requested = 50; // added manually
var start_time = '1 Jan 2000 midnight'; // added manually
var running_3D = true;
var replicate_counter = 0;
var gene_nodes = [];
var input_nodes = [];
var output_nodes = [];
var links_between_nodes = [];var minimum_x = Number.MAX_VALUE;
var maximum_x = Number.MIN_VALUE;
var minimum_y = Number.MAX_VALUE;
var maximum_y = Number.MIN_VALUE;
var parameters,
    number_of_replicates_requested,
    mutations_file_contents;

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>\n<!--\n			This GraphML document was generated from GXL by\n			a GXL-to-GraphML conversion style sheet.\n		-->\n<graphml>\n   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>\n   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>\n   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>\n   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>\n   <key id="MY-VALUE"\n        for="node"\n        attr.name="MY-VALUE"\n        attr.type="string"/>\n   <key id="MY-SHAPE"\n        for="node"\n        attr.name="MY-SHAPE"\n        attr.type="string"/>\n   <key id="PROBABILITY-OF-LINK"\n        for="edge"\n        attr.name="PROBABILITY-OF-LINK"\n        attr.type="string"/>\n   <key id="SIGN-OF-LINK"\n        for="edge"\n        attr.name="SIGN-OF-LINK"\n        attr.type="string"/>\n   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>\n   <graph id="MAPK_large_HIF1" edgedefault="directed">\n      <node id="EGFR_stimulus">\n         <data key="MY-ID">EGFR_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce EGFR activation\n</node>\n      <node id="FGFR3_stimulus">\n         <data key="MY-ID">FGFR3_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce FGFR3 activation\n</node>\n      <node id="TGFBR_stimulus">\n         <data key="MY-ID">TGFBR_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce TGFBR activation\n\n</node>\n      <node id="DNA_damage">\n         <data key="MY-ID">DNA_damage</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: DNA damaging agents/therapies action\n\nATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226</node>\n      <node id="Apoptosis">\n         <data key="MY-ID">Apoptosis</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>\n         <data key="MY-SHAPE">rect</data>Phenotype\n\nVARIABLE: apoptosis enablement\n\nModelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).\n\nFOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166\n\nIn response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508\n\nCaspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186\n</node>\n      <node id="Growth_Arrest">\n         <data key="MY-ID">Growth_Arrest</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p21</data>\n         <data key="MY-SHAPE">rect</data>Phenotype\n\nVARIABLE: growth arrest enablement\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</node>\n      <node id="Proliferation">\n         <data key="MY-ID">Proliferation</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p70 and MYC and not p21</data>\n         <data key="MY-SHAPE">rect</data>Proliferation\n\nVARIABLE: proliferation enablement\n\nProliferation is an output variable of the model.\n\nModelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive\n\nPDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186\n\nMYC contributes to E2F-induced cell cycle progression.\nPMID:11805123\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832\n\n</node>\n      <node id="ERK">\n         <data key="MY-ID">ERK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MEK1_2</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1\nMAPK3 = mitogen-activated protein kinase 3\n\nVARIABLE: ERK (any isoform) phosphorylation level\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nSpry is induced by activated ERK, through phosphorylation on Tyr55.\nPMID:15173823\n\nCaspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nRSKs interact with ERK and dissociate upon activation.\nPMID:15187187\n\nRSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nThe rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.\nPMID:16393692\n\np70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</node>\n      <node id="p38">\n         <data key="MY-ID">p38</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11\nMAPK12 = mitogen-activated protein kinase 12\nMAPK13 = mitogen-activated protein kinase 13\nMAPK14 = mitogen-activated protein kinase 14\n\nVARIABLE: p38 (any isoform) phosphorylation level\n\np38 and JNK are activated after expression of GADD45, following stress, through MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nActivated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832\n\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nMAX is phosphorylated (and activated) by p38, through complex formation.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187\n\nELK1 is a nuclear p38 target.\nPMID:20506250\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="JNK">\n         <data key="MY-ID">JNK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8\nMAPK9 = mitogen-activated protein kinase 9\nMAPK10 = mitogen-activated protein kinase 10\n\nVARIABLE: JNK (any isoform) phosphorylation level\n\np38 and JNK are activated after expression of GADD45, following stress, through MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832\n\nJNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971\n\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nJNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nELK1 is a nuclear JNK target.\nMEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="p53">\n         <data key="MY-ID">p53</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>\n         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53\n\nVARIABLE: p53 phosphorylation level\n\nATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942\n\nActivated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932\n\nThe Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.\nPMID:17158541\n\nExpression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345\n\nIn response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.\nPMID:12217521</node>\n      <node id="p21">\n         <data key="MY-ID">p21</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not AKT and p53</data>\n         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)\n\nVARIABLE: p21 activation level\n\np21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383\n\np21 is transcriptionally induced by the p53 protein.\np21 is a CDK inhibitor.\nPMID:17158541\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</node>\n      <node id="TGFBR">\n         <data key="MY-ID">TGFBR</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR_stimulus</data>\n         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1\nTGFBR2 = transforming growth factor, beta receptor II (70/80kDa)\nTGFBR3 = transforming growth factor, beta receptor III\n\nVARIABLE: TGFBR activation level\n\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta.\nPMID:21614932\n\nTGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906</node>\n      <node id="EGFR">\n         <data key="MY-ID">EGFR</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>\n         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor\n\nVARIABLE: EGFR activation level\n\nThe ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nPMID:15567848\n\nSpry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823\n\nPKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473\n\nGrb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nPMID:17496910\n\nTyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219</node>\n      <node id="FGFR3">\n         <data key="MY-ID">FGFR3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>\n         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3\n\nVARIABLE: FGFR3 activation level\n\nGrb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR\nPMID:15567848\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289\n\nBinding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221\n\n(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</node>\n      <node id="ATM">\n         <data key="MY-ID">ATM</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">DNA_damage</data>\n         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated\n\nVARIABLE: ATM phosphorylation level\n\nATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226\n\nAtaxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897\n\nATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942\n</node>\n      <node id="TAOK">\n         <data key="MY-ID">TAOK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ATM</data>\n         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1\nTAOK2 = TAO kinase 2\nTAOK3 = TAO kinase 3\n\nVARIABLE: TAOK (any isoform) phosphorylation level\n\nAtaxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897\n\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</node>\n      <node id="MAX">\n         <data key="MY-ID">MAX</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p38</data>\n         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X\n\nVARIABLE: MAX phosphorylation level\n\nMAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.\nMAX is phosphorylated (and activated) by p38, through complex formation.\nPMID:11274345</node>\n      <node id="GRB2">\n         <data key="MY-ID">GRB2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2\n\nVARIABLE: GRB2 recruitment by activated RTKs\n\nGrb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nSos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910\n\nThe signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.\nGrb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nThe ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nGrb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR\nPMID:15567848\n\nFGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436\n\nTGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</node>\n      <node id="FRS2">\n         <data key="MY-ID">FRS2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>\n         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2\n\nVARIABLE: FRS2 phosphorylation level\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289\n\nSpry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823\n\nFGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</node>\n      <node id="PI3K">\n         <data key="MY-ID">PI3K</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>\n         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide\n\nVARIABLE: PI3K activation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to\nrecruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848\n\nRecruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPhosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.\nPMID:15199124\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497</node>\n      <node id="AP1">\n         <data key="MY-ID">AP1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JUN and (FOS or ATF2)</data>\n         <data key="MY-SHAPE">ellipse</data>AP-1\n\nVARIABLE: AP-1 formation\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345\n\nAP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</node>\n      <node id="PPP2CA">\n         <data key="MY-ID">PPP2CA</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p38</data>\n         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme\n\nVARIABLE: PPP2CA activation level\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</node>\n      <node id="MEK1_2">\n         <data key="MY-ID">MEK1_2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1\nMAP2K2 = mitogen-activated protein kinase kinase 2\n\nVARIABLE: MEK (any isoform) phosphorylation level\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nAP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</node>\n      <node id="DUSP1">\n         <data key="MY-ID">DUSP1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">CREB</data>\n         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1\n\nVARIABLE: DUSP1 gene expression level\n\nDUSP1 is a CREB target gene.\nPMID:19815709\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</node>\n      <node id="MYC">\n         <data key="MY-ID">MYC</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>\n         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)\n\nVARIABLE: MYC activation level (implies gene expression)\n\nModelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein\n\nTranslocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.\nMYC inhibits p16, whereas it activates p14.\nPMID:17158541\n\nMAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.\nPMID:11274345\n\nGlycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPMID:12040186\n\nMYC contributes to E2F-induced cell cycle progression.\nPMID:11805123</node>\n      <node id="AKT">\n         <data key="MY-ID">AKT</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PDK1 and not PTEN</data>\n         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1\nAKT2 = v-akt murine thymoma viral oncogene homolog 2\nAKT3 = v-akt murine thymoma viral oncogene homolog 3\n\nVARIABLE: AKT (any isoform) phosphorylation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to\nrecruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848\n\nSignaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nGlycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPhosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186\n\np21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383\n\nPhosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359\n\nEffective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</node>\n      <node id="PLCG">\n         <data key="MY-ID">PLCG</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">EGFR or FGFR3</data>\n         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1\nPLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)\n\nVARIABLE: PLCG (any isoform) phosphorylation level\n\nBinding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221\n\nTyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219\n\nRecruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848\n\nRasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.\nPMID:17496910\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</node>\n      <node id="PKC">\n         <data key="MY-ID">PKC</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PLCG</data>\n         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha\nPRKCB = protein kinase C, beta\nPRKCG = protein kinase C, gamma\n\nVARIABLE: PKC (any isoform) enzymatic activity\n\nRecruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848\n\nPKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473\n\nPKCalpha can directly phosphorylate and activates Raf-1.\nPMID:8321321\n\n(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</node>\n      <node id="GADD45">\n         <data key="MY-ID">GADD45</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">SMAD or p53</data>\n         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha\nGADD45B = growth arrest and DNA-damage-inducible, beta\nGADD45G = growth arrest and DNA-damage-inducible, gamma\n\nVARIABLE: GADD45 gene (any isoform) expression level\n\nThe GADD45 gene is inducible by various environmental stresses.\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932\n\nExpression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345</node>\n      <node id="ELK1">\n         <data key="MY-ID">ELK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK or JNK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family\n\nVARIABLE: ELK1 phosphorylation level\n\nThe rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.\nPMID:16393692\n\nELK1 is a nuclear p38 target.\nPMID:20506250\n\nELK1 is a nuclear JNK target.\nELK1 induces c-fos expression.\nPMID:11274345</node>\n      <node id="FOS">\n         <data key="MY-ID">FOS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>\n         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog\n\nVARIABLE: FOS phosphorylation level\n\nFurther observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nELK1 induces c-fos expression.\nPMID:11274345\n\nFOS is a CREB target gene.\nPMID:19815709\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</node>\n      <node id="ATF2">\n         <data key="MY-ID">ATF2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2\n\nVARIABLE: ATF2 phosphorylation level\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</node>\n      <node id="JUN">\n         <data key="MY-ID">JUN</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK</data>\n         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene\n\nVARIABLE: JUN phosphorylation level\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nJNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nPMID:11274345</node>\n      <node id="MSK">\n         <data key="MY-ID">MSK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4\nRPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5\n\nVARIABLE: MSK (any isoform) phosphorylation level\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nMSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187\n\nTranslocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.\nPMID:17158541</node>\n      <node id="CREB">\n         <data key="MY-ID">CREB</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MSK</data>\n         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1\n\nVARIABLE: CREB phosphorylation level\n\nMSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187\n\nDUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</node>\n      <node id="RSK">\n         <data key="MY-ID">RSK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1\nRPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2\nRPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3\nRPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6\n\nVARIABLE: RSK (any isoform) phosphorylation level\n\nRSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692\n\np90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373</node>\n      <node id="SMAD">\n         <data key="MY-ID">SMAD</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2\nSMAD3 = SMAD family member 3\nSMAD4 = SMAD family member 4\n\nVARIABLE: SMAD gene (any isoform) expression level\n\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</node>\n      <node id="MTK1">\n         <data key="MY-ID">MTK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GADD45</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4\n\nVARIABLE: MTK1 phosphorylation level\n\nThe delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</node>\n      <node id="SPRY">\n         <data key="MY-ID">SPRY</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK</data>\n         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)\n\nVARIABLE: SPRY2 phosphorylation level\n\nSpry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.\nPMID:15173823</node>\n      <node id="RAF">\n         <data key="MY-ID">RAF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>\n         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1\nBRAF = v-raf murine sarcoma viral oncogene homolog B1\nARAF = v-raf murine sarcoma 3611 viral oncogene homolog\n\nVARIABLE: RAF (any isoform) phosphorylation\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nPKCalpha directly phosphorylates and activates Raf-1.\nPMID:8321321\n\nActivated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929\n\nPhosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359\n\nB-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.\nPMID:19933846</node>\n      <node id="GAB1">\n         <data key="MY-ID">GAB1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GRB2 or PI3K</data>\n         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1\n\nVARIABLE: GAB1 phosphorylation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nPMID:15567848\n\nRecruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</node>\n      <node id="PDK1">\n         <data key="MY-ID">PDK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PI3K</data>\n         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1\n\nVARIABLE: PDK1 activation level\n\nPDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nSignaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nPMID:12040186</node>\n      <node id="p70">\n         <data key="MY-ID">p70</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PDK1 and ERK</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1\nRPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2\n\nVARIABLE: p70 (any isoform) phosphorylation level\n\nPDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.\nPMID:12040186\n\np70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</node>\n      <node id="p14">\n         <data key="MY-ID">p14</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MYC</data>\n         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)\n\nVARIABLE: p14 activation level\n\nMYC inhibits p16, whereas it activates p14.\nThe MDM2 gene is transcriptionally inhibited by p14.\nPMID:17158541</node>\n      <node id="FOXO3">\n         <data key="MY-ID">FOXO3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK and not AKT</data>\n         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3\n\nVARIABLE: FOXO3 phosphorylation level\n\nJNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971\n\nFOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166\n\nPhosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186</node>\n      <node id="RAS">\n         <data key="MY-ID">RAS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">SOS or PLCG</data>\n         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog\nKRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog\nNRAS = neuroblastoma RAS viral (v-ras) oncogene homolog\n\nVARIABLE: RAS (any isoform) phosphorylation level\n\nGrb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nMost of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nActivated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929\n\nThe GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</node>\n      <node id="SOS">\n         <data key="MY-ID">SOS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GRB2 and not RSK</data>\n         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)\nSOS2 = son of sevenless homolog 2 (Drosophila)\n\nVARIABLE: SOS (any isoform) recruitment by GRB2\n\nSos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910\n\nThe signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.\nGrb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nThe process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nPMID:18039929\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n\np90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373\n</node>\n      <node id="MDM2">\n         <data key="MY-ID">MDM2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(p53 or AKT) and not p14</data>\n         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)\n\nVARIABLE: MDM2 gene expression level\n\nThe Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.\nPMID:17158541\n\nEffective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850\n</node>\n      <node id="BCL2">\n         <data key="MY-ID">BCL2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">CREB and AKT</data>\n         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2\n\nVARIABLE: BCL2 activation level (implies gene expression)\n\nModelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.\n\nDUSP1, FOS, BCL2 are CREB target genes.\nPMID:19815709\n\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186\n</node>\n      <node id="TAK1">\n         <data key="MY-ID">TAK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7\n\nVARIABLE: TAK1 phosphorylation level\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="MAP3K1_3">\n         <data key="MY-ID">MAP3K1_3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">RAS</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1\nMAP3K2 = mitogen-activated protein kinase kinase kinase 2\nMAP3K3 = mitogen-activated protein kinase kinase kinase 3\n\nVARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level\n\nThe GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</node>\n      <node id="PTEN">\n         <data key="MY-ID">PTEN</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p53</data>\n         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog\n\nVARIABLE: PTEN protein expression level\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</node>\n      <node id="HIF1">\n         <data key="MY-ID">HIF1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="EGF">\n         <data key="MY-ID">EGF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">HIF1</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="VEGF">\n         <data key="MY-ID">VEGF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">HIF1</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="Oxygen_supply">\n         <data key="MY-ID">Oxygen_supply</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="Necrosis">\n         <data key="MY-ID">Necrosis</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not Oxygen_supply</data>\n         <data key="MY-SHAPE">rect</data>\n      </node>\n      <edge source="AP1" target="MEK1_2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</edge>\n      <edge source="EGFR" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nPMID:17496910</edge>\n      <edge source="EGFR" target="PLCG" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219</edge>\n      <edge source="Oxygen_supply" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="Oxygen_supply" target="Necrosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="MSK" target="CREB" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187</edge>\n      <edge source="MSK" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.\nPMID:17158541</edge>\n      <edge source="MEK1_2" target="ERK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910</edge>\n      <edge source="HIF1" target="EGF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="HIF1" target="VEGF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="DUSP1" target="p38" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</edge>\n      <edge source="DUSP1" target="JNK" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</edge>\n      <edge source="PI3K" target="PDK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.\nPMID:15199124</edge>\n      <edge source="PI3K" target="GAB1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</edge>\n      <edge source="FGFR3" target="FRS2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</edge>\n      <edge source="FGFR3" target="PLCG" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221</edge>\n      <edge source="DNA_damage" target="ATM" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226</edge>\n      <edge source="FRS2" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</edge>\n      <edge source="MAX" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.\nPMID:11274345</edge>\n      <edge source="ATF2" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="ERK" target="MSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187</edge>\n      <edge source="ERK" target="RSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952</edge>\n      <edge source="ERK" target="SPRY" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.\nPMID:15173823</edge>\n      <edge source="ERK" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</edge>\n      <edge source="ERK" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="ERK" target="Apoptosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n</edge>\n      <edge source="ERK" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</edge>\n      <edge source="ERK" target="RAF" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.\nPMID:19933846</edge>\n      <edge source="ERK" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="CREB" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="CREB" target="DUSP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;\n\n\n\nDUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="CREB" target="BCL2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="JUN" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="FOXO3" target="Apoptosis" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166</edge>\n      <edge source="PLCG" target="RAS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.\nPMID:17496910\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</edge>\n      <edge source="PLCG" target="PKC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848</edge>\n      <edge source="SOS" target="RAS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nPMID:18039929</edge>\n      <edge source="SOS" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n</edge>\n      <edge source="p70" target="Proliferation" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186</edge>\n      <edge source="p70" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="MDM2" target="p53" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.\nPMID:17158541</edge>\n      <edge source="TGFBR" target="SMAD" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</edge>\n      <edge source="TGFBR" target="TAK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="TGFBR" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906</edge>\n      <edge source="GAB1" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</edge>\n      <edge source="MTK1" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="MTK1" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="GRB2" target="GAB1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848</edge>\n      <edge source="GRB2" target="SOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910</edge>\n      <edge source="GRB2" target="FGFR3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</edge>\n      <edge source="GRB2" target="EGFR" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nPMID:15567848</edge>\n      <edge source="GRB2" target="FRS2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</edge>\n      <edge source="BCL2" target="Apoptosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186</edge>\n      <edge source="ATM" target="TAOK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897</edge>\n      <edge source="ATM" target="p53" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942</edge>\n      <edge source="GADD45" target="MTK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="PTEN" target="AKT" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</edge>\n      <edge source="p14" target="MDM2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.\nPMID:17158541</edge>\n      <edge source="PPP2CA" target="MEK1_2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</edge>\n      <edge source="p38" target="PPP2CA" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</edge>\n      <edge source="p38" target="MSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187</edge>\n      <edge source="p38" target="p53" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932</edge>\n      <edge source="p38" target="MAX" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.\nPMID:11274345</edge>\n      <edge source="p38" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.\nPMID:20506250\n</edge>\n      <edge source="p38" target="ATF2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</edge>\n      <edge source="p53" target="GADD45" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345</edge>\n      <edge source="p53" target="p21" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.\nPMID:17158541</edge>\n      <edge source="p53" target="MDM2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.\nPMID:17158541</edge>\n      <edge source="p53" target="Apoptosis" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508</edge>\n      <edge source="p53" target="PTEN" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</edge>\n      <edge source="SMAD" target="GADD45" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</edge>\n      <edge source="SPRY" target="EGFR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823</edge>\n      <edge source="SPRY" target="FRS2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823</edge>\n      <edge source="EGFR_stimulus" target="EGFR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="ELK1" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 induces c-fos expression.\nPMID:11274345</edge>\n      <edge source="TAK1" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="TAK1" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="AKT" target="FOXO3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186</edge>\n      <edge source="AKT" target="p21" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383</edge>\n      <edge source="AKT" target="BCL2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186</edge>\n      <edge source="AKT" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPMID:12040186</edge>\n      <edge source="AKT" target="MDM2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850</edge>\n      <edge source="AKT" target="RAF" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359</edge>\n      <edge source="AKT" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="RAF" target="MEK1_2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910</edge>\n      <edge source="FOS" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="TAOK" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</edge>\n      <edge source="TAOK" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</edge>\n      <edge source="JNK" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.\nPMID:11274345</edge>\n      <edge source="JNK" target="ATF2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</edge>\n      <edge source="JNK" target="JUN" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nPMID:11274345</edge>\n      <edge source="JNK" target="FOXO3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971</edge>\n      <edge source="p21" target="Proliferation" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</edge>\n      <edge source="p21" target="Growth_Arrest" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</edge>\n      <edge source="PKC" target="EGFR" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473</edge>\n      <edge source="PKC" target="RAF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.\nPMID:8321321</edge>\n      <edge source="PKC" target="FGFR3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</edge>\n      <edge source="RSK" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</edge>\n      <edge source="RSK" target="SOS" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373</edge>\n      <edge source="PDK1" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186</edge>\n      <edge source="PDK1" target="AKT" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nPMID:12040186</edge>\n      <edge source="MYC" target="p14" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.\nPMID:17158541</edge>\n      <edge source="MYC" target="Proliferation" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.\nPMID:11805123</edge>\n      <edge source="RAS" target="RAF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929</edge>\n      <edge source="RAS" target="MAP3K1_3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="RAS" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n</edge>\n      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="MAP3K1_3" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="MAP3K1_3" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.\nProliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.\nInput signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>\n</graphml>\n';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
l[1]=
[{added: 44},
{added: 38},
{added: 23},
{added: 39},
{added: 50},
{added: 45},
{added: 49},
{added: 52},
{added: 13},
{added: 8},
{added: 37},
{added: 12},
{added: 20},
{added: 25},
{added: 27},
{added: 11},
{added: 7},
{added: 42},
{added: 36},
{added: 40},
{added: 18},
{added: 34},
{added: 30},
{added: 47},
{added: 53},
{added: 48},
{added: 14},
{added: 22},
{added: 15},
{added: 19},
{added: 5},
{added: 24},
{added: 28},
{added: 16},
{added: 51},
{added: 32},
{added: 6},
{added: 21},
{added: 46},
{added: 41},
{added: 31},
{added: 17},
{added: 33},
{added: 35},
{added: 26},
{added: 9},
{added: 4},
{added: 10},
{added: 43},
{added: 29},
{x: -2, y: -1, z: 0, who: 35},
{c: 1, x: 2, y: -1, z: 0, who: 29},
{c: 1, x: -1, y: -2, z: 1, who: 45},
{x: -1, y: 1, z: 0, who: 51},
{c: 1, x: 1, y: 2, z: 0, who: 19},
{x: 2, y: 2, z: 1, who: 41},
{c: 1, x: 1, y: -1, z: 1, who: 32},
{x: 0, y: 2, z: 1, who: 53},
{c: 1, x: 0, y: 0, z: 1, who: 34},
{c: 1, x: 0, y: -1, z: 1, who: 43},
{c: 1, x: -2, y: -2, z: 0, who: 25},
{x: 1, y: 0, z: 0, who: 10},
{x: -1, y: -1, z: 1, who: 44},
{x: -1, y: 2, z: 1, who: 33},
{c: 1, x: 2, y: -2, z: 1, who: 16},
{c: 1, x: 0, y: 2, z: 0, who: 24},
{x: -1, y: 2, z: 0, who: 20},
{x: -1, y: 0, z: 1, who: 28},
{x: -1, y: -2, z: 0, who: 13},
{c: 1, x: -2, y: 2, z: 1, who: 22},
{x: 1, y: 2, z: 1, who: 9},
{x: -1, y: 0, z: 0, who: 52},
{x: 0, y: 1, z: 1, who: 31},
{c: 1, x: 1, y: 1, z: 0, who: 39},
{x: 0, y: -2, z: 0, who: 11},
{c: 1, x: 1, y: 1, z: 1, who: 5},
{c: 1, x: -1, y: -1, z: 0, who: 37},
{x: -2, y: 2, z: 0, who: 23},
{c: 1, x: 2, y: 0, z: 1, who: 30},
{x: 0, y: -1, z: 0, who: 21},
{x: 1, y: -1, z: 0, who: 49},
{c: 1, x: -2, y: -1, z: 1, who: 26},
{x: -1, y: 1, z: 1, who: 47},
{c: 1, x: 2, y: -1, z: 1, who: 27},
{x: 0, y: 1, z: 0, who: 18},
{c: 1, x: 2, y: 1, z: 1, who: 48},
{c: 1, x: 1, y: -2, z: 0, who: 4},
{x: -2, y: 1, z: 0, who: 8},
{c: 1, x: 0, y: 0, z: 0, who: 50},
{c: 1, x: -2, y: 0, z: 0, who: 38},
{x: 2, y: 1, z: 0, who: 12},
{x: -2, y: 1, z: 1, who: 17},
{x: 2, y: -2, z: 0, who: 6},
{c: 1, x: 2, y: 0, z: 0, who: 40},
{x: 1, y: -2, z: 1, who: 46},
{c: 1, x: 1, y: 0, z: 1, who: 7},
{c: 1, x: -2, y: 0, z: 1, who: 36},
{c: 1, x: 0, y: -2, z: 1, who: 15},
{x: -2, y: -2, z: 1, who: 42},
{c: 1, x: 2, y: 2, z: 0, who: 14}];
a[1]=
{2: {'TGFBR': {a:0.42,c:0.06},
'FOXO3': {a:0.5,c:0},
'p21': {a:0.4,c:0},
'ERK': {a:0.48,c:0},
'p14': {a:0.56,c:0},
'FOS': {a:0.64,c:0},
'SOS': {a:0.44,c:0},
'FGFR3': {a:0.42,c:0.12},
'MAX': {a:0.62,c:0},
'CREB': {a:0.62,c:0},
'SPRY': {a:0.62,c:0},
'RSK': {a:0.56,c:0},
'SMAD': {a:0.56,c:0},
'MSK': {a:0.44,c:0},
'PDK1': {a:0.5,c:0},
'FRS2': {a:0.5,c:0},
'p53': {a:0.24,c:0},
'BCL2': {a:0.42,c:0},
'p70': {a:0.58,c:0},
'ATM': {a:0.44,c:0.08},
'PKC': {a:0.44,c:0},
'GAB1': {a:0.5,c:0},
'RAS': {a:0.46,c:0},
'RAF': {a:0.52,c:0},
'TAK1': {a:0.62,c:0},
'AP1': {a:0.46,c:0},
'JUN': {a:0.5,c:0},
'GADD45': {a:0.56,c:0},
'ELK1': {a:0.52,c:0},
'JNK': {a:0.42,c:0},
'MYC': {a:0.56,c:0},
'EGFR': {a:0.66,c:0.04},
'PTEN': {a:0.5,c:0},
'AKT': {a:0.4,c:0},
'HIF1': {a:0.52,c:0.1},
'GRB2': {a:0.48,c:0},
'PLCG': {a:0.58,c:0},
'PPP2CA': {a:0.54,c:0},
'MDM2': {a:0.46,c:0},
'PI3K': {a:0.56,c:0},
'MEK1_2': {a:0.5,c:0},
'DUSP1': {a:0.68,c:0},
'MTK1': {a:0.44,c:0},
'MAP3K1_3': {a:0.52,c:0},
'ATF2': {a:0.52,c:0},
'p38': {a:0.54,c:0},
'TAOK': {a:0.42,c:0}}
};
replicates[replicate_counter++] = {l: l, s: s, a: a};
l = [];
s = [];
a = [];

minimum_x = Math.min(minimum_x, -2);
maximum_x = Math.max(maximum_x, 2);
minimum_y = Math.min(minimum_y, -2);
maximum_y = Math.max(maximum_y, 2);

regulatory_graph_file_contents = '<?xml version="1.0" encoding="iso-8859-1"?>\n<!--\n			This GraphML document was generated from GXL by\n			a GXL-to-GraphML conversion style sheet.\n		-->\n<graphml>\n   <key id="MY-ID" for="node" attr.name="MY-ID" attr.type="string"/>\n   <key id="KIND" for="node" attr.name="KIND" attr.type="string"/>\n   <key id="BREED" for="node" attr.name="BREED" attr.type="string"/>\n   <key id="MY-RULE" for="node" attr.name="MY-RULE" attr.type="string"/>\n   <key id="MY-VALUE"\n        for="node"\n        attr.name="MY-VALUE"\n        attr.type="string"/>\n   <key id="MY-SHAPE"\n        for="node"\n        attr.name="MY-SHAPE"\n        attr.type="string"/>\n   <key id="PROBABILITY-OF-LINK"\n        for="edge"\n        attr.name="PROBABILITY-OF-LINK"\n        attr.type="string"/>\n   <key id="SIGN-OF-LINK"\n        for="edge"\n        attr.name="SIGN-OF-LINK"\n        attr.type="string"/>\n   <key id="BREED" for="edge" attr.name="BREED" attr.type="string"/>\n   <graph id="MAPK_large_HIF1" edgedefault="directed">\n      <node id="EGFR_stimulus">\n         <data key="MY-ID">EGFR_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce EGFR activation\n</node>\n      <node id="FGFR3_stimulus">\n         <data key="MY-ID">FGFR3_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce FGFR3 activation\n</node>\n      <node id="TGFBR_stimulus">\n         <data key="MY-ID">TGFBR_stimulus</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: any stimulus able to induce TGFBR activation\n\n</node>\n      <node id="DNA_damage">\n         <data key="MY-ID">DNA_damage</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>Model input\n\nVARIABLE: DNA damaging agents/therapies action\n\nATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226</node>\n      <node id="Apoptosis">\n         <data key="MY-ID">Apoptosis</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not BCL2 and not ERK and FOXO3 and p53</data>\n         <data key="MY-SHAPE">rect</data>Phenotype\n\nVARIABLE: apoptosis enablement\n\nModelling hypotheses: apoptosis is supposed to be enabled when at least a pro-apoptotic component is activated (p53, FOXO3) and no anti-apoptotic components are activated (ERK, BCL2).\n\nFOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166\n\nIn response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508\n\nCaspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186\n</node>\n      <node id="Growth_Arrest">\n         <data key="MY-ID">Growth_Arrest</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p21</data>\n         <data key="MY-SHAPE">rect</data>Phenotype\n\nVARIABLE: growth arrest enablement\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</node>\n      <node id="Proliferation">\n         <data key="MY-ID">Proliferation</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p70 and MYC and not p21</data>\n         <data key="MY-SHAPE">rect</data>Proliferation\n\nVARIABLE: proliferation enablement\n\nProliferation is an output variable of the model.\n\nModelling hypotheses: proliferation is supposed to be enabled when both MYC and p70 are activated, and p21 (CDK inhibitor) is inactive\n\nPDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186\n\nMYC contributes to E2F-induced cell cycle progression.\nPMID:11805123\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832\n\n</node>\n      <node id="ERK">\n         <data key="MY-ID">ERK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MEK1_2</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK1 = mitogen-activated protein kinase 1\nMAPK3 = mitogen-activated protein kinase 3\n\nVARIABLE: ERK (any isoform) phosphorylation level\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nSpry is induced by activated ERK, through phosphorylation on Tyr55.\nPMID:15173823\n\nCaspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nRSKs interact with ERK and dissociate upon activation.\nPMID:15187187\n\nRSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nThe rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.\nPMID:16393692\n\np70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</node>\n      <node id="p38">\n         <data key="MY-ID">p38</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK11 = mitogen-activated protein kinase 11\nMAPK12 = mitogen-activated protein kinase 12\nMAPK13 = mitogen-activated protein kinase 13\nMAPK14 = mitogen-activated protein kinase 14\n\nVARIABLE: p38 (any isoform) phosphorylation level\n\np38 and JNK are activated after expression of GADD45, following stress, through MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nActivated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832\n\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nMAX is phosphorylated (and activated) by p38, through complex formation.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187\n\nELK1 is a nuclear p38 target.\nPMID:20506250\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="JNK">\n         <data key="MY-ID">JNK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(TAOK and MAP3K1_3) or (MAP3K1_3 and MTK1) or (TAOK and MTK1) or (TAK1 and MTK1) or (TAK1 and MAP3K1_3) or (TAK1 and TAOK) or ((TAOK or MTK1 or MAP3K1_3 or TAK1) and not DUSP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAPK8 = mitogen-activated protein kinase 8\nMAPK9 = mitogen-activated protein kinase 9\nMAPK10 = mitogen-activated protein kinase 10\n\nVARIABLE: JNK (any isoform) phosphorylation level\n\np38 and JNK are activated after expression of GADD45, following stress, through MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832\n\nJNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971\n\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nJNK can phosphorylate the c-Juntrans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nELK1 is a nuclear JNK target.\nMEKK1 can interact with Ras in a GTP-dependent manner. The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="p53">\n         <data key="MY-ID">p53</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(ATM and p38) or ((ATM or p38) and not MDM2)</data>\n         <data key="MY-SHAPE">ellipse</data>TP53 = tumor protein p53\n\nVARIABLE: p53 phosphorylation level\n\nATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942\n\nActivated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932\n\nThe Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. p21 is transcriptionally regulated by the p53 protein.\nPMID:17158541\n\nExpression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345\n\nIn response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein.\nPMID:12217521</node>\n      <node id="p21">\n         <data key="MY-ID">p21</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not AKT and p53</data>\n         <data key="MY-SHAPE">ellipse</data>CDKN1A = cyclin-dependent kinase inhibitor 1A (p21, Cip1)\n\nVARIABLE: p21 activation level\n\np21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383\n\np21 is transcriptionally induced by the p53 protein.\np21 is a CDK inhibitor.\nPMID:17158541\n\nCell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</node>\n      <node id="TGFBR">\n         <data key="MY-ID">TGFBR</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR_stimulus</data>\n         <data key="MY-SHAPE">ellipse</data>TGFBR1 = transforming growth factor, beta receptor 1\nTGFBR2 = transforming growth factor, beta receptor II (70/80kDa)\nTGFBR3 = transforming growth factor, beta receptor III\n\nVARIABLE: TGFBR activation level\n\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta.\nPMID:21614932\n\nTGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906</node>\n      <node id="EGFR">\n         <data key="MY-ID">EGFR</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(EGFR_stimulus or SPRY) and not (PKC or GRB2)</data>\n         <data key="MY-SHAPE">ellipse</data>EGFR = epidermal growth factor receptor\n\nVARIABLE: EGFR activation level\n\nThe ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nPMID:15567848\n\nSpry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823\n\nPKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473\n\nGrb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nPMID:17496910\n\nTyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219</node>\n      <node id="FGFR3">\n         <data key="MY-ID">FGFR3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">FGFR3_stimulus and not (GRB2 or PKC)</data>\n         <data key="MY-SHAPE">ellipse</data>FGFR3 = fibroblast growth factor receptor 3\n\nVARIABLE: FGFR3 activation level\n\nGrb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR\nPMID:15567848\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289\n\nBinding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221\n\n(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</node>\n      <node id="ATM">\n         <data key="MY-ID">ATM</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">DNA_damage</data>\n         <data key="MY-SHAPE">ellipse</data>ATM = ataxia telangiectasia mutated\n\nVARIABLE: ATM phosphorylation level\n\nATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226\n\nAtaxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897\n\nATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942\n</node>\n      <node id="TAOK">\n         <data key="MY-ID">TAOK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ATM</data>\n         <data key="MY-SHAPE">ellipse</data>TAOK1 = TAO kinase 1\nTAOK2 = TAO kinase 2\nTAOK3 = TAO kinase 3\n\nVARIABLE: TAOK (any isoform) phosphorylation level\n\nAtaxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897\n\nTAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</node>\n      <node id="MAX">\n         <data key="MY-ID">MAX</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p38</data>\n         <data key="MY-SHAPE">ellipse</data>MAX = MYC associated factor X\n\nVARIABLE: MAX phosphorylation level\n\nMAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.\nMAX is phosphorylated (and activated) by p38, through complex formation.\nPMID:11274345</node>\n      <node id="GRB2">\n         <data key="MY-ID">GRB2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">EGFR or FRS2 or TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>GRB2 = growth factor receptor-bound protein 2\n\nVARIABLE: GRB2 recruitment by activated RTKs\n\nGrb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nSos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910\n\nThe signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.\nGrb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nThe ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nGrb2, bound to Cbl, does not interact directly with FGFR but, rather, binds to tyrosine-phosphorylated FRS2, promoting ubiquitination and degradation of FRS2 and FGFR\nPMID:15567848\n\nFGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436\n\nTGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</node>\n      <node id="FRS2">\n         <data key="MY-ID">FRS2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">FGFR3 and not SPRY and not GRB2</data>\n         <data key="MY-SHAPE">ellipse</data>FRS2 = fibroblast growth factor receptor substrate 2\n\nVARIABLE: FRS2 phosphorylation level\n\nFGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289\n\nSpry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823\n\nFGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</node>\n      <node id="PI3K">\n         <data key="MY-ID">PI3K</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GAB1 or (RAS and SOS)</data>\n         <data key="MY-SHAPE">ellipse</data>PIK3CA = phosphoinositide-3-kinase, catalytic, alpha polypeptide\n\nVARIABLE: PI3K activation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to\nrecruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848\n\nRecruitment and activation of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPhosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.\nPMID:15199124\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497</node>\n      <node id="AP1">\n         <data key="MY-ID">AP1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JUN and (FOS or ATF2)</data>\n         <data key="MY-SHAPE">ellipse</data>AP-1\n\nVARIABLE: AP-1 formation\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345\n\nAP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</node>\n      <node id="PPP2CA">\n         <data key="MY-ID">PPP2CA</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p38</data>\n         <data key="MY-SHAPE">ellipse</data>PPP2CA = protein phosphatase 2, catalytic subunit, alpha isozyme\n\nVARIABLE: PPP2CA activation level\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</node>\n      <node id="MEK1_2">\n         <data key="MY-ID">MEK1_2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(RAF or MAP3K1_3) and not (PPP2CA or AP1)</data>\n         <data key="MY-SHAPE">ellipse</data>MAP2K1 = mitogen-activated protein kinase kinase 1\nMAP2K2 = mitogen-activated protein kinase kinase 2\n\nVARIABLE: MEK (any isoform) phosphorylation level\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nMEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nAP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</node>\n      <node id="DUSP1">\n         <data key="MY-ID">DUSP1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">CREB</data>\n         <data key="MY-SHAPE">ellipse</data>DUSP1 = dual specificity phosphatase 1\n\nVARIABLE: DUSP1 gene expression level\n\nDUSP1 is a CREB target gene.\nPMID:19815709\n\nDUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</node>\n      <node id="MYC">\n         <data key="MY-ID">MYC</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(MSK and MAX) or (MSK and AKT)</data>\n         <data key="MY-SHAPE">ellipse</data>MYC = v-myc myelocytomatosis viral oncogene homolog (avian)\n\nVARIABLE: MYC activation level (implies gene expression)\n\nModelling hypotheses: MSK determines MYC gene expression, so it is always needed for MYC activation; MAX activates MYC protein; AKT inhibits GSK3B, which in turn inactivates MYC protein\n\nTranslocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle. c-Myc signals downstream to promote expression of cyclins that complex with CDKs.\nMYC inhibits p16, whereas it activates p14.\nPMID:17158541\n\nMAX interacts with the transcription factor c-Myc (this interaction is modelled here by complex formation) enabling c-Myc to trans-activate at least a subset of its target genes.\nPMID:11274345\n\nGlycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPMID:12040186\n\nMYC contributes to E2F-induced cell cycle progression.\nPMID:11805123</node>\n      <node id="AKT">\n         <data key="MY-ID">AKT</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PDK1 and not PTEN</data>\n         <data key="MY-SHAPE">ellipse</data>AKT1 = v-akt murine thymoma viral oncogene homolog 1\nAKT2 = v-akt murine thymoma viral oncogene homolog 2\nAKT3 = v-akt murine thymoma viral oncogene homolog 3\n\nVARIABLE: AKT (any isoform) phosphorylation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to\nrecruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848\n\nSignaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nGlycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPhosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186\n\np21 is a direct substrate of AKT and that this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383\n\nPhosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359\n\nEffective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</node>\n      <node id="PLCG">\n         <data key="MY-ID">PLCG</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">EGFR or FGFR3</data>\n         <data key="MY-SHAPE">ellipse</data>PLCG1 = phospholipase C, gamma 1\nPLCG2 = phospholipase C, gamma 2 (phosphatidylinositol-specific)\n\nVARIABLE: PLCG (any isoform) phosphorylation level\n\nBinding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221\n\nTyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219\n\nRecruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848\n\nRasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.\nPMID:17496910\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</node>\n      <node id="PKC">\n         <data key="MY-ID">PKC</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PLCG</data>\n         <data key="MY-SHAPE">ellipse</data>PRKCA = protein kinase C, alpha\nPRKCB = protein kinase C, beta\nPRKCG = protein kinase C, gamma\n\nVARIABLE: PKC (any isoform) enzymatic activity\n\nRecruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848\n\nPKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473\n\nPKCalpha can directly phosphorylate and activates Raf-1.\nPMID:8321321\n\n(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</node>\n      <node id="GADD45">\n         <data key="MY-ID">GADD45</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">SMAD or p53</data>\n         <data key="MY-SHAPE">ellipse</data>GADD45A = growth arrest and DNA-damage-inducible, alpha\nGADD45B = growth arrest and DNA-damage-inducible, beta\nGADD45G = growth arrest and DNA-damage-inducible, gamma\n\nVARIABLE: GADD45 gene (any isoform) expression level\n\nThe GADD45 gene is inducible by various environmental stresses.\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932\n\nExpression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345</node>\n      <node id="ELK1">\n         <data key="MY-ID">ELK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK or JNK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>ELK1 = ELK1, member of ETS oncogene family\n\nVARIABLE: ELK1 phosphorylation level\n\nThe rapid and efficient phosphorylation of Elk1 by ERKs is enabled by a direct interaction between the two proteins.\nPMID:16393692\n\nELK1 is a nuclear p38 target.\nPMID:20506250\n\nELK1 is a nuclear JNK target.\nELK1 induces c-fos expression.\nPMID:11274345</node>\n      <node id="FOS">\n         <data key="MY-ID">FOS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK and RSK and (ELK1 or CREB)</data>\n         <data key="MY-SHAPE">ellipse</data>FOS = FBJ murine osteosarcoma viral oncogene homolog\n\nVARIABLE: FOS phosphorylation level\n\nFurther observations: ELK1 or CREB are supposed to induce FOS gene expression, ERK or RSK are supposed to activate it\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nELK1 induces c-fos expression.\nPMID:11274345\n\nFOS is a CREB target gene.\nPMID:19815709\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</node>\n      <node id="ATF2">\n         <data key="MY-ID">ATF2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>ATF2 = activating transcription factor 2\n\nVARIABLE: ATF2 phosphorylation level\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nJNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</node>\n      <node id="JUN">\n         <data key="MY-ID">JUN</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK</data>\n         <data key="MY-SHAPE">ellipse</data>JUN = jun proto-oncogene\n\nVARIABLE: JUN phosphorylation level\n\nAP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nJNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nPMID:11274345</node>\n      <node id="MSK">\n         <data key="MY-ID">MSK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK or p38</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KA4 = ribosomal protein S6 kinase, 90kDa, polypeptide 4\nRPS6KA5 = ribosomal protein S6 kinase, 90kDa, polypeptide 5\n\nVARIABLE: MSK (any isoform) phosphorylation level\n\nMSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nMSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187\n\nTranslocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.\nPMID:17158541</node>\n      <node id="CREB">\n         <data key="MY-ID">CREB</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MSK</data>\n         <data key="MY-SHAPE">ellipse</data>CREB1 = cAMP responsive element binding protein 1\n\nVARIABLE: CREB phosphorylation level\n\nMSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187\n\nDUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</node>\n      <node id="RSK">\n         <data key="MY-ID">RSK</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KA1 = ribosomal protein S6 kinase, 90kDa, polypeptide 1\nRPS6KA2 = ribosomal protein S6 kinase, 90kDa, polypeptide 2\nRPS6KA3 = ribosomal protein S6 kinase, 90kDa, polypeptide 3\nRPS6KA6 = ribosomal protein S6 kinase, 90kDa, polypeptide 6\n\nVARIABLE: RSK (any isoform) phosphorylation level\n\nRSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952\n\nPhosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692\n\np90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373</node>\n      <node id="SMAD">\n         <data key="MY-ID">SMAD</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>SMAD2 = SMAD family member 2\nSMAD3 = SMAD family member 3\nSMAD4 = SMAD family member 4\n\nVARIABLE: SMAD gene (any isoform) expression level\n\nSmad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</node>\n      <node id="MTK1">\n         <data key="MY-ID">MTK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GADD45</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K4 = mitogen-activated protein kinase kinase kinase 4\n\nVARIABLE: MTK1 phosphorylation level\n\nThe delayed activation of p38 by TGFbeta is mediated mainly by Smad-dependent GADD45beta expression and by its subsequent activation of MTK1.\nExpression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</node>\n      <node id="SPRY">\n         <data key="MY-ID">SPRY</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">ERK</data>\n         <data key="MY-SHAPE">ellipse</data>SPRY2 = sprouty homolog 2 (Drosophila)\n\nVARIABLE: SPRY2 phosphorylation level\n\nSpry is induced by activated ERK, through phosphorylation on Tyr55. It positively regulates EGFR signalling by sequestering Cbl, whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2.\nPMID:15173823</node>\n      <node id="RAF">\n         <data key="MY-ID">RAF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(RAS or PKC) and not (ERK or AKT)</data>\n         <data key="MY-SHAPE">ellipse</data>RAF1 = v-raf-1 murine leukemia viral oncogene homolog 1\nBRAF = v-raf murine sarcoma viral oncogene homolog B1\nARAF = v-raf murine sarcoma 3611 viral oncogene homolog\n\nVARIABLE: RAF (any isoform) phosphorylation\n\nOnce activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910\n\nPKCalpha directly phosphorylates and activates Raf-1.\nPMID:8321321\n\nActivated Ras recruits cytoplasmic Raf (MAPKKK) for activation. There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929\n\nPhosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359\n\nB-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.\nPMID:19933846</node>\n      <node id="GAB1">\n         <data key="MY-ID">GAB1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GRB2 or PI3K</data>\n         <data key="MY-SHAPE">ellipse</data>GAB1 = GRB2-associated binding protein 1\n\nVARIABLE: GAB1 phosphorylation level\n\nGrb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by EGFR or FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nPMID:15567848\n\nRecruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</node>\n      <node id="PDK1">\n         <data key="MY-ID">PDK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PI3K</data>\n         <data key="MY-SHAPE">ellipse</data>PDK1 = pyruvate dehydrogenase kinase, isozyme 1\n\nVARIABLE: PDK1 activation level\n\nPDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nSignaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nPMID:12040186</node>\n      <node id="p70">\n         <data key="MY-ID">p70</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">PDK1 and ERK</data>\n         <data key="MY-SHAPE">ellipse</data>RPS6KB1 = ribosomal protein S6 kinase, 70kDa, polypeptide 1\nRPS6KB2 = ribosomal protein S6 kinase, 70kDa, polypeptide 2\n\nVARIABLE: p70 (any isoform) phosphorylation level\n\nPDK1 is a PI3K target, leading to activation of p70 (through phosphorylation) and subsequent cell growth.\nPMID:12040186\n\np70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</node>\n      <node id="p14">\n         <data key="MY-ID">p14</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">MYC</data>\n         <data key="MY-SHAPE">ellipse</data>CDKN2A = cyclin-dependent kinase inhibitor 2A (melanoma, p16, inhibits CDK4)\n\nVARIABLE: p14 activation level\n\nMYC inhibits p16, whereas it activates p14.\nThe MDM2 gene is transcriptionally inhibited by p14.\nPMID:17158541</node>\n      <node id="FOXO3">\n         <data key="MY-ID">FOXO3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">JNK and not AKT</data>\n         <data key="MY-SHAPE">ellipse</data>FOXO3 = forkhead box O3\n\nVARIABLE: FOXO3 phosphorylation level\n\nJNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971\n\nFOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166\n\nPhosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186</node>\n      <node id="RAS">\n         <data key="MY-ID">RAS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">SOS or PLCG</data>\n         <data key="MY-SHAPE">ellipse</data>HRAS = v-Ha-ras Harvey rat sarcoma viral oncogene homolog\nKRAS = v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog\nNRAS = neuroblastoma RAS viral (v-ras) oncogene homolog\n\nVARIABLE: RAS (any isoform) phosphorylation level\n\nGrb2 molecules recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nMost of the signals activating the ERK pathway are initiated through receptor-mediated activation of the small G-protein, Ras. Ras is a membrane-bound protein activated through the exchange of bound GDP to GTP. The process of activating Ras thereby requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nActivated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929\n\nThe GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</node>\n      <node id="SOS">\n         <data key="MY-ID">SOS</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">GRB2 and not RSK</data>\n         <data key="MY-SHAPE">ellipse</data>SOS1 = son of sevenless homolog 1 (Drosophila)\nSOS2 = son of sevenless homolog 2 (Drosophila)\n\nVARIABLE: SOS (any isoform) recruitment by GRB2\n\nSos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910\n\nThe signaling pathways activated by FGFRs substantially overlap with those activated by EGFRs.\nGrb2 molecules bound to FRS2 recruit the nucleotide exchange factor SOS, leading to the activation of the Ras-MAPK signaling cascade.\nThe process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nPMID:18039929\n\nPI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n\np90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373\n</node>\n      <node id="MDM2">\n         <data key="MY-ID">MDM2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">(p53 or AKT) and not p14</data>\n         <data key="MY-SHAPE">ellipse</data>MDM2 = Mdm2, p53 E3 ubiquitin protein ligase homolog (mouse)\n\nVARIABLE: MDM2 gene expression level\n\nThe Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2. The MDM2 gene is, in turn, transcriptionally inhibited by p14, providing another fine level of control for p53 activity.\nPMID:17158541\n\nEffective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850\n</node>\n      <node id="BCL2">\n         <data key="MY-ID">BCL2</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">CREB and AKT</data>\n         <data key="MY-SHAPE">ellipse</data>BCL2 = B-cell CLL/lymphoma 2\n\nVARIABLE: BCL2 activation level (implies gene expression)\n\nModelling hypotheses: CREB is needed for BCL2 gene expression, AKT is needed for BCL2 activation.\n\nDUSP1, FOS, BCL2 are CREB target genes.\nPMID:19815709\n\nAkt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186\n</node>\n      <node id="TAK1">\n         <data key="MY-ID">TAK1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">TGFBR</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K7 = mitogen-activated protein kinase kinase kinase 7\n\nVARIABLE: TAK1 phosphorylation level\n\nActivated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</node>\n      <node id="MAP3K1_3">\n         <data key="MY-ID">MAP3K1_3</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">RAS</data>\n         <data key="MY-SHAPE">ellipse</data>MAP3K1 = mitogen-activated protein kinase kinase kinase 1\nMAP3K2 = mitogen-activated protein kinase kinase kinase 2\nMAP3K3 = mitogen-activated protein kinase kinase kinase 3\n\nVARIABLE: MAP3K1/2/3 (any isoform) phosphorylation level\n\nThe GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</node>\n      <node id="PTEN">\n         <data key="MY-ID">PTEN</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p53</data>\n         <data key="MY-SHAPE">ellipse</data>PTEN = phosphatase and tensin homolog\n\nVARIABLE: PTEN protein expression level\n\np53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</node>\n      <node id="HIF1">\n         <data key="MY-ID">HIF1</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">p70 and ERK and not Oxygen_supply</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="EGF">\n         <data key="MY-ID">EGF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">HIF1</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="VEGF">\n         <data key="MY-ID">VEGF</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">HIF1</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="Oxygen_supply">\n         <data key="MY-ID">Oxygen_supply</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-SHAPE">ellipse</data>\n      </node>\n      <node id="Necrosis">\n         <data key="MY-ID">Necrosis</data>\n         <data key="BREED">objects</data>\n         <data key="KIND">Gene</data>\n         <data key="MY-VALUE">1</data>\n         <data key="MY-RULE">not Oxygen_supply</data>\n         <data key="MY-SHAPE">rect</data>\n      </node>\n      <edge source="AP1" target="MEK1_2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>AP-1 mediated gene expression inhibits ERK phosphorylation.\nPMID:18039929</edge>\n      <edge source="EGFR" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Grb2 is an adaptor protein, normally present in cytosol. It is recruited to the plasma membrane by activated RTKs.\nPMID:17496910</edge>\n      <edge source="EGFR" target="PLCG" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Tyrosine phosphorylation of PLC-gamma by EGF receptor leads to its activation.\nPMID:2472219</edge>\n      <edge source="Oxygen_supply" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="Oxygen_supply" target="Necrosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="TGFBR_stimulus" target="TGFBR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="MSK" target="CREB" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 mediates the mitogen-stimulated phoshphorylation of CREB.\nPMID:15187187</edge>\n      <edge source="MSK" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Translocation of ERK into the nucleus activates the mitogen activated and stress-activated protein kinase 1 (MSK1), a histone H3 kinase that can relax chromatin, thus making it more transcriptionally accessible. This alteration in the chromatin state induces MYC, a gene that encodes the c-Myc protein, a transcription factor that controls the cell cycle.\nPMID:17158541</edge>\n      <edge source="MEK1_2" target="ERK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910</edge>\n      <edge source="HIF1" target="EGF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="HIF1" target="VEGF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="DUSP1" target="p38" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</edge>\n      <edge source="DUSP1" target="JNK" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>DUSP1 preferentially inactivates JNK and p38.\nPMID:19436832</edge>\n      <edge source="PI3K" target="PDK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphatidylinositol-3,4,5-triphosphate (PIP3), the reaction product of PI 3-kinase, is responsible for activation of phosphoinositide-dependent kinase (PDK) and the antiapoptotic protein kinase Akt.\nPMID:15199124</edge>\n      <edge source="PI3K" target="GAB1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</edge>\n      <edge source="FGFR3" target="FRS2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</edge>\n      <edge source="FGFR3" target="PLCG" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Binding of FGF to FGFR leads to tyrosine phosphorylation of PLCgamma.\nPMID:1656221</edge>\n      <edge source="DNA_damage" target="ATM" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ATM autophosphorylates itself at serine 1981 in response to DNA damage and some other cellular stresses.\nPMID:18235226</edge>\n      <edge source="FRS2" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FGF stimulation leads to phosphorylation of Shp2 on a tyrosine residue that forms a complex with an additional molecule of Grb2. Grb2/Sos complexes are thus recruited directly and indirectly via Shp2 upon tyrosine phosphorylation of FRS2a in response to growth factor stimulation.\nPMID:11447289</edge>\n      <edge source="MAX" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MAX interacts with the transcription factor c-Myc, enabling c-Myc to trans-activate at least a subset of its target genes.\nPMID:11274345</edge>\n      <edge source="ATF2" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="ERK" target="MSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187</edge>\n      <edge source="ERK" target="RSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>RSK2 is a well known ERK substrate in the cytoplasm and has been shown to undergo autophosphorylation after ERK phosphorylation.\nPMID:15239952</edge>\n      <edge source="ERK" target="SPRY" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK, through phosphorylation on Tyr55.\nPMID:15173823</edge>\n      <edge source="ERK" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</edge>\n      <edge source="ERK" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="ERK" target="Apoptosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Caspase 9 is efficiently phosphorylated on Thr125 by ERK in vitro, suggesting that it is targeted directly by ERK in vivo. This is one of the ways ERK plays its anti-apoptotic role.\nPMID:12792650\n</edge>\n      <edge source="ERK" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p70 activation requires both ERK cascade and PI3K/AKT cascade, at least in some cell types.\nPMID:11940578, PMID:10601235, PMID:11431469</edge>\n      <edge source="ERK" target="RAF" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>B-Raf phosphorylated by activated ERK and find that feedback phosphorylation of B-Raf inhibits binding to activated Ras.\nPMID:19933846</edge>\n      <edge source="ERK" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="CREB" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="CREB" target="DUSP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>&lt;probability p = 0.438 /&gt;\n\n\n\nDUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="CREB" target="BCL2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>DUSP1, FOS and BCL2 are CREB target genes.\nPMID:19815709</edge>\n      <edge source="JUN" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="FOXO3" target="Apoptosis" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>FOXO3a promotes apoptosis through activation of PUMA.\nPMID:20978166</edge>\n      <edge source="PLCG" target="RAS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>RasGRP1 is a C1-domain containing protein that is activated by DAG and Ca2+, in a manner analogous to members of the PKC family.\nPMID:17496910\n\nCalcium liberated from internal stores by IP3 acts on the calcium- and DAG-sensitive RasGRP1 and causes it to translocate to the Golgi. RasGRP1 activates Golgi-associated Ras on this compartment.\nPMID:16488589</edge>\n      <edge source="PLCG" target="PKC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment to the membrane and tyrosine phosphorylation enhance the enzymatic activity of PLC-g, leading to the formation of two second messengers, diacylglycerol (DAG) and inositol 1,4,5-trisphosphate (IP3). IP3 releases Ca2+ from internal stores, which in turn acts in concert with DAG to translocate protein kinase C (PKC) to the cell membrane and stimulate its enzymatic activity.\nPMID:15567848</edge>\n      <edge source="SOS" target="RAS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The process of activating Ras requires the recruitment of proteins responsible for initiating GDP/GTP exchange to the membrane, such as SOS (son of sevenless).\nPMID:18039929</edge>\n      <edge source="SOS" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n</edge>\n      <edge source="p70" target="Proliferation" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186</edge>\n      <edge source="p70" target="HIF1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="MDM2" target="p53" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.\nPMID:17158541</edge>\n      <edge source="TGFBR" target="SMAD" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</edge>\n      <edge source="TGFBR" target="TAK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="TGFBR" target="GRB2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TGF-beta-induced ShcA phosphorylation induces ShcA association with Grb2 and Sos, thereby initiating the well-characterised pathway linking receptor tyrosine kinases with Erk MAP kinases.\nPMID:17673906</edge>\n      <edge source="GAB1" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Recruitment of PI 3-kinase by Gab1 results in a positive-feedback loop mediated by binding of the PH domain of Gab1 to the product of PI3-kinase activation, phosphatidylinositol-3,4,5-triphosphate.\nPMID:15199124</edge>\n      <edge source="MTK1" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="MTK1" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="GRB2" target="GAB1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Grb2 recruits the docking protein Gab1, which is tyrosine phosphorylated by FGFR, leading to recruitment and activation of the PI3K-Akt cell survival pathway.\nEGF stimulates a cell survival pathway mediated by phosphoinositide 3-kinase (PI3K) and the protein kinase Akt by an indirect mechanism in which tyrosine phosphorylation of the docking protein Gab1 or ErbB3 by EGFR leads to recruitment and activation of PI3K.\nPMID:15567848</edge>\n      <edge source="GRB2" target="SOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Sos is recruited from the cytosol to the plasma membrane as a result of its constitutive interaction with Grb2. It is in an autoinhibited state.\nPMID:17496910</edge>\n      <edge source="GRB2" target="FGFR3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</edge>\n      <edge source="GRB2" target="EGFR" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The ubiquitin ligase Cbl interacts with EGFR directly and indirectly through Grb2, promoting ubiquitination and degradation of EGFR\nPMID:15567848</edge>\n      <edge source="GRB2" target="FRS2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>FGF-induced tyrosine phosphorylation of FRS2 results in complex formation with the adaptor protein Grb2 bound to Cbl by means of its SH3 domains. FGF-induced ternary complex formation among FRS2, Grb2, and Cbl results in ubiquitination and degradation of FRS2 and FGF receptor (FGFR).\nPMID:11997436</edge>\n      <edge source="BCL2" target="Apoptosis" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186</edge>\n      <edge source="ATM" target="TAOK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Ataxia telangiectasia mutated (ATM) is activated in response to DNA damage and directly phosphorylates TAOK.\nPMID:18855897</edge>\n      <edge source="ATM" target="p53" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ATM phosphorylates p53 at ser15 and stabilize it.\nPMID:15140942</edge>\n      <edge source="GADD45" target="MTK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 genes in mammalian cells strongly activates co-expressed MTK1, as well as p38 and JNK, the MAPKs downstream of MTK1.\nPMID:21614932</edge>\n      <edge source="FGFR3_stimulus" target="FGFR3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="PTEN" target="AKT" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</edge>\n      <edge source="p14" target="MDM2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>The MDM2 gene is transcriptionally inhibited by p14.\nPMID:17158541</edge>\n      <edge source="PPP2CA" target="MEK1_2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</edge>\n      <edge source="p38" target="PPP2CA" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEK is continuously dephosphorylated by PP2A (PPP2CA), whose activity is stimulated by p38: p38 activity increases the physical association between PP2A and MEK/ERK complex.\nPMID:18039929</edge>\n      <edge source="p38" target="MSK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MSK1 and MSK2 are potently activated (by phosphorylation) in vivo by ERK1/2 and p38 but not JNK.\nPMID:15187187</edge>\n      <edge source="p38" target="p53" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated p38 phosphorylates p53 at several residues, including Ser33, and thereby increases the transcriptional activity of p53. Activated p53 then induces apoptosis in cells that have suffered extensive DNA damage. When the intensity of the damage is moderate, however, the cell can repair the damage and escape apoptotic cell death by down-regulating p38-p53 signaling. Under these conditions, p53-induced Wip1 phosphatase interacts with activated p38 and selectively dephosphorylates p38 at the phospho-Thr180 residue, thereby reducing its kinase activity towards p53.\nPMID:21614932</edge>\n      <edge source="p38" target="MAX" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MAX is phosphorylated (and activated) by p38, through complex formation.\nPMID:11274345</edge>\n      <edge source="p38" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 is a nuclear p38 target.\nPMID:20506250\n</edge>\n      <edge source="p38" target="ATF2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</edge>\n      <edge source="p53" target="GADD45" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Expression of GADD45 is the culmination of a signaling pathway that requires prior expression of the tumor suppressor protein p53 which trans-activates the GADD45 gene.\nPMID:11274345</edge>\n      <edge source="p53" target="p21" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p21 is transcriptionally induced by the p53 protein.\nPMID:17158541</edge>\n      <edge source="p53" target="MDM2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The Mdm2 protein is involved in an autoregulatory feedback loop with p53, thus controlling its activity. Increased p53 levels transactivate the MDM2 promoter causing its upregulation. The translated protein then binds to p53 and transports it to the proteasome for ubiquitin-mediated degradation. The resultant lowered p53 levels then reduce the levels of Mdm2.\nPMID:17158541</edge>\n      <edge source="p53" target="Apoptosis" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>In response to genotoxic stress such as DNA damage, PUMA is transactivated by p53 (leading to apoptosis).\nPMID:19641508</edge>\n      <edge source="p53" target="PTEN" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>p53 induces transcription of the PTEN gene and elevates cellular levels of the PTEN protein, which in turn inhibits activation of Akt.\nPMID:12217521</edge>\n      <edge source="SMAD" target="GADD45" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Smad-dependent gene expression can provoke p38 activation in response to TGFbeta. GADD45beta was identified as the TGFbeta-inducible gene whose expression activates the p38 pathway.\nPMID:21614932</edge>\n      <edge source="SPRY" target="EGFR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823</edge>\n      <edge source="SPRY" target="FRS2" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Spry is induced by activated ERK. It positively regulates EGFR signalling by sequestering Cbl (positive feedback in the model, from Spry to EGFR), whereas it negatively regulates FGFR signalling by sequestering Grb2 from FSR2 (negative feedback in the model, from Spry to FSR2).\nPMID:15173823</edge>\n      <edge source="EGFR_stimulus" target="EGFR" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="ELK1" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 induces c-fos expression.\nPMID:11274345</edge>\n      <edge source="TAK1" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="TAK1" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated TGFBR1/TGFBR2 complex attracts the TAB2/TAB3 proteins which promote activation of TAK1 (by phosphorylation), leading to activation of p38 and JNK.\nPMID:20060931</edge>\n      <edge source="AKT" target="FOXO3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Phosphorylation of FOXO3 by Akt creates a binding site for the 14-3-3 family of proteins. The complex of FOXO3 and 14-3-3 is retained in the cytosol, blocking transcription of genes normally stimulated by FOXO3.\nPMID:12040186</edge>\n      <edge source="AKT" target="p21" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p21 is a direct substrate of AKT and this may regulate the subcellular localisation of p21. Phosphorylation of p21 by AKT correlated with its exit from the nucleus and with increased cell cycle progression.\nPMID:11882383</edge>\n      <edge source="AKT" target="BCL2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Akt phosphorylation of the apoptosis-inducing protein Bad creates a binding site for 14-3-3 proteins and prevents Bad from binding to Bcl-2 family members Bcl-2 and Bcl-XL, thus releasing them for a cell survival response.\nPMID:12040186</edge>\n      <edge source="AKT" target="MYC" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Glycogen synthase kinase 3 (GSK3) is a target of Akt. This protein kinase is constitutively active in unstimulated cells and phosphorylates many proteins (including glycogen synthase, c-Myc, and cyclin D) to keep them in inactive states or promote their degradation. Phosphorylation of GSK3 (both alpha and beta isoforms) by Akt turns off the catalytic activity of this enzyme, resulting in the activation of pathways that are normally repressed by GSK3.\nPMID:12040186</edge>\n      <edge source="AKT" target="MDM2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Effective recruitment of Akt by appropriate survival signals may lead to activation of Mdm2, inactivation of p53, and eventually inhibition of p53-dependent apoptosis.\nPMID:11850850</edge>\n      <edge source="AKT" target="RAF" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Phosphorylation by Akt inhibits EGF-dependent activity of B-Raf and C-Raf.\nPMID:10869359</edge>\n      <edge source="AKT" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>\n      </edge>\n      <edge source="RAF" target="MEK1_2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Once activated, Raf family members are capable of initiating the phosphorylation cascade, whereby Raf activates MEK, and MEK in turn activates ERK.\nPMID:17496910</edge>\n      <edge source="FOS" target="AP1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>AP-1 is a heterodimer comprised of bZIP transcription factors, typically c-Jun and JunD, along with members of the fos (usually c-Fos) and ATF (usually ATF2) families. All bZIP transcription factors contain leucine zippers that enable homo- and heterodimerisation, and AP-1 components are organised into Jun-Jun, Jun-Fos, or Jun-ATF dimers.\nPMID:11274345</edge>\n      <edge source="TAOK" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</edge>\n      <edge source="TAOK" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>TAO kinases are MAP3Ks that function upstream of p38 and JNK.\nPMID:21614932</edge>\n      <edge source="JNK" target="ELK1" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>ELK1 is a nuclear JNK target.\nPMID:11274345</edge>\n      <edge source="JNK" target="ATF2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK and p38 can both phosphorylate ATF2 at Thr69 and Ser71.\nPMID:11274345</edge>\n      <edge source="JNK" target="JUN" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK can phosphorylate the c-Jun trans-activating domain at Ser63 and Ser73. JNKs, but not the ERKs or p38s, binds c-Jun quite strongly.\nPMID:11274345</edge>\n      <edge source="JNK" target="FOXO3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>JNK phosphorylates MST1 at serine 82, which leads to the enhancement of MST1 activation. The activation of MST1 phosphorylates FOXO3 at serine 207 and promotes cell death.\nPMID:20028971</edge>\n      <edge source="p21" target="Proliferation" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</edge>\n      <edge source="p21" target="Growth_Arrest" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Cell growth arrest is induced through cyclin-dependent kinase inhibitor p21 WAF1/CIP1.\nPMID:8614832</edge>\n      <edge source="PKC" target="EGFR" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>PKC may cause phosphorylation of EGFR, leading to a decrease in its activity.\nPMID:6321473</edge>\n      <edge source="PKC" target="RAF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PKCalpha can directly phosphorylate and activates Raf-1.\nPMID:8321321</edge>\n      <edge source="PKC" target="FGFR3" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>(Inferred from FGFR1) PKC-mediated phosphorylation can lead to internalization and degradation of FGFR1.\nPMID:9694798, PMID:8622701, PMID:14699054</edge>\n      <edge source="RSK" target="FOS" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Phosphorylation by both ERKs and their downstream RSKs can stabilise the c-Fos protein for several hours. The combination of these phosphorylations allows c-Fos sustained activity.\nPMID:16393692</edge>\n      <edge source="RSK" target="SOS" sign="negative">\n         <data key="SIGN-OF-LINK">negative</data>\n         <data key="BREED">influences</data>p90 Rsk-2 is involved in SOS phosphorylation and may be important in down-regulation of the growth factor response.\nPMID:9242373</edge>\n      <edge source="PDK1" target="p70" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PDK1 is a PI3K target, leading to activation of p70 and subsequent cell growth.\nPMID:12040186</edge>\n      <edge source="PDK1" target="AKT" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Signaling proteins with pleckstrin-homology (PH) domains accumulate at sites of PI3K activation by directly binding to PI(3,4,5)P3. Of particular interest are the protein serine-threonine kinases Akt and phosphoinositide-dependent kinase 1 (PDK1). Association with PI(3,4,5)P3 at the membrane brings these proteins into proximity and facilitates phosphorylation of Akt by PDK1. This phosphorylation stimulates the catalytic activity of Akt, resulting in the phosphorylation of a host of other proteins that affect cell growth, cell cycle entry, and cell survival.\nPMID:12040186</edge>\n      <edge source="MYC" target="p14" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MYC inhibits p16, whereas it activates p14.\nPMID:17158541</edge>\n      <edge source="MYC" target="Proliferation" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MYC contributes to E2F-induced cell cycle progression.\nPMID:11805123</edge>\n      <edge source="RAS" target="RAF" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>Activated Ras recruits cytoplasmic Raf (MAPKKK) for activation (through phosphorylation). There are three mammalian serine/threonine Raf kinases: A-Raf, B-Raf, and Raf-1 (also known as C-Raf). All three Raf proteins share the same downstream MAPKK substrate mitogen activated protein kinase kinases 1,2 (MEK1,2).\nPMID:18039929</edge>\n      <edge source="RAS" target="MAP3K1_3" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="RAS" target="PI3K" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>PI3K is a well characterised effector of RAS, through GRB2/SOS pathway.\nPMID:21779497\n</edge>\n      <edge source="MAP3K1_3" target="MEK1_2" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="MAP3K1_3" target="JNK" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>The GTP-dependent coupling of MEKK1 to Ras indicates that MEKK1 may be an effector for those agonists that recruit JNK through Ras-dependent mechanisms. MEKK1 selectively activates the endogenous JNK pathway. MEKK1 can activate MEK4 and MEK7 in vivo.\nMEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>\n      <edge source="MAP3K1_3" target="p38" sign="positive">\n         <data key="SIGN-OF-LINK">positive</data>\n         <data key="BREED">influences</data>MEKK2 and MEKK3 can activate JNK, p38 and ERK pathways.\nPMID:11274345</edge>Logical model representing MAPK cascades activation following EGFR, FGFR3, TGFBR and DNA damage stimuli.\nProliferation, Apoptosis and Growth arrest outputs (in blue) provide a readout of the contributions of the three main MAPK pathways (ERK, p38, JNK) to cell fate decision.\nInput signals are emphasised in pink, whereas the reference MAPK are emphasised in lila.</graph>\n</graphml>\n';
mutations_file_contents = 'Percent 0.5 0.5<br>EGFR -0.1 1.0<br>p53 -0.1 0.0<br>';
associations_file_contents = '"EGF" ""<br>"VEGF" ""<br>"Oxygen" "Oxygen_supply"<br>';
diffusion_file_contents = 'EGF 0.00 0.00 2.7e-14 0.01 1.00<br>Oxygen 0.08 0.08 1.0e-9 5e-4 0.00<br>';
input_parameters_file_contents = 'DNA_damage 0 0.5<br>EGFR_stimulus 0.01 0.5<br>TGFBR_stimulus 0 0.5<br>FGFR3_stimulus 0 0.5<br>Oxygen_supply 0.08 0.02<br>';
parameters = ['the-number-of-replicates', 4.0,'the-maximum-number-of-ticks', 10.0,'the-maximum-cell-count', 300.0,'the-initial-cell-count', 50.0];
start_time = '04:00:51.848 PM 01-Sep-2016';
number_of_replicates_requested = 4;
default_size = 1;
default_colors = ['rgba(164, 164, 164, 0.71)', 'rgba(223, 91, 84, 0.71)'];
running_3D = true;

gene_nodes[2] = [
{id: 1308, label: "RAS"},
{id: 1286, label: "AP1"},
{id: 1295, label: "FOXO3"},
{id: 1276, label: "PTEN"},
{id: 1314, label: "FGFR3"},
{id: 1291, label: "MSK"},
{id: 1321, label: "p53"},
{id: 1315, label: "MDM2"},
{id: 1331, label: "JUN"},
{id: 1304, label: "JNK"},
{id: 1284, label: "FOS"},
{id: 1292, label: "DUSP1"},
{id: 1297, label: "PLCG"},
{id: 1303, label: "TGFBR"},
{id: 1316, label: "TAOK"},
{id: 1328, label: "TAK1"},
{id: 1287, label: "ATM"},
{id: 1298, label: "CREB"},
{id: 1323, label: "PPP2CA"},
{id: 1332, label: "EGFR"},
{id: 1299, label: "PI3K"},
{id: 1279, label: "FRS2"},
{id: 1305, label: "MEK1_2"},
{id: 1294, label: "MAP3K1_3"},
{id: 1309, label: "BCL2"},
{id: 1283, label: "MYC"},
{id: 1310, label: "p70"},
{id: 1329, label: "SMAD"},
{id: 1330, label: "PDK1"},
{id: 1288, label: "ELK1"},
{id: 1318, label: "GRB2"},
{id: 1280, label: "HIF1"},
{id: 1327, label: "GADD45"},
{id: 1313, label: "RAF"},
{id: 1312, label: "PKC"},
{id: 1324, label: "RSK"},
{id: 1282, label: "MAX"},
{id: 1290, label: "MTK1"},
{id: 1289, label: "ERK"},
{id: 1326, label: "AKT"},
{id: 1319, label: "GAB1"},
{id: 1293, label: "ATF2"},
{id: 1281, label: "p14"},
{id: 1322, label: "SPRY"},
{id: 1300, label: "p38"},
{id: 1311, label: "SOS"},
{id: 1317, label: "p21"}
];
input_nodes[2] = [
{id: 1307, label: "FGFR3_stimulus"},
{id: 1301, label: "TGFBR_stimulus"},
{id: 1302, label: "Oxygen_supply"},
{id: 1285, label: "DNA_damage"},
{id: 1277, label: "EGFR_stimulus"}
];
output_nodes[2] = [
{id: 1325, label: "EGF"},
{id: 1278, label: "VEGF"}
];
links_between_nodes[2] = [
{from: 1326, to: 1309, color: "green"},
{from: 1300, to: 1288, color: "green"},
{from: 1300, to: 1323, color: "green"},
{from: 1283, to: 1281, color: "green"},
{from: 1279, to: 1318, color: "green"},
{from: 1289, to: 1324, color: "green"},
{from: 1298, to: 1309, color: "green"},
{from: 1308, to: 1299, color: "green"},
{from: 1314, to: 1297, color: "green"},
{from: 1307, to: 1314, color: "green"},
{from: 1300, to: 1321, color: "green"},
{from: 1297, to: 1308, color: "green"},
{from: 1308, to: 1294, color: "green"},
{from: 1308, to: 1313, color: "green"},
{from: 1316, to: 1300, color: "green"},
{from: 1284, to: 1286, color: "green"},
{from: 1298, to: 1292, color: "green"},
{from: 1295, to: 1306, color: "green"},
{from: 1309, to: 1306, color: "red"},
{from: 1312, to: 1332, color: "red"},
{from: 1294, to: 1305, color: "green"},
{from: 1315, to: 1321, color: "red"},
{from: 1297, to: 1312, color: "green"},
{from: 1323, to: 1305, color: "red"},
{from: 1326, to: 1295, color: "red"},
{from: 1330, to: 1326, color: "green"},
{from: 1318, to: 1279, color: "red"},
{from: 1280, to: 1278, color: "green"},
{from: 1293, to: 1286, color: "green"},
{from: 1311, to: 1308, color: "green"},
{from: 1300, to: 1293, color: "green"},
{from: 1303, to: 1328, color: "green"},
{from: 1310, to: 1320, color: "green"},
{from: 1318, to: 1319, color: "green"},
{from: 1326, to: 1317, color: "red"},
{from: 1332, to: 1297, color: "green"},
{from: 1328, to: 1304, color: "green"},
{from: 1289, to: 1322, color: "green"},
{from: 1324, to: 1284, color: "green"},
{from: 1299, to: 1330, color: "green"},
{from: 1302, to: 1280, color: "green"},
{from: 1321, to: 1306, color: "green"},
{from: 1289, to: 1288, color: "green"},
{from: 1326, to: 1310, color: "green"},
{from: 1294, to: 1304, color: "green"},
{from: 1316, to: 1304, color: "green"},
{from: 1313, to: 1305, color: "green"},
{from: 1291, to: 1283, color: "green"},
{from: 1314, to: 1279, color: "green"},
{from: 1289, to: 1310, color: "green"},
{from: 1290, to: 1300, color: "green"},
{from: 1288, to: 1284, color: "green"},
{from: 1303, to: 1329, color: "green"},
{from: 1287, to: 1321, color: "green"},
{from: 1304, to: 1331, color: "green"},
{from: 1301, to: 1303, color: "green"},
{from: 1312, to: 1314, color: "red"},
{from: 1281, to: 1315, color: "red"},
{from: 1289, to: 1291, color: "green"},
{from: 1290, to: 1304, color: "green"},
{from: 1321, to: 1276, color: "green"},
{from: 1326, to: 1313, color: "red"},
{from: 1300, to: 1291, color: "green"},
{from: 1304, to: 1288, color: "green"},
{from: 1312, to: 1313, color: "green"},
{from: 1276, to: 1326, color: "red"},
{from: 1322, to: 1279, color: "red"},
{from: 1277, to: 1332, color: "green"},
{from: 1318, to: 1314, color: "red"},
{from: 1291, to: 1298, color: "green"},
{from: 1331, to: 1286, color: "green"},
{from: 1289, to: 1280, color: "green"},
{from: 1326, to: 1283, color: "green"},
{from: 1317, to: 1320, color: "red"},
{from: 1300, to: 1282, color: "green"},
{from: 1298, to: 1284, color: "green"},
{from: 1289, to: 1306, color: "red"},
{from: 1329, to: 1327, color: "green"},
{from: 1319, to: 1299, color: "green"},
{from: 1303, to: 1318, color: "green"},
{from: 1289, to: 1284, color: "green"},
{from: 1327, to: 1290, color: "green"},
{from: 1292, to: 1304, color: "red"},
{from: 1318, to: 1332, color: "red"},
{from: 1305, to: 1289, color: "green"},
{from: 1302, to: 1296, color: "red"},
{from: 1282, to: 1283, color: "green"},
{from: 1287, to: 1316, color: "green"},
{from: 1285, to: 1287, color: "green"},
{from: 1299, to: 1319, color: "green"},
{from: 1322, to: 1332, color: "green"},
{from: 1321, to: 1317, color: "green"},
{from: 1304, to: 1295, color: "green"},
{from: 1330, to: 1310, color: "green"},
{from: 1292, to: 1300, color: "red"},
{from: 1328, to: 1300, color: "green"},
{from: 1324, to: 1311, color: "red"},
{from: 1304, to: 1293, color: "green"},
{from: 1280, to: 1325, color: "green"},
{from: 1332, to: 1318, color: "green"},
{from: 1311, to: 1299, color: "green"},
{from: 1310, to: 1280, color: "green"},
{from: 1317, to: 1275, color: "green"},
{from: 1294, to: 1300, color: "green"},
{from: 1321, to: 1327, color: "green"},
{from: 1318, to: 1311, color: "green"},
{from: 1283, to: 1320, color: "green"},
{from: 1326, to: 1315, color: "green"},
{from: 1286, to: 1305, color: "red"},
{from: 1289, to: 1313, color: "red"},
{from: 1321, to: 1315, color: "green"}
];
gene_nodes[1] = [
{id: 180, label: "MEK1_2"},
{id: 228, label: "ATF2"},
{id: 221, label: "JUN"},
{id: 176, label: "PKC"},
{id: 203, label: "p21"},
{id: 189, label: "TAK1"},
{id: 225, label: "SOS"},
{id: 186, label: "ERK"},
{id: 173, label: "p53"},
{id: 193, label: "FGFR3"},
{id: 220, label: "MAX"},
{id: 226, label: "EGFR"},
{id: 182, label: "p38"},
{id: 211, label: "PLCG"},
{id: 213, label: "CREB"},
{id: 210, label: "PI3K"},
{id: 184, label: "SPRY"},
{id: 208, label: "PDK1"},
{id: 192, label: "AP1"},
{id: 188, label: "ATM"},
{id: 199, label: "BCL2"},
{id: 215, label: "RSK"},
{id: 191, label: "TGFBR"},
{id: 205, label: "FRS2"},
{id: 200, label: "PPP2CA"},
{id: 223, label: "JNK"},
{id: 185, label: "FOXO3"},
{id: 174, label: "ELK1"},
{id: 179, label: "TAOK"},
{id: 217, label: "RAF"},
{id: 229, label: "MYC"},
{id: 222, label: "MAP3K1_3"},
{id: 197, label: "DUSP1"},
{id: 202, label: "MTK1"},
{id: 219, label: "p14"},
{id: 206, label: "AKT"},
{id: 196, label: "SMAD"},
{id: 214, label: "HIF1"},
{id: 198, label: "FOS"},
{id: 181, label: "p70"},
{id: 190, label: "PTEN"},
{id: 227, label: "MSK"},
{id: 201, label: "GAB1"},
{id: 183, label: "GRB2"},
{id: 207, label: "MDM2"},
{id: 194, label: "RAS"},
{id: 178, label: "GADD45"}
];
input_nodes[1] = [
{id: 175, label: "Oxygen_supply"},
{id: 224, label: "DNA_damage"},
{id: 195, label: "EGFR_stimulus"},
{id: 187, label: "TGFBR_stimulus"},
{id: 216, label: "FGFR3_stimulus"}
];
output_nodes[1] = [
{id: 218, label: "EGF"},
{id: 177, label: "VEGF"}
];
links_between_nodes[1] = [
{from: 226, to: 211, color: "green"},
{from: 222, to: 223, color: "green"},
{from: 191, to: 183, color: "green"},
{from: 183, to: 201, color: "green"},
{from: 206, to: 185, color: "red"},
{from: 175, to: 214, color: "green"},
{from: 179, to: 182, color: "green"},
{from: 208, to: 206, color: "green"},
{from: 175, to: 209, color: "red"},
{from: 210, to: 208, color: "green"},
{from: 203, to: 212, color: "red"},
{from: 202, to: 223, color: "green"},
{from: 189, to: 223, color: "green"},
{from: 222, to: 180, color: "green"},
{from: 192, to: 180, color: "red"},
{from: 222, to: 182, color: "green"},
{from: 186, to: 214, color: "green"},
{from: 190, to: 206, color: "red"},
{from: 191, to: 189, color: "green"},
{from: 223, to: 228, color: "green"},
{from: 226, to: 183, color: "green"},
{from: 215, to: 225, color: "red"},
{from: 184, to: 205, color: "red"},
{from: 211, to: 176, color: "green"},
{from: 187, to: 191, color: "green"},
{from: 179, to: 223, color: "green"},
{from: 188, to: 179, color: "green"},
{from: 193, to: 211, color: "green"},
{from: 186, to: 217, color: "red"},
{from: 197, to: 223, color: "red"},
{from: 195, to: 226, color: "green"},
{from: 201, to: 210, color: "green"},
{from: 213, to: 199, color: "green"},
{from: 173, to: 230, color: "green"},
{from: 185, to: 230, color: "green"},
{from: 182, to: 228, color: "green"},
{from: 194, to: 210, color: "green"},
{from: 208, to: 181, color: "green"},
{from: 200, to: 180, color: "red"},
{from: 198, to: 192, color: "green"},
{from: 206, to: 199, color: "green"},
{from: 223, to: 185, color: "green"},
{from: 188, to: 173, color: "green"},
{from: 228, to: 192, color: "green"},
{from: 173, to: 190, color: "green"},
{from: 210, to: 201, color: "green"},
{from: 182, to: 227, color: "green"},
{from: 206, to: 207, color: "green"},
{from: 206, to: 229, color: "green"},
{from: 217, to: 180, color: "green"},
{from: 181, to: 212, color: "green"},
{from: 182, to: 220, color: "green"},
{from: 173, to: 207, color: "green"},
{from: 213, to: 198, color: "green"},
{from: 219, to: 207, color: "red"},
{from: 206, to: 203, color: "red"},
{from: 206, to: 217, color: "red"},
{from: 176, to: 193, color: "red"},
{from: 194, to: 217, color: "green"},
{from: 229, to: 212, color: "green"},
{from: 181, to: 214, color: "green"},
{from: 186, to: 174, color: "green"},
{from: 194, to: 222, color: "green"},
{from: 207, to: 173, color: "red"},
{from: 221, to: 192, color: "green"},
{from: 182, to: 200, color: "green"},
{from: 227, to: 213, color: "green"},
{from: 178, to: 202, color: "green"},
{from: 173, to: 203, color: "green"},
{from: 225, to: 194, color: "green"},
{from: 186, to: 184, color: "green"},
{from: 191, to: 196, color: "green"},
{from: 197, to: 182, color: "red"},
{from: 214, to: 177, color: "green"},
{from: 213, to: 197, color: "green"},
{from: 186, to: 215, color: "green"},
{from: 183, to: 225, color: "green"},
{from: 223, to: 221, color: "green"},
{from: 176, to: 217, color: "green"},
{from: 186, to: 230, color: "red"},
{from: 211, to: 194, color: "green"},
{from: 186, to: 198, color: "green"},
{from: 176, to: 226, color: "red"},
{from: 224, to: 188, color: "green"},
{from: 183, to: 193, color: "red"},
{from: 220, to: 229, color: "green"},
{from: 206, to: 181, color: "green"},
{from: 173, to: 178, color: "green"},
{from: 186, to: 227, color: "green"},
{from: 180, to: 186, color: "green"},
{from: 193, to: 205, color: "green"},
{from: 183, to: 205, color: "red"},
{from: 214, to: 218, color: "green"},
{from: 199, to: 230, color: "red"},
{from: 183, to: 226, color: "red"},
{from: 227, to: 229, color: "green"},
{from: 223, to: 174, color: "green"},
{from: 203, to: 204, color: "green"},
{from: 229, to: 219, color: "green"},
{from: 216, to: 193, color: "green"},
{from: 182, to: 173, color: "green"},
{from: 182, to: 174, color: "green"},
{from: 186, to: 181, color: "green"},
{from: 184, to: 226, color: "green"},
{from: 174, to: 198, color: "green"},
{from: 202, to: 182, color: "green"},
{from: 215, to: 198, color: "green"},
{from: 196, to: 178, color: "green"},
{from: 225, to: 210, color: "green"},
{from: 189, to: 182, color: "green"},
{from: 205, to: 183, color: "green"}
];
</script>
</head>
<body>
</body>
</html>

